# **ACTC1 -- Hypertrophic cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:143

The ACTC1 gene has been associated with autosomal dominant hypertrophic cardiomyopathy (HCM) in at least 6 probands in 4 publications. Four unique variants (missense) with convincing evidence of pathogenicity have been reported in humans, including de novo inheritance with maternity and paternity confirmed in two cases and segregation with disease in 26 additional family members. ACTC1 was first associated with this disease in humans in 1999 (Mogensen et al, PMID 10330430). More evidence is available in the literature, but the maximum score for genetic evidence (12 pts.) has been reached. The mechanism for disease is unknown. The ACTC1 gene was significantly enriched for missense variants in Walsh et al, 2016 (PMID 27532257). Overall, the gene was found to have an Odds Ratio of 8.59 (5.06-14.5) for HCM. This gene-disease association is supported by expression studies, in vitro functional assays, and an animal model. In summary, ACTC1 is
definitively associated with autosomal dominant HCM. This has been repeatedly demonstrated in both the research and clinical diagnostic settings and has been upheld over time. This classification was approved by the ClinGen Hypertrophic Cardiomyopathy Expert Panel on September 5, 2017.

**Heterozygous missense mutations and in-frame codon deletions of ACTC1 are the major types of pathogenic variants found in patients with apical hypertrophic cardiomyopathy and left ventricular non compaction, congenital heart defects and arrhythmia (PMID:17611253, 26061005).** Thus far, there are only a few reports of mutations that could be interpreted to support haploinsufficiency of ACTC1 \[PMID 1794298, 24503780\]; however, the evidence for this remains incomplete. Large deletions or
duplications have not been described.

**Literature review:**

In a cohort of 368 unrelated patients with HCM (sporadic or familial), 3 unique missense mutations were identified. 2 of these mutations were adjacent to regions of actin-actin and actin-myosin interaction. Authors suggested the mutations may affect actin-myosin interaction and force generation.

*Olson et al. 2000 (PMID:10966831)*

Walsh et al found an excess of non-truncating variants associated with HCM. There was no excess of truncating variants in comparison to reference dataset (ExAC).

*Walsh et al, 2016 (PMID 27532257)*
<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=ACTC1&icc=HCM>

"...myofibrils isolated from the hearts of *ACTC* E99K transgenic (TG) mice have enhanced Ca^2+^ sensitivity of force. We also measured work and heat output by papillary muscles from *ACTC* E99K mice and from their non-TG littermates, allowing a direct comparison of the efficiency of contraction. These measurements show that the papillary muscle of the *ACTC* E99K HCM mouse is hypercontractile and that hypercontractility is associated with lower efficiency."

*Song W et al. 2013 (PMID:23604709)*

"In a large 3-generation family with hypertrophic cardiomyopathy [Mogensen et al. (1999)] PMID 10330430 identified heterozygosity for a missense mutation in the ACTC1 gene that was located near 2 missense mutations previously identified as causing an inherited form of dilated cardiomyopathy (CMD1R). The variant was a 253G-T transversion in exon 5 of the ACTC gene resulting in an ala295-to-ser substitution. The ala at position 295 is conserved in 19 different species. **The expression of the actin mutation in this family gave the impression of a highly penetrant disease with diverse phenotypes and variable age of onset.** Only 1 individual of 13 family members carrying the mutant allele was nonpenetrant, and morbidity was low, as only 3 of the 13 carrying the mutant allele had symptoms of the disease. **The authors stated that ACTC1 was the first sarcomeric gene described in which mutations are responsible for 2 different cardiomyopathies, and hypothesized that ACTC1 mutations affecting sarcomere contraction lead to HCM and that mutations affecting force transmission from the sarcomere to the surrounding syncytium lead to dilated cardiomyopathy."**

*OMIM <https://www.omim.org/entry/102540>*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

**Allelic requirement:**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is not definitively known but may involve impact on sarcomere force generation due to **altered gene product structure**. Heterozygous missense mutations and in-frame deletions are the major types of pathogenic variants found. There are only a few reports of mutations that could be interpreted to support haploinsufficiency of ACTC1 \[PMID 1794298, 24503780\]; however, the evidence for this remains incomplete. Large deletions or duplications have not been described. The variant identified in the original family appeared to be highly penetrant but with a variable age of onset. 

**Additional information related to ACMG evidence types**

**PM1** 
Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

ACTC1 etiological fraction across the whole gene is 0.884 (0.826–0.941) so would enable PM1_supporting to be applied

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
AF>0.1% (het) 
AF>3.16% (hom) 
Due to the rarity of our disorders this threshold was reduced and this rule is activated for all variants with a filtering allele frequency
cardioclassifier

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) <0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**List variant classes in this gene proven to cause this disease:**

Missense

In frame deletion

**List other variant classes predicted to lead to the same functional consequence**

Frameshift predicted to escape NMD

Stop_gained predicted to escape NMD

Splice donor variant

Splice donor variant predicted to escape NMD

Splice acceptor variant predicted to escape NMD

Stop_lost

In frame_insertion

