**BMPR1A -- Juvenile polyposis syndrome, (MIM 174900)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:1076

**Literature Review:**

.. identified four BMPR1A **germline truncation pathogenic**
**variants** in four juvenile polyposis kindreds. Genomic sequencing of
BMPR1A in each of these juvenile polyposis kindreds disclosed these
truncation pathogenic variants in all affected kindred members but not
in normal individuals (figure 2). Individual Case Evidence: Segregation
Among Similarly Affected Family Members ?
NM_004329.2(BMPR1A):c.44_47delTGTT (p.Leu15Serfs) ?
NM_004329.2(BMPR1A):c.715C\>T (p.Gln239Ter) ?
NM_004329.2(BMPR1A):c.812G\>A (p.Trp271Ter) ?
NM_004329.2(BMPR1A):c.961del (p.Phe320_Leu321insTer)

***Howe et al, 2001, PMID 11381269***

...a large cohort study including a total of 77 Juvenile polyposis (JP)
cases. Germline BMPR1A gene pathogenic variants were identified in 16
cases (20.8%). These included **ten truncation (including nonsense and
deletion) variants and six missense** mutations.

***Howe et al, 2004, PMID 15235019***

...a large prospective, referral-based study of 603 patients with
moderate-load colorectal polyps. Twenty cases (3.3%) were identified
carrying a BMPR1A pathogenic variants including **twelve BMPR1A focal
deletion and/or truncation variants and eight misssense** variants. All
the BMPR1A pathogenic variants carriers were diagnosed with Juvenile
polyposis and/or other types of cancers (supplementary table).

***Ngeow et al, 2013, PMID 23399955***

Forty-three types of BMPR1A gene mutations were deposited in the Human
Mutation Database at the Institute of Medical Genetics in Cardiff. Most
often, these were nucleotide changes generating a stop codon
(**nonsense**) or leading to amino acid changes (**missense**). These
mutations are **distributed evenly in the entire gene** sequence. Five
other mutations were reported at the gene assembly: **two in intron 1,
and one in each of introns 3, 4 and 5.** **Small deletions**, most often
identified between codons 224 and 359, constitute a considerable
proportion of mutations in the BMPR1A gene.

***Cichy et al, 2014, PMID: 25097590***

Overall, **frameshift, nonsense, and missense** variants accounted for
the majority of pathogenic SMAD4 (72.9%) and BMPR1A (61.8%) alterations
in the ECS as well as the

LBSB group (SMAD4: 79.9%; BMPR1A: 70.8%; Supplementary Table 3). Only
**large genomic, i.e., single or multiexon deletions** in SMAD4 were
significantly overrepresented in the ECS compared with the LBSB group.
**Splice site variants were noted** in 4-6% of SMAD4 and 10-16% of
BMPAR1A

***Blatter et al, 2020, PMID: 32398773***

**OMIM:** https://www.omim.org/entry/174900 (and 601299

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1469/

*Mechanism action*

BMPR1A is not known to be a tumor suppressor gene, although few studies
have examined it in cancer. BMPR1A is a **type I cell surface receptor**
for the BMP pathway. Ligands, such as TGF-β or BMP, bind to a receptor
and **activate signaling pathways** leading to protein complexes that
migrate to the nucleus and bind directly to DNA sequences to regulate
**transcription** \[Heldin et al 1997\]. The downstream genes under the
control of these signaling pathways are still being actively
investigated.

*Pathogenic variants.*

Sixty pathogenic variants, including **insertions, deletions, and
missense, nonsense, and splice site alterations,** have been described
\[Calva-Cerqueira et al 2009\]. **Germline deletions or pathogenic
missense variants of the promoter** have also been described
\[Calva-Cerqueira et al 2010\]. **Large deletions** of BMPR1A may also
occur in up to 6% of individuals and be associated with additional or
more severe clinical features \[Aretz et al 2007, van Hattem et al 2008,
Calva-Cerqueira et al 2009\].

*Abnormal gene product.*

Abnormal BMPR1A proteins frequently result from pathogenic DNA variants
in **the protein kinase domain and occasionally by variants in the
cysteine-rich region of the extracellular domain**. No pathogenic
variants have been described in the transmembrane domain \[Howe et al
2004\]. In vitro studies have shown that proteins resulting from BMPR1A
pathogenic **missense** variants as seen in individuals with JPS are
**retained in the cytoplasm and do not traffic to the cell membrane**
like the wild-type protein \[Howe et al 2013\].

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Dose change* -decreased gene product level

*Altered gene product structure*

**Narrative summary of molecular mechanisms:**

The precise molecular mechanism is not known and BMPR1A is not known to
be a tumour suppressor gene. Variants cluster in the protein kinase
domain and occasionally in the cysteine-rich region of the extracellular
domain. Variant classes include small insertions, small (considerable
proportion of mutations ) and large deletions, missense, nonsense, and
splice site alterations. Germline deletions or pathogenic missense
variants of the promoter have also been described.

**List variant classes in this gene proven to cause this disease:**

Stop_gained

missense

Frameshift_variant

Splice_acceptor_variant

Splice_donor_variant

regulatory_region_variant (deletions/missense in promotor region)

**Potential novel variant classes based on predicted functional
consequence**

splice_region_variant

splice_acceptor_variant predicted to undergo NMD

splice_acceptor_variant predicted to escape NMD

splice_donor_variant predicted to undergo NMD

splice_donor_variant predicted to escape NMD

start_lost

frameshift_variant predicted to undergo NMD

frameshift_variant predicted to escape NMD

stop_gained predicted to undergo NMD

stop_gained predicted to escape NMD

stop_lost

inframe_insertion

inframe_deletion

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[uORF\]

Stop lost \[oORF\]

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])

**Not included**

**synonymous_variant**

**intron_variant**

**intergenic_variant**

**5_prime_UTR_variant**

**3_prime_UTR_variant**
