**BRCA2 -- Breast-ovarian cancer, familial 2 (MIM 612555)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:1101

Many **loss of function mutations** have been described; see OMIM gene
entry for details. Heterozygous mutations of BRCA2 cause susceptibility
to breast and ovarian cancer (see OMIM), a phenotype that is not
appropriate for our purposes. Homozygous mutations of BRCA2 cause
Fanconi anemia (see GeneReviews), which is an appropriate phenotype but
is an inappropriate mutational mechanism. No evidence of dosage
pathology

**Literature Review:**

**OMIM:** https://www.omim.org/entry/612555

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1247/

Finally, *Brca2* knockout mice can be partially rescued by crossing with
a *Tp53* knockout strain, suggesting that these genes interact with
the *TP53* -mediated DNA damage checkpoint \[[Brugarolas & Jacks
1997](https://www.ncbi.nlm.nih.gov/books/NBK1247/)\]. Therefore, the
available evidence indicates that ***BRCA2* is a \"caretaker,\"
like *TP53*, serving to
maintain [genomic] integrity**
\[[Zhang et al 1998]. It
is likely that *BRCA2* will eventually be implicated in a variety of
cellular processes, only some of which will be related to their role in
the etiology of breast and ovarian cancer.

Most *BRCA2* pathogenic
variants reported to date consist of **frameshift deletions, insertions,
or nonsense variants that predict premature truncation of protein transcription, consistent with the loss of function** that is expected with clinically significant
variants of tumor suppressor genes. Cells lacking BRCA2 are deficient in
the repair of double-strand DNA breaks, as reflected in a
hypersensitivity to ionizing radiation \[[Venkitaraman
2001](https://www.ncbi.nlm.nih.gov/books/NBK1247/)\].

Reduced-penetrance variant. There is evidence that p.Lys3326Ter is
associated with risk of developing breast and ovarian cancers
independent of other **pathogenic variants in BRCA2, although penetrance
is reduced**. This was demonstrated through a large case-control study
based on an international consortium of patients with cancer. Further
studies are needed to determine the biologic mechanism of action
responsible for these associations \[Meeks et al 2015\].

1,650 unique BRCA1 and 1,731 unique BRCA2 mutations were identified. The
unique mutations and number of families in which each mutation was
observed are listed in Supplementary Table 1. In each gene, the five
most common mutations (including founder mutations) accounted for 33% of
all mutations in BRCA1..

..most common mutation type was **frameshift followed by nonsense**. The
most common effect of BRCA1 and BRCA2 mutations was **premature
translation termination and most of the mutant mRNAs were predicted to
undergo nonsense-mediated mRNA decay (NMD)** (Anczukow, et al., 2008).
Despite having the same spectrum of mutations in BRCA1 and BRCA2, the
frequency distribution by mutation type, effect, or function differed
significantly (p\<0.05) between BRCA1 and BRCA2 mutation carriers for
many groups, as shown in Table 1. These observed differences are largely
**because genomic rearrangements and missense mutations account for a
much higher proportion of mutations in BRCA1 when compared with BRCA2**

As expected, the most common mutations in the entire data set were the
**founder mutations** BRCA1 c.5266dup (5382insC), BRCA1 c.68_69del
(185delAG), and **BRCA2 c.5946del (6174delT).....**

***Rebbeck et al, 2018, PMID: 29446198***

9/18 kindreds identified potentially deleterious sequence alterations in
the BRCA2 gene. All except 1, a **deletion of 3 nucleotides, involved
nucleotide deletions that altered the reading frame, leading to
truncation of the BRCA2 protein. No missense or nonsense mutations were
found.** The authors noted that the mutational profile of BRCA2 differs
from that of BRCA1: microinsertions and point mutations are about as
common in BRCA1 as **microdeletions, which predominate in BRCA2**

***Tavtigian et al, 1996,*** ***8589730***

Spliceogenic mutations of group B included 6 already analyzed (BRCA1
c.212G\>A, c.213−11T\>G, and c.4484G\>T, and **BRCA2.... c.631G\>A,
c.8754+3G\>C, and c.9117G\>A)** \[11\], \[18\], \[19\], \[21\], \[22\],
\[26\], \[44\]--\[50\] and 2 newly characterized (c.134+3_134+6delAAGT,
c.4986+5G\>A in BRCA1). Three mutations caused the skipping of an entire
exon: BRCA1 c.4484G\>T (exon 14) (Fig. 2A), BRCA2 c.631G\>A (exon 7)
(Fig. 2B) and BRCA2 c.9117G\>A (exon 23) (Fig. 2C). Conversely, partial
intronic retention caused by the activation of cryptic splice sites was
observed for BRCA1 c.213−11T\>G (59 bp at the 3′-end of intron 5) (Fig.
2D), BRCA1c.4986+5G\>A (65 bp at the 5′-end of intron 16) (Fig. 2E), and
**for BRCA2 c.8754+3G\>C (46 bp at the 5′-end of intron 21**) (Fig. 2F).
Finally, the BRCA1 c.134+3_134+6delAAGT and c.212G\>A variants were
associated with a relevant increase, in comparison with normal controls,
of the Δexon3 and Δexon5q (missing 22 bp at the 3′-end of exon 5)
isoforms, respectively (Fig. 2G--H). Both isoforms contain PTCs

***Colombo et al, 2013, PMID: 23451180***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

Optional modifiers: incomplete penetrance

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly loss of function (*BRCA2* is a tumour
suppressor gene) leading to genomic instability. Premature translation
termination is the most common outcome with most (though not all) mutant
mRNAs predicted to undergo nonsense-mediated mRNA decay. Variants
include large deletions/duplications, frameshift, in-frame
deletions/insertions, missense, nonsense, splice site (including
non-canonical, deeper intronic) and large scale structural
rearrangement. The mutational spectrum is similar to BRCA1 though
deletions are more common with BRCA2, with proportionatly less missense
and large scale genomic rearrangements. Less evidence identified for
variants in 3'and 5' UTRs,

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

(frameshift_variant predicted to escape NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Missense

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

stop_lost

Inframe_insertion

5_prime_UTR_variant

3_prime_UTR_variant

intron_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
