**BRCA1 -- Breast-ovarian cancer, familial 1 (MIM 604370)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:1100

Loss of function mutations in BRCA1 (**nonsense, frameshift, splice site, and exonic deletions**) as well as **whole gene deletions** of BRCA1 have been associated with cancer development (Genereviews and
PMIDs: 21989022, 17661172, and 22762150). The penetrance associated with BRCA1 mutations is still an active area of study; however, patients with pathogenic BRCA1 mutations are thought to have an **increased lifetime
risk of developing breast cancer (50-80% in females, 1-2% in males), ovarian cancer (24-40%), prostate cancer (up to 30%), and pancreatic
cancer (1-7%)** (Genereviews Table 3).

No evidence triploidy

**Literature Review:**

**OMIM:** <https://www.omim.org/entry/604370>

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1247/

Pathogenic variants. More than **1800 pathogenic variants** have been
identified in BRCA1. While a small number of these variants have been
identified repeatedly in unrelated families, the vast majority have not
been reported in more than a few families.

**Reduced-penetrance variant**. p.Arg1699Gln is established to be a
reduced penetrance variant in BRCA1 \[Spurdle et al 2012\]. Data from
functional assays were ambiguous for deficiency across multiple assays.
Thus, this allele was determined to be associated with intermediate risk
for breast and ovarian cancer, highlighting challenges for risk modeling
and clinical management of individuals of this and other potential
moderate-risk variants.

Abnormal gene product. **Most BRCA1 pathogenic variants lead to frameshifts resulting in a missing or non-functional protein**. In
cancers from individuals with a BRCA1 germline pathogenic variant, the
normal allele is deleted or inactivated, resulting in somatic
inactivation of BRCA1. This strongly suggests that **BRCA1 is a
tumor-suppressor gene whose loss of function can result in genomic
instability,** resulting in a high susceptibility to malignant
transformation \[Smith et al 1992, Deng 2006\]. Additional evidence in
support of a tumor suppressor function is that overexpression of the
BRCA1 protein leads to growth suppression similar to that seen with the
classic tumor suppressors TP53 and the retinoblastoma gene (RB1) \[Holt
et al 1996\]. **Loss of function of BRCA1 results in defects in DNA
repair, defects in transcription, abnormal centrosome duplication,
defective G2/M cell-cycle checkpoint regulation, impaired spindle
checkpoint, and chromosome damage** \[Brodie & Deng 2001, Deng 2002,
Venkitaraman 2002\].

1,650 unique BRCA1 and 1,731 unique BRCA2 mutations were identified. The
unique mutations and number of families in which each mutation was
observed are listed in Supplementary Table 1. In each gene, the five
most common mutations (including founder mutations) accounted for 33% of
all mutations in BRCA1..

..most common mutation type was **frameshift followed by nonsense**. The
most common effect of BRCA1 and BRCA2 mutations was **premature
translation termination and most of the mutant mRNAs were predicted to
undergo nonsense-mediated mRNA decay (NMD)** (Anczukow, et al., 2008).
Despite having the same spectrum of mutations in BRCA1 and BRCA2, the
frequency distribution by mutation type, effect, or function differed
significantly (p\<0.05) between BRCA1 and BRCA2 mutation carriers for
many groups, as shown in Table 1. These observed differences are largely
because **genomic rearrangements and missense mutations account for a
much higher proportion of mutations in BRCA1** when compared with BRCA2

As expected, the most common mutations in the entire data set were the
**founder mutations** BRCA1 c.5266dup (5382insC), BRCA1 c.68_69del
(185delAG), and BRCA2 c.5946del (6174delT)..... The **large
rearrangement mutation** in BRCA1 c.548-?4185+?del (ex9-12del) appears
to be an important founder mutation in Mexico, with findings of a common
haplotype and an estimated age at 74 generations (\~1,500 years)

***Rebbeck et al, 2018, PMID: 29446198***

***...***dominantly **inherited 5\' UTR variant associated with
epigenetic BRCA1 silencing due to promoter hypermethylation** in two
families affected by breast and ovarian cancer....multiple women are
affected by grade 3 breast cancer or high-grade serous ovarian
cancer.....RNA sequencing revealed the allelic loss of BRCA1 expression
in both families and that this loss of expression segregated with the
heterozygous variant c.-107A\>T in the BRCA1 5\' UTR.

***Evans et al, 2018, PMID: 30075112***

variant in the 3\'UTR of BRCA1 is functional, leading to decreased BRCA1
expression, modest increased breast cancer risk, and most importantly,
presentation with stage IV breast cancer

***Dorairaj, 2013, PMID: 24915755***

Spliceogenic mutations of group B included 6 already analyzed (*BRCA1* c.212G\>A, c.213−11T\>G, and c.4484G\>T, and *BRCA2....*c.631G\>A, c.8754+3G\>C, and
c.9117G\>A) and 2 newly characterized (c.134+3_134+6delAAGT, c.4986+5G\>A in *BRCA1*).
Three mutations caused the skipping of an entire
exon: *BRCA1* c.4484G\>T (exon 14) ([Fig.
2A](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)), *BRCA2* c.631G\>A
(exon 7) ([Fig.
2B](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly))
and *BRCA2* c.9117G\>A (exon 23) ([Fig.
2C](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)).
Conversely, partial intronic retention caused by the activation of
cryptic splice sites was observed for *BRCA1* c.213−11T\>G (59 bp at the
3′-end of intron 5) ([Fig.
2D](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)), *BRCA1*c.4986+5G\>A
(65 bp at the 5′-end of intron 16) ([Fig.
2E](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)),
and for *BRCA2* c.8754+3G\>C (46 bp at the 5′-end of intron 21) ([Fig.
2F](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)).
Finally, the *BRCA1* c.134+3_134+6delAAGT and c.212G\>A variants were
associated with a relevant increase, in comparison with normal controls,
of the Δexon3 and Δexon5q (missing 22 bp at the 3′-end of exon 5)
isoforms, respectively ([Fig.
2G--H](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)).
Both isoforms contain PTCs

***Colombo et al, 2013, PMID: 23451180***

BRCA1 c.5073 A \> T variant might play a pathogenic role in HOC syndrome
in this family (synonymous -- single case study)

***Minucci, 2018, PMID: 29760936***

The intronic BRCA1 c.5407-25T\>A variant causing partly skipping of exon
23-a likely pathogenic variant with reduced penetrance?

***Høberg-Vetti H, 2020,PMID: 32203205***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

Optional modifiers: incomplete penetrance

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly loss of function (*BRCA1* is a tumour suppressor gene) leading to genomic instability. Premature translation termination is the most common outcome with most (though not all) mutant
mRNAs predicted to undergo nonsense-mediated mRNA decay. Variants include large deletions/duplications, frameshift, in-frame deletions/insertions, missense, nonsense, splice site (including non-canonical, deeper intronic) and large scale structural rearrangement. Less frequently reported variants involve both 3'and 5'
UTRs, in the latter associated with promotor hypermethylation. There is
insufficient evidence for pathogenicity of synonymous variants. Genomic
rearrangements and missense mutations are more common in *BRCA1* when
compared with *BRCA2*

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

(frameshift_variant predicted to escape NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Inframe_insertion

Missense

5_prime_UTR_variant

3_prime_UTR_variant

intron_variant

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

stop_lost

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
