# **DSC2 -- Arrhythmogenic Right Ventricular Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:3036

The relationship between DSC2 and arrhythmogenic right ventricular
dysplasia (autosomal dominant) and arrhythmogenic right ventricular dysplasia with mild palmoplantar keratoderma with or without woolly hair (autosomal recessive) was evaluated using the ClinGen Clinical Validity Framework as of July, 2018. Variants in DSC2 were first reported in humans with this disease as early as 2006 (Syrris et al., PMID: 17033975). At least 13 variants (e.g. missense, nonsense, frameshift) have been reported in humans. Evidence supporting this gene-disease relationship includes case-level data and experimental data. Summary of Case Level Data: 8.5 points. Variants in this gene have been reported in at least 13 probands in 6 publications (PMIDs: 17963498, 21062920,
23863954, 17186466, 18957847, 17033975). This gene-disease relationship is supported by animal models, expression studies, and protein interactions. In summary, DSC2 is definitively associated with ARVD (AD) and ARVD with mild palmoplantar keratoderma with or without woolly hair (AR). This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This
classification was approved by the ClinGen Arrythmogenic Right
Ventricular Cardiomyopathy Gene Curation Expert Panel on September 14,2018 (SOP Version 6)

**ClinGen Haploinsufficiency comments**

DSC2 encodes the desmosomal cadherin, desmocollin 2. Sequence-level variants in DSC2 are associated with arrhythmogenic right ventricular cardiomyopathy 11 (AVRC) (OMIM: 610476), a genetically heterogeneous cardiovascular disorder characterized by fibrofatty replacement of the right ventricular myocardium, ventricular arrhythmia, and an increased risk of premature sudden cardiac death.

**A large number (n\>50) of DSC2 mutations have been reported in the literature, the majority of which are missense mutations.** Several mutations that may confer a loss-of-function have also been reported, including splice-site, intragenic deletions/duplications, and nonsense mutations, which have been observed in association with both autosomal dominant (PMID: 17186466 and 17033975) and autosomal recessive (PMID:20400443, 19863551, 24793512, 23863954) forms of AVRC. Incomplete penetrance for some DSC2 variants has also been observed.

Although more than two independent publications with loss-of-function-type variants provides evidence in support of DSC2 haploinsufficiency, due to the variable inheritance patterns, mutational spectrum, and lack of reports of whole gene deletions, the haploinsufficiency score is a 2.

**Literature review:**

More than 50 pathogenic variants have been described. In addition, **multiple instances of digenic inheritance have been identified with DSC2 variants along with other desmosomal gene mutations [Bhuiyan et al 2009, Groeneweg et al 2015] and compound heterozygosity [Lorenzon et al 2015] and homozygosity [Wong et al 2014].**

Desmocollin pathogenic isoforms that lack the last 37 amino acid residues of the carboxyl-terminal domain of DSC2a are unable to bind plakoglobin. It is unknown how the variants affect desmosome formation, but it is speculated that the result would be impaired desmosome structure or function.

*Gene reviews 
https://www.ncbi.nlm.nih.gov/books/NBK1131/#arvd.Molecular_Genetics*

**The molecular links between desmosome mutations and the pathological hallmarks of ACM --- cardiomyocyte loss, fibrosis, adipogenesis, inflammation and arrhythmogenesis --- remain poorly defined. Probable pathogenic mechanisms include loss of mechanical integrity at cell--cell junctions, altered signalling pathways at intercalated discs, disruption of ion channels and gap junctions, and aberrant protein trafficking.**

*Austin KM et al. 2019. (PMID: 31028357)*

**The ARVC variant database can be found at https://molgenis136.gcc.rug.nl/**

In 2019, Ye et al reevaluated ARVC variants using large population databases. "...more than 10% of variants previously
reported to cause ARVC were found unlikely to be associated with highly penetrant monogenic forms of ARVC."
There were some variants that were found in population databases but were nevertheless associated with serious cardiac phenotypes suggesting they could be disease-modifiers of ARVC. Updated classification of variants by gene is available in the supplementary data

*Ye JZ et al 2019 PMID: 31402444*

"This study assessed the origin of ARVC-associated P/LP desmosomal variants and has 2 main findings. First, most P/LP desmosomal variants in ARVC are nonunique (75.3%), that is, occurring in multiple families and inherited (98.6%). Second, most nonunique PKP2 variants share haplotypes. Taken together **these results suggest most ARVC-associated variants originate from common founders.**

The high percentage of inherited variants will inform genetic counseling and approaches to cascade screening. Specifically, anticipatory guidance given during pretest counseling should include the likelihood that a variant, if identified, has very likely been inherited. Thus, signs or symptoms in a relative should be considered with a high index of suspicion.

it is also worth emphasizing that **13 (4.0%) of the P/LP variants in this study were large deletions.** While these large deletions in ARVC have been previously reported16,17,35 our findings highlight the importance of using a genetic test capable of identifying deletions."

*van Lint FHM et al 2019 PMID: 31386562*

We analyzed the genes in (a) 57 patients who fulfilled the ARVD/C TFC (TFC+), (b) 28 patients with probable ARVD/C (1 major and 1 minor, or 3 minor criteria), and (c) 31 patients with 2 minor or 1 major criteria. In the TFC+ ARVD/C group, 23 patients (40%) had PKP2 mutations, 4 (7%) had DSG2 mutations, and 1 patient (2%) carried a mutation in DSC2, whereas **1 patient (2%) had a mutation in both DSG2 and DSC2.** Among the DSG2 and DSC2 mutation-positive TFC+ ARVD/C probands, 2 carried compound heterozygous mutations and 1 had digenic mutations.

*Bhuiyan et al 2009 PMID: 20031616*

In 2006, two heterozygous *DSC2* mutations---a deletion and an
insertion---were detected in four probands and three family members. The deletion of a single nucleotide (1430delC) was detected in exon 10 of *DSC2.* It leads to a frameshift and a premature termination codon at position 480 (M477fsX480). The authors speculate that this results in haploinsufficiency, however functional studies to support this association are lacking. The other mutation was an insertion of two bases in exon 17 (2687_2688insGA) which is predicted to generate a
termination codon 4 aa residues downstream (E896fsX900).

*Syrris et al., 2006 (PMID: 17033975)*

Beffagna et al Identified two heterozygous mutations (c.304G\>A
(p.E102K) and c.1034T\>C (p.I345T)) in two probands and in four family members. The two missense mutations in the N-terminal domain affect the normal localisation of DSC2.

*Beffagna et al. 2007.(PMID:17963498)*

A heterozygous splice-acceptor--site mutation was identified in intron 5 (c.631-2A→G) of the *DSC2* gene, which led to the use of a cryptic splice-acceptor site and the creation of a downstream premature termination codon. The truncated protein encoded by the mutant allele in this case may have a dominant negative effect disrupting cell-cell adhesion, but the markedly decreased levels of mRNA observed with the mutant *DSC2* allele suggest that nonsense-mediated mRNA decay and consequent haploinsufficiency likely contribute to the disease mechanism.

*Heuser A et al. 2006 (PMID: 17186466)*

E102K and I345T mutant DSC2 proteins delocalize from the plasma membrane to the cytosol [\[90\]] inferring trafficking defects, while the R203C and T275M mutants show impaired proteolytic processing [\[64\]]. A pair of NMD mutations, R375X and Q554S, are found in the EC3 and EC4 domains, respectively, as well as deletions due to frameshift mutations [\[54\]].
The EC5 domain contains a single pathogenic mutation,
I603T [\[91\]], as well as frameshifts and deletions that would cause NMD. 

*Al-jassar C et al. 2013. PMID: (23911551)*

**Homozygous DSC2 mutations have been associated with ACM forms with or without cutaneous manifestations.**

A homozygous missense mutation p.D179G was associated with biventricular ACM and no cutaneous features. The mutation involves an amino acid in the first extracellular protein domain. Incorporation of the mutant protein in intercalated discs may affect the normal cadherin interactions in the extracellular space, leading to adhesion impairment between cells.

*Lorenzon A et al. 2015. (PMID: 26310507)*

Homozygosity of a truncation mutation (p.Q554X) in the desmosomal
cadherin DSC2 causes an early onset arrhythmic cardiomyopathy in the North American Hutterite population. Immunohistochemistry of diseased myocardium and recombinant expression in cell culture showed that the extracellular truncated protein remains stable, partially processed and localized at the intercalated discs, suggesting a more complex mechanism with partial contribution in maintaining cell junction integrity rather than the expected complete loss of function attributable to non-functional alleles.

*Gerull B et al. 2013. (PMID: 23863954)*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

Autosomal recessive

Optional modifiers: incomplete penetrance; digenic

**Allelic requirement:**

Monoallelic_aut

Biallelic_aut

Optional modifiers: digenic

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism likely loss of function of *DSC2* due to **reduction/absence of gene product or altered gene product structure** due to a variety of mechanisms (e.g. null alleles, trafficking defects, impaired proteolytic processing, absence of or impaired protein-protein interactions). A number of DSC2 mutations have been reported in the literature, the majority of which are missense mutations. Although loss of function is a likely mechanism, there is remaining uncertainty about haploisufficiency as a mechanism given variable inheritance patterns, mutational spectrum and lack of reports of whole gene deletions.(ClinGen) Autosomal dominant inheritance with incomplete penetrance is the most common mode of transmission, although autosomal recessive mutations have been described causing ARVC with and without cutaneous features. Instances of digenic inheritance have been identified with DSC2 variants along with other desmosomal gene mutations.

**Additional information related to ACMG evidence types**

As per Cardioclassifier:

**PVS1** - null variant in a gene where Loss of Function (LoF) is a known mechanism of disease
Consider activating for DSG2 and ARVC as there is a significant burden of truncating variants in cases against controls from analysis of 7,855 cardiomyopathy cases and 60,706 controls 

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework). Adjusted for ICC from original ACMG guidelines
0.1% (het) 
3.16% (hom) 

**For BS1 and PM2:**

Estimated Prevalence: 1/1000
Max Allelic Contribution: 0.092	
Maximum population AF: 0.000092

*Whiffin N et al 2018 PMID: 29369293*

**List variant classes in this gene proven to cause this disease**

- Splice region variant
- ?cryptic splice site variant
- Spice acceptor variant
- Splice acceptor variant predicted to undergo NMD
- Splice donor variant
- Splice donor variant predicted to undergo NMD
- Frameshift variant
- Frameshift variant predicted to undergo NMD
- Stop gained
- Stop gained predicted to undergo NMD
- Missense
- In frame insertion
- In frame deletion

**List other variant classes predicted to lead to the same functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift variant predicted to escape NMD
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
- ?larger deletions?
