# **DSG2 -- Arrhythmogenic Right Ventricular Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:3049

The relationship between DSG2 and arrhythmogenic right ventricular
cardiomyopathy (autosomal dominant) was evaluated using the ClinGen Clinical Validity Framework as of July, 2018. Variants in DSG2 were first reported in humans with this disease as early as 2006 (Pilchou et al., PMID: 16505173). Variation in DSG2 is a well-known cause of ARVC and accounts for 5%-26% of cases (McNAlly et al., 2005; PMID: 20301310). Since this gene-disease relationship is well-known, there is a significant amount of case-level data, segregation data and experimental data available in the literature, therefore the maximum score for both genetic evidence and experimental evidence has been reached. Note, this
curation effort may not be exhaustive of all literature related to this gene-disease relationship. This gene-disease relationship is supported by animal models, in vitro assays, expression assays, and protein interactions. In summary, DSG2 is definitively associated with autosomal dominant ARVC. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen Arrythmogenic Right Ventricular Cardiomyopathy Gene Curation Expert Panel on September
14, 2018 (SOP Version 6).

**ClinGen Haploinsufficiency phenotype comments:**

PMID 21397041 - Lahtinen et al. (2011) identified a proband with ARVC who had a novel 3059_3062delAGAG frameshift mutation leading to abolishing 99 carboxy-terminal amino acids of desmoglein-2. This mutation was found in four additional family members: two of whom met minor criteria for first-degree family members, one had minor repolarization abnormalities, and the fourth was unaffected. A screen for this mutation in the control samples revealed one carrier who then was found to have a conduction defect. Since the immunoreactive signal for desmoglein-2 was reduced by over 50%, a dominant-negative effect for the mutation was suggested as a possibility.

PMID 26296472 - Zhang et al. (2015) obtained DNA from post-mortem heart tissues, 25 patients had ARVC as a cause of death while the other 25 had sudden unexplained death. Two novel mutations in DSG2 were identified in each group. A frameshift insertion c.3075_3076insC (S1026Q fsX12) and a missense c.2032G>A (G678R) mutation was found in the ARVC group. In the samples from patients with sudden unexplained death, the missense mutations were c.2686G>A (E896K) and c.2573C>T (A858V). None of these variants was found in 96 control samples. No commentary as to possible mutation mechanism was included.

PMID 16773573 - Awad et al (2006) identified four patients with ARVD/C with mutations in DSG2. One proband had two mutations in trans, a missense c.143G>A (R48H) on his paternal allele and a c.915G>A change resulting in a premature termination codon, W305X, in exon 8 of his maternal allele. Of note, his 77-year-old mother was found to carry the same nonsense mutation, but was unaffected based upon extensive clinical assessments. His 24 year-old sister also carries the W305 mutation and is believed to be unaffected. His father who presumably passed on the R48H mutation died due to non-cardiac issues at age 74 and was never tested. The remaining three probands in this study had a single, heterozygous missense mutation each: c.134G>A (R45Q), c.1517G>A (C506Y), and c.2431G>T (G811C). Of note, the mother of the proband with R45Q and the mother of the proband with G811C were both found to be carriers of their respective familial mutation and were determined to be unaffected after extensive cardiac assessments. **Discussion regarding mutational mechanism included the possibility of both haploinsufficiency and dominant-negative as possible mechanisms.**

Desmoglein-2 (DSG2) is a member of the desmoglein family and is expressed in myocardium. DSG2 is an essential component of the desmosome so mutations of this gene disrupt the proper organization of desmosomal junctions. Mutations of DSG2 and other desmosomal genes have been demonstrated to be associated with arrhythmogenic right ventricular dilatation/cardiomyopathy (ARVD/C), but the phenotype is widely variable ranging from individuals who meet strict diagnostic criteria to those who are asymptomatic and identified in control populations. Some affected patients have compound heterozygous mutations. So variants in DSG2 might be susceptibility variants, rather than pathogenic variants. Also, digenic, homozygous, and compound heterozygous inheritance has been suggested.

**While a few protein truncating mutations have been described, the mechanism by which these mutations cause disease has not been clearly determined and no patients with complete gene deletions and ARVD/C have been reported.**

**Literature review:**

"More than 20 pathogenic variants have been described (see Table A, Locus Specific) In addition, **multiple instances of digenic inheritance have been identified with other desmosomal gene variants [Rigato et al 2013, Groeneweg et al 2015]. Biallelic inheritance has also been described in ARVC with compound heterozygous variants** [Awad et al 2006, Pilichou et al 2006, Bhuiyan et al 2009].

Much of the underlying pathogenesis of DSG2 pathogenic variants is still unknown; it is believed that loss of DSG2 compromises cell-to-cell adhesion between cardiomyocytes [Kant et al 2015]"

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK1131/*

"The observed autosomal dominant inheritance of desmoglein 2 gene
(*DSG2*)--related AC cases suggests that reduction of the Dsg2 wild-type (WT) protein leads to a loss of cardiac function or that Dsg2 mutant (MT) protein exerts a dominant negative effect on the remaining Dsg2 WT protein. Two major types of nonexclusive pathomechanisms have been described to depend on Dsg2. As a cell--cell adhesion molecule, it provides adhesive force and thereby supports force transmission between contractile cardiomyocytes. Alternatively, Dsg2 regulates intracellular signaling. This function is expected to be linked to the cytoplasmic domain of Dsg2 and may involve associated signaling molecules, such as Pg and Pkp2/PKCα acting through downstream effectors implicating the Wnt or Hippo pathways."

*Kant S et al. 2015 (PMID:26085008)*

"In a series of 80 unrelated ARVC probands, 26 carried a mutation in DSP (16%), PKP2 (14%), and transforming growth factor-β3 (2.5%) genes; the remaining 54 were screened for DSG2 mutations by denaturing high-performance liquid chromatography and direct sequencing. **Nine heterozygous DSG2 mutations (5 missense, 2 insertion-deletions, 1 nonsense, and 1 splice site mutation) were detected in 8 probands (10%).**
**In 1 patient (patient 5), 2 different mutations (E331K and 1881 to 2A→G) were detected.** Family members were available for molecular analysis (Figure 3, family 172). Subject II,1 carried both mutations, subjects I,2 and III,2 carried the missense mutation, and subject III,1 carried the splice site mutation, thus demonstrating that the 2 mutations are in trans orientation. Although both mutations are potentially pathogenic, the possibility that the missense mutation is a rare polymorphism cannot be ruled out.

*Pilichou K et al 2006 PMID: 16505173*

**"Among the 27 *DSG2* mutation carriers, 18 (67%) carried a missense mutation, one (4%) a stop mutation, three (11%) a frameshift mutation, and five (18%) a splice site mutation. Most patients (17/18, 94%) with *DSG2* missense mutations carried a mutation located within the *DSG2* pro‐peptide cleavage site, a hot‐spot for *DSG2* mutations *(Awad M. 2008 (PMID: 18382419)***

Among the DSG2 mutation carriers, one patient was compound heterozygous (DSG2 p.Gly640AspfsX15+DSG2 p.Val295SerfsX6) and five were homozygous [p.Thr335Ala, p.Lys834ArgfsX3 (n=2), c.523+1G >A and p.Gly720Ter].

**DSG2 mutation carriers more often displayed biventricular involvement and left ventricular dysfunction at diagnosis...**

...Although DSG2 mutation carriers more frequently display more severe right and left ventricular involvement than PKP2 mutation carriers, their risk for VA is similar.

Although the main aim of our study was not to assess the role
of multiple mutations on prognosis, **unadjusted survival analysis
showed that a complex genetic status was associated with a
higher risk of HF-related death/transplantation.** This finding is
in accordance with previous studies in which multiple mutations
were associated with a worse prognosis, especially a higher risk
of arrhythmic events and SCD.17,27 In addition to the higher
arrhythmic risk which was previously reported, our results suggest
that multiple/homozygous mutations carriers are at higher risk of
developing end-stage HF.

The relatively small number of DSG2 patients followed and the
dominance of Caucasian subjects in our population could hinder
the generalization of these results to other populations."

*Hermida A et al. 2019  PMID: 30790397*

There is significant excess of truncating and non truncating variants in DSG2 in ARVC cases vs population controls.

*Walsh et al, 2016 (PMID 27532257)*

**"the amino-terminal propeptide and cadherin repeats I-III domains (residues 24 through 388) of DSG2 demonstrated an over-representation of missense mutations compared to controls** (16/21 vs. 2/20, P < 2.5×10−5, Figure 6B). These mutation hot spots localize to key regions of the desmosomal macromolecular complex as demonstrated in Figure 7." 

*Kapplinger JD et al 2011 PMID: 21636032*

**The ARVC variant database can be found at https://molgenis136.gcc.rug.nl/**

In 2019, Ye et al reevaluated ARVC variants using large population databases. "...more than 10% of variants previously
reported to cause ARVC were found unlikely to be associated with highly penetrant monogenic forms of ARVC."
There were some variants that were found in population databases but were nevertheless associated with serious cardiac phenotypes suggesting they could be disease-modifiers of ARVC. Updated classification of variants by gene is available in the supplementary data

*Ye JZ et al 2019 PMID: 31402444*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

Autosomal recessive

Optional modifiers:

Incomplete penetrance; Digenic (PKP2, DSP)

**Allelic requirement:**

Monoallelic_aut

Biallelic_aut

Optional modifiers:

Digenic (PKP2, DSP)

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism likely due to **reduction/absence of gene product or altered gene product structure** due to a variety of mechanisms.  Much of the underlying pathogenesis of DSG2 pathogenic variants is still unknown; it is believed that loss of DSG2 compromises cell-to-cell adhesion between cardiomyocytes. Both haploinsufficiency and dominant-negative have been proposed as possible mechanisms. Instances of digenic inheritance have been identified with other desmosomal gene variants and biallelic inheritance has also been described with compound heterozygous and homozygous variants.
Hermida et al found DSG2 mutation carriers display more severe disease than PKP2 mutation carriers, with an increased risk of biventricular involvement and evolution to end-stage HF. Also a complex genetic status had a worse prognosis with a higher risk of end stage HF and arrhythmic events. Most patients (17/18, 94%) with *DSG2* missense mutations carried a mutation located within the *DSG2* pro‐peptide cleavage site.

**Additional information related to ACMG evidence types**

As per Cardioclassifier:

**PVS1** - null variant in a gene where Loss of Function (LoF) is a known mechanism of disease
Consider activating for DSG2 and ARVC as there is a significant burden of truncating variants in cases against controls from analysis of 7,855 cardiomyopathy cases and 60,706 controls 

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework). Adjusted for ICC from original ACMG guidelines
0.1% (het) 
3.16% (hom) 

**For BS1 and PM2:**

Estimated Prevalence: 1/1000
Max Allelic Contribution: 0.092	
Maximum population AF: 0.000092

*Whiffin N et al 2018 PMID: 29369293*

**List variant classes in this gene proven to cause this disease**

- Splice acceptor variant
- Splice donor variant
- Frameshift variant
- Stop gained
- Stop gained variant predicted to undergo NMD
- Stop gained variant predicted to escape NMD
- Missense
- In frame insertion
- In frame deletion

**Potential novel variant classes based on predicted functional
consequence:**

- Splice acceptor variant predicted to undergo NMD
- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to undergo NMD
- Splice donor variant predicted to escape NMD
- Frameshift variant predicted to undergo NMD
- Frameshift variant predicted to escape NMD
- start_lost
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
