# **DSP -- Dilated Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:3052

DSP was evaluated for autosomal dominant dilated cardiomyopathy (DCM). Human genetic evidence supporting this gene-disease relationship includes case-level data, segregation data, and case-control data. There are multiple families with DCM with truncating variants in DSP segregating in the family in the literature. Only case-level and segregation data was scored for DCM and there was no evidence suggesting the probands and/or family members met ARVC TFC. Case-control evidence shows that DSP variants are over-represented in cases versus controls (Mazzarotto et al, 2020, PMID 31983221). Additional human genetic evidence is available in the literature that was not included in the curation scoring as the maximum score for genetic evidence was reached. In addition, this gene-disease association is supported by experimental evidence, including expression data and animal models. DSP has been shown to be expressed in the heart, and also one of the intercalated disc components highly related to LV function, providing evidence of protein interaction (Kazerounian et al, 2002, PMID: 12366696; Ortega et al, 2017, PMID: 28934278). DSP KO mice exhibit a biventricular cardiomyopathy, although it is particularly arrhythmic with sudden death (Lyon et al, 2014, PMID: 24108106). In summary, there is strong evidence to support the relationship between DSP and autosomal dominant DCM. Of note, DSP has also been curated by the ARVC Gene Curation Expert Panel for arrhythmogenic cardiomyopathy with wooly hair and keratoderma (Strong, July 12, 2019), as there are also several families in which individuals meet arrhythmogenic right ventricular cardiomyopathy (ARVC) task force criteria (TFC). Experts in the DCM GCEP encourage clinicians to carefully evaluate for a possibility of an ACM phenotype and treat patients and families with DSP variants accordingly. This classification was approved by the ClinGen Dilated Cardiomyopathy Working Group on June 12, 2020.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**Literature review:**

Protein-truncating variants in constitutive exons (Percent Spliced In>90%) of TTN and in DSP were significantly enriched in patients with DCM compared with healthy controls (TTN 11.3% DCM vs 0.4% healthy controls, P=6.2×10−27; DSP 1.4% vs 0.0%, P=4.2×10−3). 

*Mazzarotto F et al 2020 PMID: 31983221*

**From in house atlas of variants in DSP**

12/32 truncating variants

Nonsense

Frameshift

Splice donor variant

20/32 non-truncating variants (mainly VUS)

Missense (all VUS)

in frame insertion (likely pathogenic) 

https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=DSP&icc=DCM

*Walsh et al, 2016 (PMID 27532257)*
*Pugh TJ et al, 2014 (PMID: 24503780)*

Mutations in the DSP gene encoding desmoplakin were first identified in an autosomal recessive form of arrhythmogenic cardiomyopathy. The present study makes a case for DSP cardiomyopathy being a distinct form of cardiomyopathy. DSP cardiomyopathy results in episodic inflammation, which precedes the development of fibrosis. Frequent PVCs occurring before LV systolic dysfunction or LV enlargement is also a key distinction from most cases of DCM. In DSP patients, late gadolinium enhancement was found in the LV subepicardium rather than mid-myocardial as is often seen with nonischemic cardiomyopathy. The correct diagnosis must include the identification of the DSP mutation. This paper is likely to be very impactful, as it suggests that some of the patients suspected of myocarditis and cardiac sarcoidosis may actually have DSP cardiomyopathy.

Mutations in this study were primarily truncating mutations (ie, frameshift, nonsense, or splice site
mutations). Truncating mutations in this and other
studies occur throughout the gene without evident
clustering. There was no clear correlation between
truncating mutation location and clinical presentation. These findings indicate a similar loss of function consequence of DSP truncating mutations through nonsense mediated RNA decay of mutant
transcripts, rather than effects from truncated proteins,
consistent with reduced DSP protein levels in skin and
myocardium from patients with DSP truncating mutations.18,30 

The specific mutation location may contribute
to the disease phenotype in the case of missense mutations (eg, mutations in the desmin versus plakophilin/
plakoglobin binding domains), but this study did not include enough missense mutation carriers to analyze this
relationship. Supporting this concept, a previous case
series of 10 patients with missense mutations versus
17 patients with truncating mutations found that the
former were less likely to have LV dysfunction.28 Also
suggesting the possibility of locus-specific effects for
missense variants is the evidence of mutation clustering in the plakophilin/plakoglobin-binding and desmin binding protein domains..

*Smith ED et al 2020 PMID: 32372669*


#### **Pilot application of harmonised terms**

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Allelic requirement:**

Monoallelic_aut

Optional modifiers:

**Inheritance:**

Autosomal dominant

Optional modifiers:

Incomplete penetrance

**Narrative summary of molecular mechanisms:**

The disease mechanism is not clear and haploinsufficiency, dominant negative or both have all been proposed as mechanisms leading to **reduction/absence of gene product or altered gene product structure** in DSP. In DCM patients, protein truncating variants (including nonsense, frameshift, essential splice) in DSP are enriched when compared to controls. Non-truncating variants were not enriched. Truncating mutations appear to occur throughout the gene whereas missense mutations are more likely to cluster in either the plakophilin/plakoglobin-binding or desmin binding protein domains. Smith E et al have suggested that DSP mutations are associated with a distinct type of cardiomyopathy with a high prevalence of LV inflammation, fibrosis, and systolic dysfunction, and DSP cardiomyopathy should be considered in the differential diagnosis for myocarditis and sarcoidosis. These authors suggest a loss of function consequence of DSP truncating mutations through nonsense mediated RNA decay of mutant transcripts, rather than effects from truncated proteins, consistent with reduced DSP protein levels in skin and myocardium from patients with DSP truncating mutations.

**List variant classes in this gene proven to cause this disease**

- Splice donor variant
- Splice donor variant predicted to undergo NMD
- Frameshift variant
- Frameshift variant predicted to undergo NMD
- Stop gained
- Stop gained predicted to undergo NMD
- Missense

**Potential novel variant classes based on predicted functional
consequence:**

- Start lost
- In frame insertion
- In frame deletion
- Spice acceptor variant
- Spice acceptor variant predicted to undergo NMD
- Spice acceptor variant predicted to escape NMD
- Frameshift variant predicted to escape NMD
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
