**[FBN1 - Marfan Syndrome]{.ul}**

**Review of source material:**

**ClinGen:**

https://search.clinicalgenome.org/kb/genes/HGNC:3603

"The first report indicating a relationship between the FBN1 gene and autosomal dominant Marfan syndrome was reported by Dietz et al in 1991 (PMID: 1852208). 

The nomenclature for Marfan syndrome is ascribed to Antoine Marfan, who described a individual with part of the phenotypic features associated with the greater syndrome in 1896 (reviewed in Gott 1998, PMID:9798380). Marfan syndrome a multisystemic disorder affecting mainly the connective tissue. Marfan comprises a broad phenotypic spectrum and severity and thus represents a continuum of disease (reviewed in Dietz 1991, Marfan GeneReviews, PMID: 20301510). Cardiovascular, ocular, and skeletal phenotypes represent the most common phenotypic manifestations among individuals diagnosed with Marfan syndrome. The cardiovascular
phenotypes, including dilatation of the aorta, are the areas that
represent the major morbidity and mortality of Marfan syndrome. 

**Over 1800 variants have been identified in FBN1 (Collod-Beroud et al.,2003 PMID: 12938084), including missense, nonsense, frameshift, splice site, small deletions, and large deletions, according to two databases that house variant information: (1) The UMD-FBN1 database (http://www.umd.be/FBN1/); and (2) the LOVD FBN1 database (https://databases.lovd.nl/shared/genes/FBN1).**

The mechanism for the gene-disease relationship is **protein loss of function,** as mutation in the FBN1 protein, fibrillin, results in the inability of the protein to be excreted from cells to help in the formation and stabilization of connective tissue (Sakai et al., 1986, PMID: 3536967). 

Evidence supporting this gene-disease relationship includes case-level data, segregation data, functional data, and model organisms. This gene-disease relationship has been studied for more than 20 years, therefore a significant amount of case-level data, segregation data, and experimental data is available and the maximum score for genetic evidence (12 points) and experimental evidence (6 points) has been reached. Note, this curation effort may not be exhaustive of all literature related to this gene-disease relationship. In particular, earlier compelling evidence suggestive of the gene-disease relationship, such as linkage data, may not be reflected in the current curation. 

In summary, FBN1 is definitively associated with autosomal dominant Marfan syndrome. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was reviewed by the FBN1 VCEP with final approval by the ClinGen General GCEP on March 4, 2019 (SOP Version 6)."

ClinGen Haploinsufficiency Comments:

**In summary, Loss of function variants (exonic deletions and whole gene deletions) of the FBN1 gene have been described in individuals with Marfan phenotype.** Additional PMIDs: 21936929, 30479897, 17492313, 28842177

**[Literature review:]{.ul}**

"When information about transmission type (available for 398 mutations) is examined, there are a surprising number of de novo mutations as compared to transmitted mutations (188 de novo vs. 210 familial).

The global molecular analysis of the FBN1 mutations reveals two classes of mutations. The first, which represents more than one-third of the mutations (38.6%), corresponds to mutations predicted to result in shortened fibrillin-1 molecules; 61 nonsense mutations, 71 splicing errors, 23 insertions and duplications, 51 deletions, and 10 inframe deletions are in this class. They act as dominant negative but display a highly variable clinical phenotype, the severity of which is directly related to the quantitative expression of the mutant allele and to the
percentage of truncated or shortened proteins incorporated in the
microfibrils. The second class represents less than two-thirds (60.3%) of the mutations and corresponds to missense mutations, most among them are located in cbEGF-like modules (78%). They can be subclassified into:
1) mutations creating or substituting cysteine residues potentially implicated in disulfide bonding and consequently in the correct folding of the monomer; and 2) amino acids implicated in calcium binding and subsequently in interdomain linkage, rigidification of monomer, and in protease susceptibility; see www.hgvs.org/mutamen for updated recommendations.)"

*Collod-Beroud G et al 2003 PMID: 12938084*

**"Most of the mutations that cause Marfan syndrome change a single amino acid in the fibrillin-1 protein. The remaining FBN1 gene mutations result in an abnormal fibrillin-1 protein that cannot function properly.
FBN1 gene mutations that cause Marfan syndrome reduce the amount of fibrillin-1 produced by the cell, alter the structure or stability of fibrillin-1, or impair the transport of fibrillin-1 out of the cell.**

These mutations lead to a severe reduction in the amount of fibrillin-1 available to form microfibrils. Without enough microfibrils, excess TGF-β growth factors are activated and elasticity in many tissues is decreased, leading to overgrowth and instability of tissues and the signs and symptoms of Marfan syndrome.)"

*Genetics Home reference:
[https://ghr.nlm.nih.gov/gene/FBN1\#conditions]{.ul} *

"\...These data are consistent with a model that invokes
haploinsufficiency for WT fibrillin-1, rather than production of mutant
protein, as the primary determinant of failed microfibrillar assembly.)"

*Judge DP et al. 2004 PMID:15254584*

"In case of FBN1, **about 1,850 different mutations
that are widely spread all over the protein have been described so
far (http://www.umd.be/fbn1/), of which the majority is unique
to each family. The mutational spectrum encompasses nonsense,
frameshift, splice altering, insertion/deletion, and missense variations as well as whole-gene or multiexon deletions.** 

**It has been a longstanding matter of debate, however, whether these mutations cause marfanoid syndromes through haploinsufficiency, a dominant-negative mechanism or a combination of both** (Dietz et al., 1993; Judge et al., 2004; Matyas et al., 2007). Lately, particular the latter hypothesis is gaining momentum in the MFS field (Colovati et al., 2012; Franken et al., 2015). 
Despite significant progress in the understanding of the molecular defects underlying MFS, a limited number of convincing genotype–phenotype correlations has emerged. 

**Cysteine-destroying or cysteine-creating mutations
are commonly associated with ectopia lentis (Schrijver et al., 1999). Haploinsufficient mutations, on the other hand, more frequently correlate with aortic events at young ages as well as pectus carinatum, dural ectasia, and skin striae (Baudhuin et al., 2015; Franken et al., 2015). Additionally, the vast majority of the mutations in exons 24–32 are linked to either neonatal MFS or other severe marfanoid presentations (Tiecke et al., 2001).** 

Although MFS mostly inherits in an autosomal-dominant manner, rare cases with recessive homozygous or compound heterozygous FBN1 mutations have been described (Hogue et al., 2013). They exhibit a more severe phenotype compared with their heterozygous relatives, who are usually asymptomatic or only mildly affected. Taken together, the nature of the implicated FBN1 defect can, at least to some extent, explain phenotypic variability. Yet, also related individuals carrying an identical FBN1 mutation vary widely with respect to onset age, organ-system involvement, and disease severity."

*Verstraeten A et al. 2016 Jun (PMID:26919284)*

**[Pilot application of harmonised terms:]{.ul}**

**Disease associated variant consequences:**

Dose change: dose reduction: Decreased gene product level

Dose change: dose reduction: Absent gene product

Altered gene product structure

**Allelic requirement:**

Monoallelic_aut

**Inheritance:**

Autosomal dominant

Optional modifiers: 

**Narrative summary of molecular mechanisms:**

The mutational spectrum encompasses nonsense, frameshift, splice altering, insertion/deletion, and missense variations as well as whole-gene or multiexon deletions. Most of the mutations that cause Marfan syndrome (MFS) are missense mutations. The mechanism for the gene-disease relationship is loss of function **primarily due to altered gene product structure** which ultimately leads to a reduction in the amount of protein at it's target location due to increased degradation, impaired trafficking etc but **also due to variants that reduce the gene product level**. There has been considerable debate about whether Marfan syndrome is caused through haploinsufficiency, a dominant-negative mechanism or a combination of both.
MFS is mostly inherited in an autosomal-dominant manner, however rare cases with recessive homozygous or compound heterozygous FBN1 mutations have been described (Hogue et al., 2013). They exhibit a more severe phenotype compared with their heterozygous relatives.
According to a review by Verstraeten A et al in 2016, "a limited number of convincing genotype–phenotype correlations has emerged. 
**Cysteine-destroying or cysteine-creating mutations
are commonly associated with ectopia lentis (Schrijver et al., 1999). Haploinsufficient mutations, on the other hand, more frequently correlate with aortic events at young ages as well as pectus carinatum, dural ectasia, and skin striae (Baudhuin et al., 2015; Franken et al., 2015). Additionally, the vast majority of the mutations in exons 24–32 are linked to either neonatal MFS or other severe marfanoid presentations (Tiecke et al., 2001).** "
 
**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
>1%
estimated prevalence between 1.5 to 17.2 per 100 000 individuals

**BS1** (MAF too high for disease)
an allele frequency above 0.0002 (1:5000) was considered as a strong criterion for benignity

case–control studies were only considered to use as a strong criterion **(PS4)** if >1000 controls were tested.

**PM1** located in a mutational hotspot and critical established functional domain Cys substitution outside cb-EGF domain Introduction of a new Cys within cb-EGF domain Cb-site substitution in cb-EGF 

**PS3** well-established in vitro or in vivo studies show damaging effect Cys substitution within cb-EGF domain

**PP4**
The combination of ARD or dissection and EL seems to be more specific for MFS,17and we, therefore, only applied the PP4 criterion when a patient presented these 2 features combined.

**PVS1**
Because loss of expression or haploinsufficiency is known as a disease-causing mechanism in the FBN1 gene, the PSV1 criterion was used for all frameshift and nonsense variants not affecting the final exon 65 and for all splice-site variants in positions ±1 to 2. Frameshift and nonsense variants affecting exon 65 usually produce a protein which is shorter2 and therefore we used for these cases the PM4 criterion.

More info including additional criteria
*Muino-Mosquera L et al 2018 PMID: 29875124*

**List variant classes in this gene proven to cause this disease**

Splice region variant

Spice acceptor variant

Splice donor variant

Start lost

Frameshift variant

Stop gained

Stop gained predicted to undergo NMD

Missense

In frame insertion

In frame deletion

**Potential novel variant classes based on predicted functional
consequence**

stop_gained predicted to escape NMD

Splice acceptor predicted to undergo NMD

Splice acceptor predicted to escape NMD

Splice donor predicted to undergo NMD

Splice donor predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
