# **KCNH2 -- Long QT Syndrome**

**Review of source material:**
*Adler A et al 2020 (PMID: 31983240)*

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:6251

**ClinGen Evidence for Haploinsufficiency:**
There is a significant amount of evidence suggesting that loss of KCNH2 results in Long QT syndrome (LQTS). In addition to the evidence above, Itoh et al 2016 (PMID: 26669661) report 280 KCNH2 variants in families with LQTS; 95/280 were nonsense, frameshift or splice site variants (>100 families with multiple affected individuals) (Supplementary Table 2). Stattin et al 2012 (PMID: 23098067 ) reports 3 additional affected probands with truncating variants. HGMD also catalogs a number of putative loss of function variants, including: 70 disease causing nonsense variants,17 disease causing splice site variants (+/-2), 6 large deletions (multiple exons), and 141 disease causing frame shift variants.

Of note, large deletions involving KCNH2 in addition to other genes have also been observed in individuals diagnosed with Long QT syndrome. Barc et al (2011) (PMID: 21185499) studied 93 patients with LQTS who had experienced symptoms such as syncope, arrhythmia, and cardiac arrest. Two probands were found with deletion in KCNH2. The proband of Family 1 was 23 years old with a 650 kb deletion of exons 4-14 (+19 other genes centromeric to KCNH2). Six other carrier family members had LQTS but were asymptomatic. The proband of Family 2 was 28 years old with a 145 kb deletion of the entire KCNH2 and ABP1 genes. The deletion was also found in her affected mother and "healthy" brother.

**Literature review:**

"Class 1 mutations disrupt the synthesis/translation of Kv11.1 α-subunits (decrease n); Class 2 mutations reduce the intracellular transport or trafficking of Kv11.1 proteins to the cell membrane (decrease n); Class 3 mutations disrupt Kv11.1 channel gating (decrease Po); and Class 4 mutations negatively affect K+ permeation (decrease i).

**About 40% of LQT2-linked KCNH2 mutations are nonsense mutations, frameshift mutations, insertions, deletions, duplications, or involve a splice site that inhibits Kv11.1 protein synthesis/translation by generating incomplete proteins or causing nonsense-mediated RNA decay (NMD) (class 1 mechanism)** [18], [19], [20]. By provoking NMD, class 1 mutations are expected to cause haploinsufficiency. **The remaining ~60% of LQT2 mutations are missense,** where a single nucleotide change alters an amino acid codon to a different amino acid to cause a LOF by disrupting channel trafficking to the cell membrane (class 2 mechanism), gating (class 3 mechanism), and/or single channel current (class 4 mechanism) [16], [20], [21], [22], [23], [24], [25], [26], [27]. Over 150 suspected LQT2-causing missense mutations have been studied using heterologous expression systems and these studies demonstrate that ~90% of LQT2-linked missense mutations disrupt Kv11.1 channel function via a class 2 mechanism (Fig. 3) [20], [21], [22], [23], [24], [25], [26], [27], [28]. Class 2 LQT2 mutations decrease the folding efficiency of Kv11.1 proteins and increase their retention in the endoplasmic reticulum (ER) by cellular quality control mechanisms."

*Smith J et al.2016 (PMID: 27761161)*

"KCNH2 spans approximately 19 kb. The longest isoform consists of 16 exons and produces a protein of 1,159 amino acids (NM_000238.3). Two shorter isoforms of KCNH2 exist.

More than 700 pathogenic variants have been reported, including pathogenic missense, nonsense, splice site, and frameshift variants as well as large multiexon deletions...

Long QT syndrome (LQTS) associated with biallelic pathogenic variants or heterozygosity for pathogenic variants in two different genes (i.e., digenic pathogenic variants) is generally associated with a more severe phenotype with longer QTc interval and a higher incidence of cardiac events [Schwartz et al 2003, Westenskow et al 2004, Tester et al 2005, Itoh et al 2010]."

*Gene reviews LQT
https://www.ncbi.nlm.nih.gov/books/NBK1129/*

Compendium of Cardiac channel mutations: 29 (10% of genotype positive patients) had 2 mutations in either KCNQ1, KCNH2, SCN5A, KCNE1, KCNE2 9 with multiple KCNQ1 variants, 7 with a KCNQ1 and a KCNH2 variant, 5 with a KCNQ1 and a SCN5A variant, 2 with two KCNH2 variants, 4 with a KCNH2 and a SCN5A variant, and 2 with two SCN5A variants.

*Tester D et al. 2005. PMID: 15840476*

"...an ex6-14del of the KCNH2 gene [was found] in a 22-year-old woman misdiagnosed with epilepsy since age 9 years (QTc 560 ms) and a sibling with sudden death at age 13 years; and (3) an ex9-14dup of the KCNH2 gene [was found] in a 12 year-old boy (QTc 550 ms) following sudden nocturnal death of his 32-year-old mother."

*Eddy C-A et al 2008 PMID: 18774102*

"The major findings of the present study from 858 type-2 LQTS subjects with genetically confirmed KCNH2 mutations derived from 4 LQTS Registries are that 1) **there is a significant mutation type-location interaction;** specifically that the relative risk between C-terminus and the regions is different for missense versus non-missense locations, 2) patients with missense mutations in the transmembrane pore region have significantly higher cardiac event rates than those with missense mutations in either N-terminus, transmembrane non-pore or C-terminus regions, 3) patients with non-missense mutations were at significantly higher risk than those with missense mutations in the C-terminus region, 4) patients with mutations located in putative α-helical domains have significantly higher cardiac event rates than in those with mutations in either the β-sheet domains or other uncategorized locations, and these higher event rates are independent of traditional clinical risk factors and of β-blocker therapy..." 

*Shimizu W et al 2009 PMID: 19926013*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 
Incomplete penetrance;
Digenic (other LQT genes)

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers 
Digenic (other LQT genes)

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is likely loss of function of KCNH2 due to **reduction/absence of gene product or altered gene product** due to a variety of mechanisms including disruption of synthesis of channel subunits, reduction in intracellular transport or trafficking, defects in ion permeation or channel gating. Both haploinsufficiency and a dominant negative effect are proposed mechanisms causing loss of function of KCNH2. About 60% of LQT2 mutations are missense variants, the remaining 40% are nonsense mutations, frameshift mutations, insertions, deletions, duplications, or involve a splice site. There has been a report of specific mutation type-location interaction. Patients with missense mutations in the transmembrane pore region have significantly higher cardiac event rates than those with missense mutations in either N-terminus, transmembrane non-pore or C-terminus regions. Digenic inheritance has been described with mutations in other LQT genes. Biallelic pathogenic variants or digenic pathogenic variants are generally associated with a more severe phenotype with longer QTc interval and a higher incidence of cardiac events. 

**List variant classes in this gene proven to cause this disease:**

- Spice acceptor variant
- Splice donor variant
- Frameshift variant
- Stop gained
- Stop gained predicted to undergo NMD
- Missense
- In frame insertion
- In frame deletion
- structural variants

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice acceptor variant predicted to undergo NMD
- Splice donor variant predicted to escape NMD
- Splice donor variant predicted to undergo NMD
- Frameshift predicted to escape NMD
- Frameshift variant predicted to undergo NMD
- Stop_gained predicted to escape NMD
- Stop_lost
- Start_lost
- gain of upstream Start [uORF]
- gain of upstream Start [oORF]
- Stop lost [oORF]
- Frameshift [oORF]
