# **LDLR -- Familial Hypercholesterolemia**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:6547

**ClinGen evidence for haploinsufficiency**
https://dosage.clinicalgenome.org/clingen_gene.cgi?sym=ldlr&subject

Five different classes of LDLR mutations have been identified, dependent on the effect on the phenotype. Class 1 mutations are null mutations that result in no detectable LDLR protein. In class 2 mutations, the transport of the LDLR from the endoplasmic reticulum to the Golgi apparatus is blocked completely (class 2a) or partially (class 2b). A class 3 mutation leads to expression of a non-functional LDLR. Class 4 mutations result in LDL binding but the LDLR-LDL complexes cannot be internalized, and in class 5 mutations, recycling of the LDLR is not efficient and therefore do not reach the cell surface (PMID: 26482752). LDLR is subject to Alu-mediated partial gene duplications. Multiple (~66) gross insertions and duplications (some publications: PMID: 22698793 (ex. 2-15), PMID: 23669246 (ex. 2-8), PMID: 20145306 (ex. 3-12), PMID: 11313767 (ex. 8-10), PMID: 23415438 (ex. 7-12), and PMID: 25461735 (prom.-ex. 6). These mutations lead to dysfunction of the LDLR protein

**Literature review:**

"FH is caused by a mutation in the gene encoding the LDLR in more than 90 % of the molecular diagnosed cases, and this mutation leads to absent or dysfunctional LDLR at the surface of the hepatocytes [7]. As a consequence, hepatic uptake of LDL-C is decreased which results in elevated plasma levels of LDL-C [1]. The LDLR gene is located on the short arm of chromosome 19, and to date, over 1700 mutations in the LDLR gene have been described (http://www.ucl.ac.uk/ldlr/Current/). Five different classes of LDLR mutations have been identified, dependent on the effect on the phenotype. Class 1 mutations are null mutations that result in no detectable LDLR protein. In class 2 mutations, the transport of the LDLR from the endoplasmic reticulum to the Golgi apparatus is blocked completely (class 2a) or partially (class 2b). A class 3 mutation leads to expression of a non-functional LDLR. Class 4 mutations result in LDL binding but the LDLR-LDL complexes cannot be internalized, and in class 5 mutations, recycling of the LDLR is not efficient and therefore do not reach the cell surface..."

*Hartgers M et al 2015 PMID: 26482752*

"The updated database (http://www.lovd.nl/LDLR) now includes 2925 curated variants, representing 1707 independent events. All 129 nonsense variants, 337 small frame-shifting and 117/118 large rearrangements were classified as 4 or 5. Of the 795 missense variants, 115 were in classes 1 and 2, 605 in class 4 and 75 in class 3. 111/181 intronic variants, 4/34 synonymous variants and 14/37 promoter variants were assigned to classes 4 or 5. Overall, 112 (7%) of reported variants were class 3."

*Leigh S et al 2017 PMID: 27821657*

"Pathogenic variants have been reported in the promoter, introns, and exons of LDLR. The majority of pathogenic variants fall within the ligand-binding (40%) or epidermal growth factor precursor-like (47%) domains, with the highest frequency of pathogenic variants reported in exon 4 (20%) [Leigh et al 2008, Usifo et al 2012]. More than 1,500 LDLR pathogenic variants have been reported in the University College London (UCL) database, highlighting the molecular heterogeneity of the disorder. See Table A, Locus-Specific Databases and ClinVar, for a list of reported variants.

Pathogenic variants in LDLR usually either reduce the number of LDL receptors produced within the cells or disrupt the ability of the receptor to bind LDL-C. Either way, heterozygous pathogenic variants in LDLR cause high levels of plasma LDL-C.

Complete loss-of-function variants in LDLR generally lead to more severe disease due to higher LDL-C levels [Khera et al 2016]. Partial loss-of-function variants in LDLR result in less severe disease due to lower LDL-C levels.

Recent findings suggest that only 73% of individuals with a heterozygous LDLR pathogenic variant have an LDL level >130 mg/dL, suggesting lower penetrance than previously proposed [Khera et al 2016]."

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK174884 #hyperchol.Molecular_Genetics*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

FH is caused by a mutation in the gene encoding the LDLR in more than 90 % of the molecular diagnosed cases, and this mutation leads to absent or dysfunctional LDLR at the surface of the hepatocytes. As a consequence, hepatic uptake of LDL-C is decreased which results in elevated plasma levels of LDL-C.
The mechanism appears to be loss of function of LDLR resulting in **decreased/absent or altered gene product** which leads (through various mechanisms including null mutations, defective transport, defective receptor binding, inability to internalise LDLR-LDL complexes, inefficient LDLR recycling) to increased LDL-C particles accumulating in the blood. LDLR is genetically heterogeneous with more than 1500 pathogenic variants reported. Pathogenic variants have been reported in the promoter, introns, and exons of LDLR. The majority of pathogenic variants fall within the ligand-binding (40%) or epidermal growth factor precursor-like (47%) domains, with the highest frequency of pathogenic variants reported in exon 4 (20%)  Penetrance is incomplete.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Splice region variant
- Missense
- In frame deletion
- In frame duplication
- synonymous
- intron_variant
- regulatory_region_variant

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- 5_prime_UTR_variant
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[uORF\]
- Stop lost \[oORF\]
- Start lost \[uORF\]
- Frameshift \[oORF\]
- Stop gained \[uORF\]
- intergenic_variant