# **LMNA -- Dilated Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:6636

**ClinGen evidence for haploinsufficiency**

LMNA encodes Lamins A and C, produced by alternative splicing of
transcripts encoded by this gene. Lamin A/C are intermediate filament proteins that localize to the inner nuclear membrane and nucleoplasm where they function in structural support, DNA organization and function. Mutations in LMNA are associated with a wide range of disorders (neuromuscular, cardiac, lipodystrophy and premature aging), some of which have overlapping and variably present clinical phenotypes. LMNA-associated phenotypes include both autosomal dominant and recessive inheritance patterns. A large number (\>400) of disease-associated LMNA variants have been reported; the vast majority are sequence-level alterations and are predominantly missense mutations. However frameshift, nonsense, and intragenic deletions and duplications, which are capable of resulting in a loss-of-function, are also reported. Thus LMNA haploinsufficiency is a possible mechanism of pathogenicity,
although it is not apparently the predominant mechanism. As yet, whole gene deletion of LMNA has not been reported in association with clinical phenotypes. Although the presence of \>2 independent publications with numerous loss-of-function-type variants, some of which include supportive functional data, provide evidence in support of LMNA haploinsufficiency, due to the extensive clinical heterogeneity,inheritance patterns, mutational spectrum, and lack of reports of whole gene deletions, the haploinsufficiency score given here is 2.

Gupta et al 2010 [[20127487]{.ul}](https://www.ncbi.nlm.nih.gov/pubmed/20127487) describe a deletion in LMNA in a patient with dilated cardiomyopathy, and the deletion caused less LMNA expression in cardiomyocytes. They identified a partial LMNA deletion (9 of 12 3? exons; exons 3-12, RefSeq NM_170707.3) in a patient with dilated cardiomyopathy by MLPA and qPCR. Functional studies on cardiomyocytes included immunostaining, which showed reduced lamin A/C levels, and electron microscopy, which showed
intracellular and nuclear abnormalities, supportive of a
loss-of-function due to the deletion. The authors propose
haploinsufficiency as the mechanism of pathogenicity based on the extent of the deletion, functional studies and supportive data from the literature.

MacLeod et al (2003) [12854972](https://www.ncbi.nlm.nih.gov/pubmed/12854972) identified a novel 2 base-pair deletion c.908_909delCT, causing a frameshift and truncated LMNA/C in a proband with history of paroxysmal atrial fibrillation, sick sinus syndrome, dilated cardiomyopathy. Per discussion: \"It is presently unknown whether autosomal dominant lamin A/C mutations impart their phenotype through dominant negative or
haploinsufficient mechanisms. The majority of reported LMNA mutations are missense. In this case, the mutant protein is thought to be expressed and to act through a dominant interfering mechanism. A smaller number of LMNA mutations are frameshifting and are expected to create truncated lamin A and C. In at least one case, a nonsense mutation at amino acid position 6 was described \[11\]. This mutation is effectively a null allele and therefore, dominant mutations, at least in this case, would be expected to arise from haploinsufficiency of lamin A/C.\"

Sebillion et al (2003) [12920062](https://www.ncbi.nlm.nih.gov/pubmed/12920062) identified an insertional mutation (28insA) that led to a stop codon and an aberrant truncated protein of 38 amino acids in a family affected by dilated cardiomyopathy with conduction defects\*. mRNA expression analysis was performed by signal intensity measurement of a RE-digested cDNA product
prepped from total RNA extraction from patient lymphoblastoid cell
lines; this analysis showed a reduction or loss of mutant fragment
compared to wild-type. In vitro cell transfection experiments showed a significant decrease in transfection efficiency for the 28insA cDNA, suggestive of reduced mRNA expression or stability leading to reduction of protein. The authors suggest this mutation may act via
haploinsufficiency. \*Note: the authors found no mutation in LMNA in
cases with isolated dilated cardiomyopathy.

**Literature review:**

*"LMNA*-related dilated cardiomyopathy (DCM) results
from [missense](https://www.ncbi.nlm.nih.gov/books/n/gene/glossary/def-item/missense/) variants,
with
occasional [nonsense](https://www.ncbi.nlm.nih.gov/books/n/gene/glossary/def-item/nonsense-variant/) or
splice-site variants and short insertions or deletions of *LMNA*.

The mechanism of cellular injury that causes *LMNA*-related DCM remains
incompletely understood. Because lamin A/C is a structural protein of
the nuclear membrane, it has been suggested that fragility of the
nuclear membrane in the setting of repetitive contraction of skeletal or
cardiac muscle may predispose to nuclear injury and cellular apoptosis.
An alternative hypothesis suggests that an abnormal lamin A/C protein
may disrupt the chromatin/lamin-associated protein complex, thereby
disturbing gene expression."

*Gene reviews <https://www.ncbi.nlm.nih.gov/books/NBK1674/>*

*"LMNA* missense and truncating mutations account for 5--8% of genetic
DCM
The mechanisms responsible for autosomal dominant DCM *LMNA* mutations
may be a mix of multiple defects including dominant-negative function as
well as
haploinsufficiency
Lamins A and C are implicated in many different cellular processes from
regulating gene expression, mechanosensing, DNA replication, and nuclear
to cytoplasmic transport."

*McNally et al. 2018 PMID: [28912180]*

Walsh et al found an excess of truncating and non truncating variants in LMNA associated with DCM in comparison to the reference dataset (ExAC)

*Walsh et al, 2016 (PMID 27532257)*

**From our in-house Atlas of DCM:**

**16/46 truncating**

Nonsense

Frameshift

Splicing

**30/46 non-truncating**

Missense

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=LMNA&icc=DCM>

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

Optional modifiers: 
Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism of cellular injury that causes *LMNA*-related DCM remains incompletely understood. Both haploinsufficiency and a dominant negative mechanism leading to **reduced or absent gene product or** **altered gene product structure** have been proposed. Because lamin A/C is a structural protein of the nuclear membrane, it has been suggested that fragility of the nuclear membrane in the setting of repetitive contraction of skeletal or cardiac muscle may predispose to nuclear injury and cellular apoptosis. An alternative hypothesis suggests that an abnormal lamin A/C protein may disrupt the chromatin/lamin-associated protein complex, thereby disturbing gene expression. LMNA*-related dilated cardiomyopathy (DCM) results from missense variants,
with occasional nonsense or splice-site variants and short insertions or deletions of *LMNA*. Penentrance is incomplete.

**List variant classes in this gene proven to cause this disease:**

Stop gained

Stop gained (predicted to undergo NMD)

Frameshift

Frameshift (predicted to undergo NMD)

Splice acceptor variant

Splice acceptor variant (predicted to undergo NMD)

Splice donor variant

Splice donor variant (predicted to undergo NMD)

Missense

In frame deletion

In frame duplication

**Potential novel variant classes based on predicted functional
consequence:**

Splice acceptor variant (predicted to escape NMD)

Splice donor variant (predicted to escape NMD)

Frameshift variant (predicted to escape NMD)

start_lost

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
