# **MLH1 -- Lynch Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7127

**ClinGen Evidence for Haploinsufficiency**
PMID 14635101 - Taylor et al (2003) screen 215 subjects referred to a clinic for phenotype consistent with autosomal dominant familial nonpolyposis colon cancer (Lynch Syndrome) and identify 6 MLH1 deletions (as well as 10 novel MLH1 mutations.)

PMID 15942939 - van der Klift (2005) screened a large cohort of families with hereditary nonpolyposis colon cancer and identified 13 MLH1 deletions from 68 unrelated kindreds.

**Literature review:**

Lynch syndrome is caused by pathogenic variants in genes involved with the mismatch repair (MMR) pathway. This pathway functions to identify and remove single-nucleotide mismatches or insertions and deletion loops. Pathogenic variants in four of the MMR genes can cause Lynch syndrome [Peltomäki 2003]. The functions of the MMR genes can be disrupted by missense variants, truncating variants, splice site variants, large deletions, or genomic rearrangements.

More than 200 different pathogenic variants have been reported in MLH1 [Peltomäki 2003, Peltomäki & Vasen 2004]; see Table A. Deletions account for 5%-10% of germline MLH1 pathogenic variants.

Constitutional inactivation of MLH1 by methylation, along with somatic loss of heterozygosity of the functional allele, has been reported as a rare cause of Lynch syndrome (~0.6%) [Niessen et al 2009]. These individuals have silencing of one MLH1 allele, throughout their tissues, due to methylation of the promoter and a Lynch syndrome phenotype. Most of such cases are simplex (i.e., a single occurrence in a family), but a few families with inherited hypermethylation have been reported [Goel et al 2011]. MLH1 promoter methylation is not detectable by either sequence analysis or duplication/deletion analysis of MLH1.

MLH1 acts in a recessive manner at the cellular level where there is an absence of functional Mlh1 protein in the tumor cells. This results from inactivation of both MLH1 alleles in the tumor, which often occurs as a result of an inactivating variant or silencing of the MLH1 promoter by hypermethylation.

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1211/*

Types of variant in MLH1:

Missense 40%
Nonsense or frameshift 40%
In-frame 2% (includes deletions, insertions or indels which do not affect the reading frame)
Splice  11%
Large rearrangement 10%

*Data from the InSiGHT database presented in Tamura K et al 2019 PMID: 31273487*

Variant types taken from InSiGHT database
*Plazzer JP et al 2013 PMID: 23443670*

Thirteen (27%) of the 48 genomic rearrangements reported from this study occurred in MLH1,
accounting for 14 patients (Fig. 1B). The exon 16
deletion was found in patients from 2 apparently
unrelated families. Notably, this deletion is different from the 3.5-kb MLH1 exon 16 deletion found
to represent a founder mutation in the Finnish
population (Nystrom-Lahti et al., 1995).

*van der Klift H et al 2005 PMID: 15942939*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

A heterodimer of MLH1 and PMS2 coordinates the interplay between the mismatch recognition complex and other proteins necessary for mismatch repair. Loss of function variants including large genomic rearrangements can result in an altered/reduced or absent MLH1 protein. Inactivation of both MLH1 alleles (following a second hit) resulting in an absence of functional Mlh1 protein can be seen at a cellular level in tumour cells. Rarely (~0.6% of patients) constitutional MLH1 promoter methylation can cause Lynch syndrome.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions
- In frame insertions


**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
