# **MSH2 -- Lynch Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7325

**ClinGen Evidence for Haploinsufficiency**
PMID 10850409 - Germline mutations within the MSH2 gene at 2p15 ( mismatch repair gene) have been seen in families Hereditary nonpolyposis colorectal cancer (HNPCC). HNPCC is an autosomal dominant disease characterized clinically by increased risk of early development of colorectal cancer as well as increased risk for other tumors. Charbonnier et al. analyzed a cohort of of families with Amsterdam criteria positive (AC+) fHNPCC and Amsterdam criteria negative (AC-) HNPCC. **A total of 12 genomic rearrangements and 12 point mutations were detected in the MSH2 gene (4 frameshift, 4 nonsense, 1 splice site, 2 missese).** One of the genomic rearrangments is a previously reported paracentric inversion of chromosome 2 (PMID 12203789) the remainder were exonic deletions of various lengths. The author also descibed a family that was negative for mutations within the MSH2 gene and other mis match repair (MMR) genes that had classical HNPCC and showed a high microsattelite insatbility phenotype and loss of MSH2 protein expression by IHC indicating the functional impairment of MSH2. Additional articles showing intragenic deletions include PMID 15949572, 9843200

PMID 12203789 - Wagner et al. describe a family with HNPCC (also known as Lynch syndrome) with a ~10 MB paracentric inversion that included exons 8-16 of the MSH2 gene. Inverse PCR showed the paracentric inversion breakpoints mapped within intron 7 and to a contig 10 Mb 3' of the MSH2 gene. Expression analysis was performed by a conversion analysis model which takes an Msh2 deficient rodent cellular background and using the patients lymphocytes to generate a somatic cell hybrid, analyzed the expression of the abnormal inverted MSH2 allele. Northern and Western blot analysis of the hybrid containing the abnormal inverted MSH2 show no detectable MSH2 mRNA or MSH2 protein, whereas the full length MSH2 allele was detected in the wild type allele hybrid.

PMID 9843200 - Wijnen et al. discussed the frequency of genomic deletion in HNPCC. A total of 137 families (51 Amsterdam criteria positive HNPCC and 86 Amsterdam negative with familial clustering of colorectal cancer reminiscent of HNPCC) were investigated by southern blot. Eight patients had aberrant restriction fragments indicating the presence of a genomic rearrangement. Four had aberrant restriction fragments indicative of an intragenic deletion in MSH2 encompassing various exons and a few of the deletions were confirmed by RT-PCR of the affected individuals. Author suggest that current technologies may not be able to detect all forms of mutations including genomic deletions (including complete gene deletions) and rearrangemnts and that mutation analysis of the MSH2 gene should include the examination of its genomic structure by Southern blot.

Although there have been no publications based on complete loss of the gene (hemizygous gene), there have been several reports of interstitial deletions, inversions, and point mutations (missense, framshift, nonsense) described in the literature that represent and autosomal dominant inheritance pattern. Functional studies have shown that deletions or disruptions result in a loss of function (inactivating mutations) (12203789, 9843200).

**Literature review:**

Lynch syndrome is caused by pathogenic variants in genes involved with the mismatch repair (MMR) pathway. This pathway functions to identify and remove single-nucleotide mismatches or insertions and deletion loops. Pathogenic variants in four of the MMR genes can cause Lynch syndrome [Peltomäki 2003]. The functions of the MMR genes can be disrupted by missense variants, truncating variants, splice site variants, large deletions, or genomic rearrangements.

More than 170 pathogenic variants have been identified in MSH2 [Peltomäki 2003, Peltomäki & Vasen 2004]. **The higher proportion of Alu repeats may contribute to the higher rate of genomic rearrangements in MSH2 than in MLH1 [van der Klift et al 2005]. At least 20% of germline MSH2 pathogenic variants are exon or multiexon deletions.**

MSH2 acts in a recessive manner at the cellular level where there is an absence of functional Msh2 protein in the tumor cells. This results from inactivation of both MSH2 alleles in the tumor, which often occurs by the mechanism of loss of heterozygosity. MSH2 promoter methylation has been shown to be the inactivating event that silences the normal allele in individuals with an MSH2-inactivating pathogenic variant. Of note, this is not a common cause of sporadic colon cancer.

**Heterozygosity for an MSH2 pathogenic variant is associated with the greatest risk for extracolonic cancers.**

MSH2 pathogenic variants have been reported more commonly than a pathogenic variant in the other three MMR genes in individuals with the Muir-Torre variant of Lynch syndrome [South et al 2008].

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1211/*

Types of variant in MSH2:

Missense 31%
Nonsense or frameshift 49%
In-frame 2% (includes deletions, insertions or indels which do not affect the reading frame)
Splice  8%
Large rearrangement 10%

Apart from exons andintron/exon borders, a few point mutations have been identified in the promoter regions of MSH2

*Data from the InSiGHT database presented in Tamura K et al 2019 PMID: 31273487*

Among the 48 genomic rearrangements, the
majority (60%) affected the MSH2 gene.
A single deletion of MSH2 exons 1–6 is responsible for several North American HNPCC cases and has been shown to represent a
U.S. founder mutation (Wagner et al., 2003; Lynch
et al., 2004). 

*van der Klift H et al 2005 PMID: 15942939*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The protein encoded by MSH2 forms a heterodimer with either DNA MMR protein MSH6 or MSH3 and functions to identify mismatches for mismatch repair. Loss of function variants including large genomic rearrangements (10% of pathogenic variants) can result in an altered/reduced or absent MSH2 protein. Inactivation of both MSH2 alleles (following a second hit) resulting in an absence of functional protein can be seen at a cellular level in tumour cells. The higher proportion of Alu repeats may contribute to the higher rate of genomic rearrangements in MSH2. MSH2 pathogenic variants are associated with the greatest risk for extracolonic cancers compared to the other MMR genes.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions
- In frame insertions


**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
