# **MSH6 -- Lynch Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7329

**ClinGen Evidence for Haploinsufficiency**

PMID 14974087 - Plaschke et al 2004 identifies 7 truncating mutations in MSH6 associated with tumors displaying high levels of micro satellite instability.

PMID 2815724 - Baglietto et al (2010) assess the risk of Lynch syndrome in **113 MSH6 mutation carriers (truncation, loss, frameshift indel, or missense damaging) collected from multiple cancer centers.** "For MSH6 mutation carriers, the estimated cumulative risks to ages 70 and 80 years, respectively, were as follows: for colorectal cancer, 22% (95% confidence interval [CI]?=?14% to 32%) and 44% (95% CI?=?28% to 62%) for men and 10% (95% CI?=?5% to 17%) and 20% (95% CI?=?11% to 35%) for women; for endometrial cancer, 26% (95% CI?=?18% to 36%) and 44% (95% CI?=?30% to 58%); and for any cancer associated with Lynch syndrome, 24% (95% CI?=?16% to 37%) and 47% (95% CI?=?32% to 66%) for men and 40% (95% CI?=?32% to 52%) and 65% (95% CI?=?53% to 78%) for women. Compared with incidence for the general population, MSH6 mutation carriers had an eightfold increased incidence of colorectal cancer (HR?=?7.6, 95% CI?=?5.4 to 10.8), which was independent of sex and age. Women who were MSH6 mutation carriers had a 26-fold increased incidence of endometrial cancer (HR?=?25.5, 95% CI?=?16.8 to 38.7) and a sixfold increased incidence of other cancers associated with Lynch syndrome (HR?=?6.0, 95% CI?=?3.4 to 10.7)."

Gene Reviews (http://www.ncbi.nlm.nih.gov/books/NBK1211/) **estimates ~7-10% Lynch Syndrome subjects have germline mutation in MSH6 (deletions, frameshift, stops) that cause loss of MSH6. Throughout literature this estimate is repeated. While the mechanism is not explicit, overall it is thought heterozygous mutations in MSH6 impair mismatch repair.**

From Bellizzi et al (2009, PMID:19851131): "The MMR proteins function as heterodimers. The MSH2-MSH6 complex recognizes mispaired bases and insertion/deletion loops. It recruits MLH1-PMS2, which subsequently directs the remainder of the MMR machinery. MSH2 and MLH1 are the dominant (obligate) constituents of their respective pairs. In the absence of MSH6, MSH2 can pair with MSH3, and in the absence of PMS2, MLH1 can pair with PMS1. This may partially explain the somewhat attenuated Lynch phenotype attributed to MSH6 and PMS2 mutation, (reduced penetrance and older age of onset relative to MSH2 and MLH1 mutants?)"

**Literature review:**

Lynch syndrome is caused by pathogenic variants in genes involved with the mismatch repair (MMR) pathway. This pathway functions to identify and remove single-nucleotide mismatches or insertions and deletion loops. Pathogenic variants in four of the MMR genes can cause Lynch syndrome [Peltomäki 2003]. The functions of the MMR genes can be disrupted by missense variants, truncating variants, splice site variants, large deletions, or genomic rearrangements.

More than 30 pathogenic variants have been identified in MSH6 [Peltomäki & Vasen 2004]. Exon or multiexon deletions are a rare cause of germline MSH6 pathogenic variants.

MSH6 acts in a recessive manner at the cellular level where there is an absence of functional MSH6 protein in the tumor cells. This results from inactivation of both MSH6 alleles in the tumor, which often occurs by the mechanism of loss of heterozygosity.

Heterozygosity for a pathogenic variant in MSH6 is associated with MSI-low tumors. The colorectal cancers in families with an MSH6 pathogenic variant may be later in onset and more distally located than the cancers in families with Lynch syndrome resulting from a pathogenic variant in one of the other MMR genes; endometrial cancer is commonly observed in females with an MSH6 pathogenic variant [Wu et al 1999, Berends et al 2002]. Slightly lower risks for colorectal cancer and higher risks for endometrial cancer have been reported in families with an MSH6 pathogenic variant than in families with an MLH1 or MSH2 pathogenic variant [Berends et al 2002, Baglietto et al 2010].

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1211/*

Types of variant in MSH6:

Missense 49%
Nonsense or frameshift 43%
In-frame 3% (includes deletions, insertions or indels which do not affect the reading frame)
Splice  3%
Large rearrangement 2%

*Data from the InSiGHT database presented in Tamura K et al 2019 PMID: 31273487*

Variant types taken from InSiGHT database
*Plazzer JP et al 2013 PMID: 23443670*


#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The MSH2-MSH6 complex recognizes mispaired bases and insertion/deletion loops. It recruits MLH1-PMS2, which subsequently directs the remainder of the MMR machinery.
Loss of function of MSH6 results in dysfunction of this repair pathway. MSH2 and MLH1 are the dominant (obligate) constituents of their respective pairs. In the absence of MSH6, MSH2 can pair with MSH3, and in the absence of PMS2, MLH1 can pair with PMS1. This may partially explain the somewhat attenuated Lynch phenotype attributed to MSH6 and PMS2 mutation.
Truncating and non-truncating pathogenic variants in MSH6 have been reported; exon or multiexon deletions are a rare cause of germline MSH6 pathogenic variants.
Slightly lower risks for colorectal cancer and higher risks for endometrial cancer have been reported in families with an MSH6 pathogenic variant than in families with an MLH1 or MSH2 pathogenic variant.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions
- In frame insertions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]

