# **MUTYH -- Polyposis**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7527

**ClinGen Evidence for Haploinsufficiency**

**Literature review:**

Autosomal recessive colorectal adenomatous polyposis is a disorder characterized by adult-onset of multiple colorectal adenomas and adenomatous polyposis. Affected individuals have a significantly increased risk of colorectal cancer.

Among 614 families recorded in 6 regional registers of polyposis in the UK, **Sampson et al. (2003) identified 111 with neither dominant transmission nor evidence of APC mutation. Molecular genetic analysis showed that 25 had biallelic mutations of the MYH gene. The data showed that MYH polyposis can be transmitted as an autosomal recessive trait**

Al-Tassan et al. (2002) studied a British family in which 3 sibs had multiple colorectal adenomas and carcinoma. There was no clear pathogenic change in the APC gene. Analysis of the MYH gene showed that the sibs were compound heterozygous for nonconservative missense variants.

Rouleau et al. (2011) reported a 45-year-old French man who was found to have 25 colorectal adenomas on colonoscopy. He had no family history of the disorder. Analysis of the APC gene was negative, and molecular analysis identified compound heterozygosity for mutations in the MUTYH gene: a missense mutation and a large rearrangement resulting in the deletion of exons 3 to 16. 

*Omim
https://omim.org/entry/608456*

Adenine DNA glycosylase (encoded by MUTYH): recognizes and excises misincorporated adenine bases to prevent G:C>T:A transversions from occurring.

A lack of functional adenine DNA glycosylase leads to accumulation of G:C>T:A transversions in daughter DNA strands post replication. Studies indicate that this transversion is common in colorectal tumor DNA from individuals with MAP. These pathogenic variants result in loss of adenine DNA glycosylase function [Lipton & Tomlinson 2004].

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK107219/*

Thirty-six different mutations were found in 185 index patients (Table 1). The Y179C mutation represented 44% of all mutations (164/370), and the G396D represented 24% of all mutations (90/370). Truncating mutations (64/370) represented 17% of all changes.

We found that Y179C homozygotes presented earlier and had a significantly greater CRC hazard than G396D homozygotes and G396D/Y179C compound heterozygotes.

In contrast to the differences in phenotypic expression we observed between the most common MUTYH missense mutations, we found no significant difference between truncating and nontruncating mutations except for polyp count (ie, the number of truncating alleles was inversely correlated with the proportion of cases with <10 polyps).

*Nielsen M et al PMID: 19032956*

about 80 pathogenic mutations distributed throughout the gene and located at positions corresponding to different functional domains of the protein (see: Leiden Open Variation Database). **Although various types of alterations have been reported in MAP patients, including nonsense, small insertion/deletion, and splicing variants, missense mutations represent the great majority of the detected changes.**

**A number of variants appear recurrent in different populations, with Y179C (previously annotated as Y165C) and G396D (previously annotated as G382D) missense mutations accounting together for about 70% of germline alterations found in European patients** (reviewed by Cheadle and Sampson, 2007) however, in Asian populations, Y179C and G396D must be rare, since neither mutation has been found in MAP patients. On the other hand, other mutations have proven to be recurrent in patients from particular populations (reviewed by Poulsen and Bisgaard, 2008). Taken together, these findings indicate that sequencing of the entire MUTYH open reading frame has to be performed for the genetic testing, especially in populations of mixed ethnicity.

Recently, a large gene deletion spanning exons 4â€“16 has been found in two unrelated patients showing an attenuated phenotype; due to this observation, appropriate methods to detect gene rearrangements should be considered, at least for patients carrying either a single heterozygous mutation or a (apparently) homozygous disease-causing mutation (Rouleau et al., 2011; Torrezan et al., 2011). Besides rare mutations, a polymorphic allele (SNP rs3219468: G>C) associated with a significant reduction of a MUTYH transcription product has recently been implicated in CRC risk (Plotz et al., 2012).

*Venesio T et al 2012 PMID: 22876359*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal recessive

(optional) modifiers: 

**Allelic requirement:**

biallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Pathogenic variants in MUTYH result in loss of adenine DNA glycosylase function leading to an accumulation of G:C>T:A transversions in daughter DNA strands post replication. Studies indicate that this transversion is common in colorectal tumor DNA from individuals with MAP. Various types of alterations have been reported, including nonsense, small insertion/deletions, and splicing variants, however missense mutations represent the great majority of the detected changes. A number of variants appear recurrent in different populations, with Y179C (previously annotated as Y165C) and G396D (previously annotated as G382D) missense mutations accounting together for about 70% of germline alterations found in European patients; other mutations have proven to be recurrent in patients from particular populations (reviewed by Poulsen and Bisgaard, 2008). There is no clear genotype phenotype correlation however there appears to be some association between truncating variants and polyp number.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions


**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame insertions
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]

