# **MYBC3 -- Hypertrophic cardiomyopathy**


**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

<https://search.clinicalgenome.org/kb/genes/HGNC:7551>

The MYBPC3 gene has been associated with autosomal dominant hypertrophic cardiomyopathy (HCM) using the ClinGen Clinical Validity Framework. This association was made using case-level data and case control data. MYBPC3 was first associated with HCM in 1995 (Watkins et al, 1995, PMID 7493025). There are over 290 variants asserted as pathogenic for MYBPC3 for HCM in ClinVar, and mutations in MYBPC3 are reported in 40% of the reported cases of HCM (Cirino and Ho, 2014, GeneReviews, PMID 20301725).
More evidence is available in the literature, but the maximum score for genetic evidence and/or experimental evidence (12 pts.) has been reached. Of note, MYBPC3 has been shown to cause HCM in an autosomal recessive fashion, with earlier and more severe presentation of phenotypes associated with HCM, and represents a semi-dominant condition. **The molecular mechanism for HCM is loss of function (LOF), and missense, nonsense, frameshift and splice site mutations in MYBPC3 have been shown to be pathogenic for cardiomyopathy.** Of note, this gene has been implicated in dilated cardiomyopathy and left ventricular noncompaction. This gene-disease association is supported by biochemical, expression, protein interaction, and animal models evidence. In summary, MYBPC3 is definitively associated with autosomal dominant HCM. This has been repeatedly demonstrated in both the research
and clinical diagnostic settings and has been upheld over time. This classification was approved by the ClinGen Hypertrophic Cardiomyopathy Expert Panel on September 5, 2017.

**Numerous frameshift, splicing site, exonic deletion and nonsense
mutations that all lead to premature termination support
haploinsufficiency as a pathogenetic mechanism. Complete deletion of MYBPC3 in patients with large CNV has not been reported with
cardiomyopathy phenotype.**

**Literature review:**

Walsh et al found an excess of truncating and non-truncating variants in MYBPC3 associated with HCM in comparison to the reference dataset (ExAC)


**From our in-house Atlas of HCM:**

*566/1176 truncating*

*610/1176 non-truncating*

<https://www.cardiodb.org/acgv/acgv_gene.php?gene=MYBPC3>

*Walsh et al, 2016 (PMID 27532257)*

"Truncating variants in MYBPC3, which we can estimate based on case and reference frequencies are causative in over 9% of HCM cases, have an EF>0.99 confirming that this variant class has a high likelihood of pathogenicity concordant with pedigree and functional studies."

*Walsh et al 2019 PMID: 30696458*

"90% of *MYBPC3* mutations are heterozygous frameshift, nonsense, or splice site mutations that result in premature termination codons (PTCs) on 1 allele. As such, these mutations are thought to result in HCM from an allelic loss of function via NMD of PTC-containing transcripts, leading to a reduction in MyBP-C content in the sarcomere. Alternatively, these mutations may result in production of truncated MyBP-C, though truncated MyBP-C has never been detected in human heart tissue."

*Helms et al.2020 PMID: 31877118*

"MYBPC3 mutations are the most common cause of hypertrophic
cardiomyopathy, accounting for about half of identified mutations...**many are within introns and are predicted to cause aberrant splicing leading to a frameshift and a premature chain termination, yet the truncated peptides have never been identified in human heart tissue carrying these mutations. Instead of expression of a poison peptide we consistently observe haploinsufficiency of MyBP-C in MYBPC3 mutant human heart
muscle."**

*Marston S et al. 2012 PMID: 22057632*

In a cohort of 114 Chinese patients with HCM, a total of 20 different mutations (8 novel and 12 known mutations) of *MYBPC3* were identified from 25 patients (21.9%). 

"Phenotype-genotype analyses showed that the patients with double
mutations (n = 2) or premature termination codon mutations (n = 12) showed more severe manifestations, compared with patients with missense mutations (n = 11). Particularly, we identified a recurrent truncation mutation (p.Y842X) in four unrelated cases (4/25, 16%), who showed severe phenotypes and suggest that the p.Y842X is a frequent mutation in Chinese HCM patients with severe phenotypes."

*Lui X et al. 2015 PMID: 26573135*

"Ten of the 18 probands with two or more P/LP variants had compound heterozygous or homozygous variants in the same gene, including 5 probands with two or more *MYBPC3* variants.

The average age (±SD) of probands referred for genetic testing tended to be younger among those with two or more P/LP variants (29 ± 3 years) compared with those with only one P/LP variant (39 ± 21 years; *P* = 0.29)."

*Alfares A et al. 2015 PMID: 25611685*

"We found 29 rare MYBPC3 splice-site variants in 56 of 557 (10%) unrelated HCM probands. Three variants were not predicted to alter RNA splicing, and 13 essential splice dinucleotide, nonsense, and short insertion or deletion variants were not further assessed. RNA analysis was performed on 9 variants (c.654+5G>C, c.772G>A, c.821+3G>T, c.927-9G>A, c.1090G>A, c.1624G>A, c.1624+4A>T, c.3190+5G>A, and c.3491-3C>G), and RNA splicing errors were confirmed for 7. Four variants in 4 families resulted in clinically meaningful reclassifications...

There were 180 unique variants [in ClinVar] in splice-site regions of MYBPC3, of which 56 (32%) were variants of uncertain significance, 98 (54%) were pathogenic or likely pathogenic, and 26 (14%) were benign or likely benign.

RNA sequence analysis can directly confirm the outcomes of RNA splicing. We found that 3 variants annotated as causing a missense change, in fact caused exon skipping and lead to frameshifts. MYBPC3 splice-site variants may cause other unexpected outcomes, such as splicing from alternative sites, as demonstrated with the c.654+5G>C variant and intron retention.."

*Singer ES et al 2019 PMID: 30645170*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

Optional modifiers:
Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is likely **loss of function** due to **reduced or absent gene product or** **altered gene product structure** leading to a reduction in MyBP-C content in the sarcomere. 90% of *MYBPC3* mutations are heterozygous frameshift, nonsense, or
splice site mutations that result in premature termination codons on 1 allele. There are reports of patients with double mutations or premature termination codon mutations showing more severe manifestations, compared with patients with missense mutations. Usual mode of inheritance is autosomal dominant but homozygous and compound heterozygous variants have been reported and appear to confer a more severe phenotype. It is characterised by incomplete penetrance. Hot spot regions for HCM in MYBPC3 include amino acid residues 485–502, 1248–1266.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**PM1** 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

Hot spot regions for HCM in MYBPC3 include amino acid residues 485–502, 1248–1266
Etiological fraction - 0.979 (0.971–0.987) for these HCM clusters. PM1_strong could be applied for variants in these regions.

**PVS1**
Can be applied for truncating variants in MYBPC3,

*Walsh et al 2019 PMID: 30696458*


**List variant classes in this gene proven to cause this disease:**

Stop gained

Stop gained (predicted to undergo NMD)

Frameshift

Frameshift (predicted to undergo NMD)

Splice acceptor variant

Splice acceptor variant (predicted to undergo NMD)

Splice donor variant

Splice donor variant (predicted to undergo NMD)

Splice region variant

Missense

In frame deletion

In frame insertion

**List other variant classes predicted to lead to the same functional consequence:**

Splice acceptor variant (predicted to escape NMD)

Splice donor variant (predicted to escape NMD)

Frameshift variant (predicted to escape NMD)

start_lost

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
