**MYH7 - HCM**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen**: 

https://search.clinicalgenome.org/kb/genes/HGNC:7577

MYH7 HCM definitive (narrative not given)

**ClinGen Haploinsufficiency phenotype comments:**

To date only articles discussing missense mutations in the MYH7 gene have been described in the literature as being associated with Familial hypertrophic cardiomyopathy, dilated cardiomyopathy, Laing Distal Myopathy (LDM), Myosin Storage Myopathy (PMID:23346452, 12788380,
12707239, 21846512, 25576864,24664454 ). PMID: 12788380 does however describe a three base pair deletion leading to an amnio acid loss glutamic acid in position 927. There was no disruption of the reading frame downstream of the deletion.

**Literature review:**

"Missense variants in MYH7 are believed to cause HCM
through gain-of-function actions: variants produce an abnormal activated protein that incorporates into the sarcomere as a ‘poison peptide’ [2]. Haploinsufficiency in MYH7 is not
a recognised disease mechanism for HCM, and heterozygous variants that introduce premature termination codons (PTCs - i.e. nonsense, frameshift and splice variants) have not been demonstrated to be associated with this disease. 

Here we report a frameshift variant in MYH7, c.5769delG,
that is associated with HCM in an Egyptian cohort (3.3%) compared with ethnically-matched controls. This variant is absent from previously published large-scale Caucasian HCM cohorts. We further demonstrate strong evidence of co-segregation of c.5769delG with HCM in a large family (LOD score: 3.01). The predicted sequence of the variant MYH7 transcript shows that the frameshift results in a premature termination codon (PTC) downstream of the last exon-exon junction of the gene that is expected to escape nonsense-mediated decay (NMD). RNA sequencing of myocardial tissue obtained from a patient with the variant during surgical myectomy confirmed the expression of the variant MYH7 transcript."

*Allouba M et al 2020 submitted*

"MYH7 LOF variants are very rare and their contribution to inherited cardiomyopathy is incompletely understood. While there is currently no evidence for a disease-causing role in the heterozygous state, compound heterozygosity of LOF variants along with missense variants can lead to extremely severe presentations, mimicking recessive inheritance.25, 26"

*Kelly MA et al 2018 PMID: 29300372* 

**From our in-house Atlas of HCM:**

849/864  missense

10/864  in frame deletion

There are some unpublished truncating variants in the terminal exon which seem to be enriched in certain case cohorts and may escape NMD and also at least one splice region variant that turns out to cause an in-frame exon skipping event (at a relatively distal exon), leading to expression of an altered gene product structure.

https://www.cardiodb.org/acgv/acgv_gene.php?gene=MYH7

*Walsh R et al 2017 PMID:27532257*

**Pilot application of harmonised terms:**

**Inheritance:**

Autosomal dominant

Optional Modifiers: Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

Optional Modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily increase in sarcomere force generation due to **altered protein product structure**. Missense variants in MYH7 are believed to cause HCM "through gain-of-function actions: variants produce an abnormal activated protein that incorporates into the sarcomere as a ‘poison peptide’."

A novel frameshift variant has been identified in 3.3% of Egyptian HCM patients. It is predicted to result in a premature termination codon (PTC) downstream of the last exon-exon junction of the gene that is expected to escape nonsense-mediated decay (NMD).

MYH7 LOF variants are very rare and their contribution to inherited cardiomyopathy is incompletely understood. While there is currently no evidence for a disease-causing role in the heterozygous state, compound heterozygosity of LOF variants along with missense variants can lead to extremely severe presentations, mimicking recessive inheritance.

**Additional information related to ACMG evidence types**

**PM2** A filtering allele frequency (FAF) <0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1%
Assumptions (note: values deliberately set conservative to add “safety padding”)
Disease prevalence: 1/200 individuals (1/400 chromosomes) was used as a “safe” value.
Penetrance: Although MYH7 is generally regarded as a “penetrant” cardiomyopathy gene, this is not well characterized at the variant level and therefore, to accommodate all variants, a penetrance value of 30% was used.
Gene contribution was set at 10.6% based on the detection rate for HCM, which is the highest among MYH7
associated cardiomyopathies4

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM1** Applicable when missense variant is located within the head domain (codons 181-937, NM_000257)
*Kelly MA et al 2018 PMID: 29300372* 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

**MYH7 (residues 167–931) etiological fraction 0.976 (0.972–0.981)** so for this region PM1_strong could be applied

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease**

Missense

**List other variant classes predicted to lead to the same functional consequence**

splice_donor_variant

splice donor variant predicted to escape NMD

splice acceptor variant predicted to escape NMD

frameshift variant predicted to escape NMD

stop_gained predicted to escape NMD

stop_lost

inframe_insertion

inframe_deletion
