# **MYL2 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

<https://search.clinicalgenome.org/kb/gene-validity/8768>

The MYL2 gene has been associated with autosomal dominant hypertrophic cardiomyopathy (HCM) in at least 40 probands in 10 publications (Poetter et al, 1996, PMID 8673105; Flavigny et al, 1998, PMID 9535554; Kabaeva et al, 2002, PMID 12404107; Richard et al, 2003, PMID 12707239; Morner et al, 2003, PMID 12818575; Garcia-Pavia et al, 2011, PMID 21896538; Santos et al, 2012, PMID 22429680; Berge et al, 2013, PMID 24111713; Lopes et al, 2015, PMID 25351510; Claes et al, 2015, PMID 26497160).
**More than 8 unique variants (missense, splice site, nonsense,
frameshift) have been identified in humans, and convincing segregation data has been reported** (Flavigny et al, 1998, PMID 9535554). In addition, at least 8 missense VUSs in MYL2 have been reported in patients with HCM. MYL2 was first associated with HCM in humans in 1996 (Poetter et al, 1996, PMID 8673105). The MYL2 gene was significantly enriched for missense variants in Walsh et al, 2016 (PMID 27532257) with and odds ratio of 6.74 (95% CI 4.69-9.70) for non-truncating variants and 3.10 (95% CI 0.89-10.7) for truncating variants. This gene-disease
association is supported by expression data (Price et al, 1980, PMID 7236212), interaction with other known HCM gene products (MYH7, MYL3) (Rayment et al, 1993, PMID 8316857), and animal models (Szczesna-Cordary et al, 2005, PMID 16076902; Wang et al, 2006, 16837010; Kerrick et al, 2009, PMID 18987303). In summary, MYL2 is definitively associated with autosomal dominant HCM. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen Hypertrophic
Cardiomyopathy Gene Curation Expert Panel on February 7th, 2017.

**ClinGen Haploinsufficiency phenotype comments:**
MYL2 is associated with cardiomyopathy by different pathogenic mechanisms with different modes of inheritance that can be recessive or dominant, depending on genotype. Homozygous or compound heterozygous loss-of -function variants have been reported to cause autosomal recessive form cardiomyopathy. Furthermore, the heterozygous carriers did not show evidence of the disorder (PMID: 24111713, 23365102). Other missense changes cause autosomal dominant hypertrophic cardiomyopathy presumably due to loss of the normal physiologic function, for instance, loss of the contraction power stoke or altered actin sliding velocities (PMID: 25825243; 25324513). One paper describes a founder missense pathogenic variant capable of causing HCM by itself or augmented by other factor such as arterial hypertension (PMID: 26497160). MYL2 variants, among others, identified as incidental findings in an exome cohort (PMID: 23861362). There are several heterozygous frameshifts, stop gains, and splice site disruptions listed in ExAc for MYL2 implying that haploinsufficiency is not a disease mechanism for this gene.

**Literature review:**

[[Poetter et al. (1996)]analyzed the MYL2 gene in 399 unrelated probands with hypertrophic cardiomyopathy
(see CMH10, [[608758]{.ul}](https://omim.org/entry/608758)), and
identified heterozygosity for 3 different missense mutations in 4
probands ([[160781.0001]{.ul}](https://omim.org/entry/160781?search=MYL2&highlight=myl2#0001)-[[160781.0003]{.ul}](https://omim.org/entry/160781?search=MYL2&highlight=myl2#0003)),
3 of whom had an unusual mid-left ventricular chamber thickening on echocardiography.

Flavigny et al. (1998) screened 42 probands from unrelated families with CMH for mutations in the MYL2 gene and identified 2 novel mutations, R58Q (160781.0004) and P18L (160781.0005), in 3 probands. The mutations were subsequently found in all affected family members, who were classified morphologically as Maron type 1, 2, or 3; none had the variant form of CMH described by Poetter et al. (1996). 

Szczesna et al. (2001) studied the effects of 5 mutations in the MYL2 gene on Ca(2+) binding and phosphorylation and found that both processes were significantly affected by all of the mutations. For example, the E22K mutation resulted in a 17-fold decrease in calcium binding compared with wildtype, and the R58Q mutant did not bind Ca(2+) at all. Ca(2+) binding to the R58Q mutant was restored upon phosphorylation, whereas the E22K mutant could not be phosphorylated. In addition, the alpha-helical content of phosphorylated R58Q greatly increased with Ca(2+) binding.

*OMIM https://www.omim.org/entry/160781#molecularGenetics*

*Poetter et al, 1996, PMID 8673105*
*Flavigny et al 1998 PMID 9535554*
*Szczesna et al 2001 PMID 11102452*

**From our in-house Atlas of HCM:**
There is a significant excess of non-truncating variants in MYL2 in HCM cases vs population controls. The excess of truncating variants is not statistically significant.

42/46 missense

1/46 inframe deletion (VUS)

2 nonsense (VUS)

1 frameshift (VUS)

The frameshift variant is described as 'hot VUS'. There are 2
submissions in ClinVar. It is expected to cause a premature truncatation codon and result in a disrupted or absent protein. As LoF is not an established mechanism of disease it has been classed as a VUS.

1 nonsense variant is also classed as a 'hot VUS' for similar reasons and has 4 submissions on ClinVar.

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=MYL2&icc=HCM>

*Walsh et al, 2016 PMID 27532257*

Homozygous or compound heterozygous mutations in the last exon of MYL2 have been reported as responsible for an autosomal recessive lethal myosinopathy characterised by rapidly progressive generalised muscle weakness and heart failure due to primarily dilated cardiomyopathy (but in some cases restrictive or septal hypertrophy).

*Weterman et al 2013. PMID 23365102*

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily due to **altered gene product structure** as the main class of pathogenic variants described are missense. There have been a few reports of nonsense and frameshift variants but as loss of function does not appear to be the mechanism, their pathogenicity has not been confirmed. Inheritance is primarily autosomal dominant with incomplete penetrance.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

**MYL2 across whole gene etiological fraction is 0.890 (0.851–0.930)** so for this region PM1_supporting could be applied

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease:**

- Missense

**List other variant classes predicted to lead to the same functional consequence**

- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Splice acceptor variant predicted to escape NMD
- Stop_lost
- In frame_insertion
- In frame deletion
