# **MYL3 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

<https://search.clinicalgenome.org/kb/gene-validity/8499>

The MYL3 gene has been associated with autosomal dominant hypertrophic cardiomyopathy (HCM) in at least 20 probands in 13 publications. More than 8 unique variants (mostly missense, 1 splice acceptor variant) have been reported in humans, and variants in this gene segregated with disease in 24 additional family members. MYL3 was first associated with HCM in humans in 1996 (Poetter et al, PMID 8673105). The MYL3 gene was
significantly enriched for missense variants in Walsh et al. 2016 (PMID 27532257), with an Odds Ratio of 5.00 (3.43-7.27) for HCM. This gene-disease association is supported by expression studies (Fujimoto et al, 1993, PMD 8417110), a mouse model (Vemuri et al, 1991, PMID 9927691), and evidence of interaction with MYH7 (Petzhold et al, 2011, PMID 21262909) as well as ACTC1 (Haase et al, 2006, PMID 16675844). In summary, MYL3 is definitively associated with autosomal dominant HCM. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification
was approved by the ClinGen Hypertrophic Cardiomyopathy Gene Curation Expert Panel on December 22, 2017.

**ClinGen Evidence for Haploinsufficiency**
**"There is no evidence at present to show that MYL3 variants have a loss of function effect.**

MYL3 is associated with dominant hypertrophic cardiomyopathy. Described pathogenic missense variants in the MYL3 gene cause the disorder by altering its normal physiologic function (e.g: impairing essential interaction with other components of the sarcomere) (PMID: 22131351 ). There is one report of a case with recessive inheritance due to autozygosity. The variant reported in recessive HCM was a homozygous missense change (p.Glu143Lys). Functional studies were not performed (PMID: 12021217, 21823217). ClinVar classifies the Glu143Lys variant as "uncertain" and reports it found in individuals with pathogenic changes in other genes. One report describes a splice site alteration predicted to cause exon skipping and no parental studies were done (PMID: 19035361). One frameshift variant in the middle of the gene has been described but it is unclear if it was causative of HCM or not (PMID: 25132132), and there are two instances of frameshift and one of an early canonical splice site change in ExAc (all at very low freq). There are three transcripts for MYL3 with different reading frames at the frameshift locations. The Database of Genomic Variants (DGV) lists a number of deletion CNVs including MYL3 and reported by multiple groups."

**Literature review:**

**Pathogenic missense variants are described in MYL3, likely causing disease by altering normal function and impairing protein protein interaction with components of the sarcomere. There are reports on ClinVar of frameshift, UTR and splice site variants but these are classified as uncertain significance or conflicting.
There is a splice acceptor variant classified as VUS-favour pathogenic but functional studies are unavailable.**

"Olson et al. (2002) reported a consanguineous family in which 3 sibs had presented with childhood-onset CMH characterized by midcavitary left-ventricular hypertrophy (CMH8; 608751). Both parents had completely normal hearts in their 40s. Mutation screening in a surviving affected sib revealed a homozygous missense G-to-A point mutation at codon 143 of the MYL3 gene, resulting in a glutamic acid-to-lysine (E143K) substitution. Heterozygotes had normal hearts. Sequence alignment of myosin essential light chains demonstrated high conservation of glutamic acid at position 143 across species. The E143K mutation was absent from 150 normal control DNA samples. **The authors concluded that this was a true autosomal recessive form of CMH8.**"
*Omim https://www.omim.org/entry/160790*

The same variant has been described in conjunction with pathogenic variants in other HCM genes. Additionally the same variant has been identified in the heterozygous state in \>35 individuals (primarily Latino ancestry) with HCM. See ClinVar for additional evidence and publications (*Gomez 2014,
McNamara 2017*) https://www.ncbi.nlm.nih.gov/clinvar/variation/14063/.

*Olson 2002 PMID 12021217; Caleshu 2011 PMID 21823217*

"[Poetter et al.(1996)]analyzed the MYL3 gene in 383 unrelated probands with hypertrophic cardiomyopathy and identified a
heterozygous missense mutation at a conserved residue
(M149V; [160790.0001] that segregated with disease in a large 3-generation family."

*Omim https://www.omim.org/entry/160790* 

**From our in-house Atlas of HCM:**

37/37 missense

[[https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=MYL3&icc=HCM]{.ul}](https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=MYL3&icc=HCM)

*Walsh et al. 2016 (PMID 27532257)*

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily due to **altered gene product structure.** The pathogenic variants described are missense variants likely causing disease by altering normal function and impairing protein protein interaction with components of the sarcomere. There are reports on ClinVar of frameshift, UTR and splice site variants but these are classified as uncertain significance or conflicting. The MYL3 gene was significantly enriched for missense variants in Walsh et al. 2016 (PMID 27532257), with an Odds Ratio of 5.00 (3.43-7.27) for HCM. There was no enrichment in cases for truncating variants.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**PM1** 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

**MYL3 HCM cluster (amino acid residues 143–180) etiological fraction 0.925 (0.886–0.965)** so for this region PM1_moderate could be applied

EF across the whole gene is 0.833 (0.772–0.895)

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease:**

- Missense

**List other variant classes predicted to lead to the same functional consequence**

- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Splice acceptor variant predicted to escape NMD
- Stop_lost
- In frame_insertion
- In frame deletion
