**NF2 -- Neurofibromatosis, type 2 (MIM 101000)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:7773

The first report indicating a relationship of the NF2 gene with
autosomal dominant Neurofibromatosis type 2 was reported by Rouleau et al., 1993 (PMID: 8379998). Numerous variants have been reported, in both ClinVar and in LOVD (https://databases.lovd.nl/shared/genes/NF2).
Evidence supporting this gene-disease relationship includes case-level data, segregation data, functional data, and model organisms. This gene-disease relationship has been studied for more than 20 years, therefore a significant amount of case-level data, segregation data, and experimental data is available and the maximum score for genetic evidence (12 points) and experimental evidence (6 points) has been
reached... The mechanism for the gene-disease relationship is **protein loss of function**, as the NF2 protein product, termed Merlin or Schwannomin, is a **tumor suppressor protein** (Trofatter et al., 1993;
PMID: 8453669). NF2 tumor suppressor function is regulated by the
protein confirmation, with the closed form acting as the active tumor
suppressor form (reviewed in Cooper and Giancotti, 2014 PMID:24726726).
NF2 has been associated with multiple disease entities and/or
phenotypes, including: (1) Meningioma, NF2-related, somatic (MIM:
607174) (2) Schwannomatosis, somatic Of note, these other phenotypes are
noted as somatic, and therefore are not represented/ counted in this
gene-disease relationship, and will be assessed separately. In summary,
**NF2 is DEFINITIVELY associated with autosomal dominant
Neurofibromatosis Type 2**. This has been repeatedly demonstrated in
both the research and clinical diagnostic settings, and has been upheld
over time. This classification was approved by the ClinGen General Gene
Curation Expert Panel on February 27, 2019.

**Literature Review:**

**OMIM**: https://www.omim.org/entry/101000

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1201/

Pathogenic variants. At least 400 NF2 pathogenic variants have been
described, with **missense, nonsense, and splicing variants and small deletions being the most common** (Table A, HGMD and Locus-Specific
Databases). A wide variety of pathogenic variants have been identified
in **all NF2 exons, except for the alternatively spliced exons.** 90% of
single-nucleotide variants are predicted to **truncate the protein** by
introduction of a **premature stop codon, a frameshift with premature
termination, or a splicing alteration**, supporting the view that loss
of the protein\'s normal function is necessary for the development of
tumors. C-to-T transitions in CGA codons causing pathogenic nonsense
variants are an especially frequent occurrence. Fewer than 10% of
detected pathogenic variants are **in-frame deletions and missense
variants**, which may indicate that alteration of particular functional
domains can abolish the NF2 **tumor suppressor activity** \[Baser et al
2006\].

Attempts to identify truncated protein product have been unsuccessful in the main, although the **non-truncated product from pathogenic missense variants may have partial function**. It is thought that nonsense-mediated decay may account for the lack of identifiable product from most variant types; however, this does not explain why **phenotypes are more severe for this type of variant than for whole-gene deletions**.

analyzed the **mosaic risk in de novo patients** with NF2 by age at the time of vestibular schwannoma diagnosis....... The risk of NF2 to an offspring of a patient presenting with bilateral vestibular schwannoma at less than 20 years of age was 29.3%, whereas the offspring risk for a
patient presenting with asymmetric disease after 40 years of age was
only 5.5%, as there is a 99% chance that they are mosaic.

***Evans and Wallace, 2009, PMID 19880713***

reported 5 NF2 patients with **constitutional rearrangements of
chromosome 22** and vestibular schwannomas, multiple intracranial
meningiomas, and spinal tumors.

***Tsilchorozidou et al , 2004, PMID 15235024***

Sixty-seven individuals (56.2%) from 41 of these kindreds revealed 36
different putative disease-causing mutations. These include 26 proposed
protein-truncating alterations (frameshift deletions/insertions and
nonsense mutations), 6 splice-site mutations, 2 missense mutations**, 1
base substitution in the 3\' UTR of the NF2 cDNA, and a single 3-bp
in-frame insertion.** Seventeen of these mutations are novel, whereas
the remaining 19 have been described previously in other NF2 individuals
or sporadic tumors.... Twenty-four of 28 patients with mutations that
cause premature truncation of the NF2 protein, schwannomin, present with
severe phenotypes. In contrast, all 16 cases from three families with
mutations that affect only a single amino acid have mild NF2. These data
provide conclusive evidence that a phenotype/genotype correlation exists
for certain NF2 mutations.

***Ruttledge, 1996, PMID: 8755919***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

Optional modifiers: mosaicism

**Disease associated variant consequences:**

*Dose change* -decreased gene product level

Altered protein product

**Narrative summary of molecular mechanisms:**

NF2 is a tumour suppression and mechanism of disease is principally loss of function, through protein truncation, haploinsufficiency and protein alteration. Classes of pathogenic variants include frameshift, deletions/insertions, nonsense, splice-site, missense mutations.
Less frequently, 3'UTR and inframe insertions have been reported.

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Inframe_insertion

Missense

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

(frameshift_variant predicted to escape NMD)

stop_lost

5_prime_UTR_variant

3_prime_UTR_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
