# **PKP2 -- Arrythmogenic Right Ventricular Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:9024

The relationship between PKP2 and arrhythmogenic right ventricular
cardiomyopathy (autosomal dominant) was evaluated using the ClinGen Clinical Validity Framework as of September, 2018. Variants in PKP2 were first reported in humans with this disease as early as 2004 (Gerull et al., PMID: 15489853). PKP2 is the major causative gene for ARVC and accounts for 34%-74% of cases (McNAlly et al., 2005; PMID: 20301310).
**There are over 250 PKP2 variants listed in ClinVar for ARVC (missense, nonsense, frameshift, complex rearrangements,** etc) (Novelli et al., 2018; PMID: 30619891). This gene-disease relationship is well-known and therefore a significant amount of case-level data, segregation data and experimental data is available in the literature, therefore the maximum
score for both genetic evidence and experimental evidence has been
reached. Note, this curation effort may not be exhaustive of all
literature related to this gene-disease relationship. This gene-disease relationship is supported by animal models, in vitro assays, and protein interactions. In summary, PKP2 is definitively associated with autosomal dominant ARVC. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen Arrythmogenic Right Ventricular Cardiomyopathy Gene Curation Expert Panel on October,
26, 2018 (SOP Version 6).
(https://www.clinicalgenome.org/curation-activities/gene-disease-validity/educational-and-training-materials/standard-operating-procedures/)

**Literature review:**

"...More than 170 pathogenic variants have been
described, including gross deletions \[[Li Mura et al
2013], [Roberts et al 2013] 

Multiple instances of [digenic] inheritance
have been identified with one [pathogenic variant] in *PKP2* and
a second in another desmosomal [gene]\[[Cox
et al 2011] [Bao et al 2013][Bhonsale et al 2015][Groeneweg et al
2015].

Abnormalities in plakophilin are thought to perturb intercellular connections and lead to arrhythmia.

The estimate of digenic/biallelic inheritance varies from 4% to 47% [[Xu et al 2010] [Bao et al 2013] [Rigato et al 2013] [Bhonsale et al 2015] [Groeneweg et al 2015] 
In most studies, digenic/biallelic carriers have a more severe
arrhythmia [phenotype].

[Rigato et al [2013\] examined a cohort of 113 individuals of whom 84% had a single desmosomal [gene]variant and 16% had [compound
heterozygous]or [digenic]inheritance. Over an observation period that averaged 39 years, 16% had major arrhythmic events. Specific risk factors for having events were male sex and having multiple genetic variants."

*Gene reviews
<https://www.ncbi.nlm.nih.gov/books/NBK1131/>*

"Direct sequencing of 5 AC genes in 71 unrelated patients with AC identified 10 different PKP2 mutations in 12 index patients. One patient, heterozygous for a PKP2 nonsense mutation, developed severe heart failure and underwent cardiac transplantation. Western blotting and immunohistochemistry of the explanted heart showed a significant decrease in PKP2 protein expression without detectable amounts of truncated PKP2 protein. Cultured keratinocytes of the patient showed a similar reduction in PKP2 protein expression. Nine additional PKP2 mutations were investigated in both cultured keratinocytes and endomyocardial biopsies from affected individuals. **It was evident that PKP2 mutations introducing a premature termination codon in the reading frame were associated with PKP2 transcript and protein levels reduced to ≈50%, whereas a missense variant did not seem to affect the amount of PKP2 protein."**

*Rasmussen TB et al 2014 PMID: 24704780*

"...both truncating and missense mutations in the desmosome genes PKP2 (encoding plakophilin 2), DSG2 (encoding desmoglein 2) and DSC2 (encoding desmocollin 2) have been identified in patients with ACM.  PKP2 is the most commonly affected gene in adult cohorts...
**The molecular links between desmosome mutations and the pathological hallmarks of ACM --- cardiomyocyte loss, fibrosis, adipogenesis, inflammation and arrhythmogenesis --- remain poorly defined. Probable pathogenic mechanisms include loss of mechanical integrity at cell--cell junctions, altered signalling pathways at intercalated discs, disruption of ion channels and gap junctions, and aberrant protein trafficking."**

*Austin KM et al. 2019. PMID: 31028357*

**The ARVC variant database can be found at https://molgenis136.gcc.rug.nl/**

In 2019, Ye et al reevaluated ARVC variants using large population databases. "...more than 10% of variants previously
reported to cause ARVC were found unlikely to be associated with highly penetrant monogenic forms of ARVC."

There were some variants that were found in population databases but were nevertheless associated with serious cardiac phenotypes suggesting they could be disease-modifiers of ARVC. Updated classification of variants by gene is available in the supplementary data

*Ye JZ et al 2019 PMID: 31402444*

"Of 501 arrhythmogenic right ventricular cardiomyopathy probands, 322 (64.3%) carried 327 desmosomal P/LP variants. Most variants (n=247, 75.6%, in 245 patients) were identified in more than one proband and, therefore, considered nonunique. For 212/327 variants (64.8%) genetic cascade screening was performed extensively enough to identify the parental origin of the P/LP variant. **Only 3 variants were de novo, 2 of which were whole gene deletions.** For 24 nonunique P/LP PKP2 variants, haplotyping was conducted in 183 available families. For all 24 variants, multiple seemingly unrelated families sharing identical haplotypes were identified, suggesting that these variants originate from common founders.

Deletions encompassing one or more exons were seen in 13 patients (4.0%), including 3 whole gene deletions (2 PKP2 and one DSP)."

*van Lint FHM et al 2019 PMID: 31386562*

**In house atlas of cardiac variants**

There was a significant excess of truncating variants in ARVC cases vs population controls. This was not the case for non-truncating variants.

**95/101 truncating**

Nonsense

Splice donor variants

Splice acceptor variants

**6/101 non-truncating**

Missense (all VUS)

*Walsh R et al 2017 PMID: 27532257*

If one focuses on P/LP variants, only 5 of them (6%) are missense,
whereas most are nonsense (25%), frameshift variants (46%), splicing alterations ±1 or ±2 (17%) or large deletions (6%). In contrast with this finding, among the ones classified as variant of unknown significance (VUS) 59% are missense, 12% are intron variants located relatively far from the donor or acceptor sites, 3% are synonymous variants and 25% are nucleotide substitutions in the 3′UTR. **This data shows that the majority of *Pkp2* variants (\~77%) associated with ACM in ClinVar and annotated as P/LP are radical alterations,** hence considered at high probability of causing a disruption of the protein and resulting in extensive transcriptional and posttranslational alterations, **while single amino acid changes are of more complex interpretation, especially in light of the known "signal-to-noise ratio"** 

*Novelli et al., 2018; PMID: 30619891*

There are rare reports of recessive mutations in PKP2 causing non-syndromic ARVC.

*Soveizi M et al 2017 PMID: 28523642*

#### **Pilot application of harmonised terms:**

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Allelic requirement:**

Monoallelic_aut

Optional modifiers:

Digenic (DSG2, DSP)

**Inheritance:**

Autosomal dominant

Optional modifiers:
Digenic (DSG2, DSP)
Incomplete penetrance

**Narrative summary of molecular mechanisms:**

Mechanism likely loss of function of *PKP2* due to **reduction/absence of gene product or altered gene product structure** causing impaired desomsome function. Most pathogenic variants are nonsense, frameshift, splicing alterations or large deletions. A minority are missense variants (5%). The majority of missense variants are VUS. Truncating mutations have been associated with a decreased protein expression suggesting that PKP2 haploinsufficiency contributes to the pathogenesis. Multiple instances of digenic inheritance have been identified with one pathogenic variant in *PKP2* and a second in another desmosomal gene. In most studies, digenic/biallelic carriers have a more severe arrhythmia phenotype. The common mode of inheritance is autosomal dominant with incomplete penetrance, however there are 2 reports of rare recessive mutations in PKP2 causing non-syndromic ARVC. "The molecular links between desmosome mutations and the pathological hallmarks of ACM --- cardiomyocyte loss, fibrosis, adipogenesis, inflammation and arrhythmogenesis --- remain poorly defined. Probable pathogenic mechanisms include loss of mechanical integrity at cell--cell junctions, altered signalling pathways at intercalated discs, disruption of ion channels and gap junctions, and aberrant protein trafficking." A 2019 study found that the majority of ARVC variants were nonunique (low rate of de novo variants) and suggested that most originate from common founders.

**Additional information related to ACMG evidence types**

As per Cardioclassifier:

**PVS1** - null variant in a gene where Loss of Function (LoF) is a known mechanism of disease
Consider activating for DSG2 and ARVC as there is a significant burden of truncating variants in cases against controls from analysis of 7,855 cardiomyopathy cases and 60,706 controls 

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework). Adjusted for ICC from original ACMG guidelines
0.1% (het) 
3.16% (hom) 

**For BS1 and PM2:**

Estimated Prevalence: 1/1000
Max Allelic Contribution: 0.092	
Maximum population AF: 0.000092

*Whiffin N et al 2018 PMID: 29369293*

**List variant classes in this gene proven to cause this disease**

- Splice region variant
- Spice acceptor variant
- Splice acceptor variant predicted to undergo NMD
- Splice donor variant
- Splice donor variant predicted to undergo NMD
- Frameshift variant
- Frameshift variant predicted to undergo NMD
- Stop gained
- Stop gained predicted to undergo NMD
- Missense

**Potential novel variant classes based on predicted functional
consequence:**

- Start lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
??
- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift variant predicted to escape NMD
- Stop gain variant predicted to escape NMD
- In frame insertion
- In frame deletion
- Stop lost ??