# **PLN -- Intrinsic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

https://search.clinicalgenome.org/kb/gene-validity/8772

The PLN gene is associated with hypertrophic cardiomyopathy, dilated cardiomyopathy, arrhythmogenic right ventricular cardiomyopathy (van der Zwaag, 2012 PMID: 22820313) and heart failure of unknown etiology. The typical inheritance for PLN related cardiomyopathy is autosomal dominant, albeit autosomal recessive inheritance has been noted and appears to follow a dosage effect or semidominance, as loss of both alleles results in an earlier and more severe phenotypic presentation (Haghighi, 2003 PMID: 12639993). PLN is encoded by one exon, and missense, nonsense, and frameshift mutations in the coding exon and the promoter region have been reported. Given that PLN only has one exon, many of the mutations (even LOF) result in a protein product. PLN encodes the 52 amino acid protein, phospholamban, that functions to regulate SERCA2 function in the sarcoplasmic reticulum. Phospholamban exists in both monomeric and homopentameric forms. The monomeric form is thought to inhibit SERCA2 activity. PLN-mediated SERCA2 inhibition is released upon phosphorylation of monomeric PLN by either PKA or CAMKII, and thus stabilization of the pentameric form (reviewed in Haghighi, 2014 PMID: 25451386, Young, 2015 PMID: 25563649). While the distinct genetic mechanism of PLN-mediated cardiomyopathy is unclear, the overall disease mechanism for PLN associated cardiomyopathy is dysregulation of SERCA2 function, Ca2+ handling, and disrupted relaxation and contraction of the heart. Multiple cases of PLN-mediated cardiomyopathy are reported in the literature, allowing and extending beyond the maximum score for genetic evidence (12 pts). This gene-disease association is supported by the function of the gene product, alteration of normal function in nonpatient cells expressing patient-derived mutant PLN, and animal models. In summary, PLN is definitively associated with intrinsic cardiomyopathy. This association has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the Hypertrophic Cardiomyopathy Gene Curation committee on September 19, 2017.

Lumping and Splitting: Per criteria outlined by the ClinGen Lumping and Splitting Working Group, we found no difference in molecular mechanism(s) underlying the disease entities: (1) Cardiomyopathy, dilated, 1P (MIM: 609909) and (2) Cardiomyopathy, hypertrophic, 18 (MIM:613874). Evidence suggests that the mechanism of the disease is impaired SERCA2 regulation of Ca2+ handling for all conditions associated with PLN. Furthermore, a progressive cardiomyopathy beginning with hypertrophic and leading to dilated has been observed in a proband (Haghighi, 2003 PMID: 12639993). Both interfamilial and intrafamilial variability were observed between PLN variants (Haghighi, 2003 PMID: 12639993; van der Zwaag, 2012 PMID: 22820313). For clinical management, no striking differences should occur, as all of the phenotypes and conditions associated with PLN are of cardiovascular nature, and individuals should be monitored appropriately. Therefore, all of the disease entities have been lumped into one disease entity, Intrinsic cardiomyopathy.

**Literature review:**

"Schmitt et al. (2003) sequenced the PLN gene in 20 unrelated individuals with inherited dilated cardiomyopathy and heart failure (see CMD1P, 609909). In 1 sample, **an arginine-to-cysteine substitution at codon 9 in the cytosolic PLN domain was identified and segregated with disease in that 4-generation family** (R9C;172405.0001). Affected individuals had increased chamber dimensions and decreased contractile function at age 20 to 30 years, with progression to heart failure within 5 to 10 years after symptom onset. Congestive heart failure was severe in 12 individuals, necessitating cardiac transplantation in 4. The average age at death of affected individuals was 25.1 +/- 12.7 years. 

In 2 unrelated families with CMD1P (609909), Haghighi et al. (2003) identified **a truncating mutation in the PLN gene, leu39 to ter (L39X; 172405.0002). The 2 homozygous individuals developed dilated cardiomyopathy and heart failure requiring cardiac transplantation at ages 16 and 27 years, respectively; 11 heterozygous individuals exhibited variable clinical findings, indicating incomplete penetrance of the cardiomyopathy phenotype.** Haghighi et al. (2003) concluded that in contrast to mice in which Pln deficiency enhances myocardial inotropy and lusitropy without adverse effects, PLN is essential for cardiac health in humans, and its absence results in lethal heart failure. 

In affected members of a 7-generation family with CMD, Haghighi et al. (2006) identified heterozygosity for **a 3-bp deletion** in the PLN gene (172405.0003). 

Haghighi et al. (2008) analyzed the PLN gene in 381 CMD patients and 296 controls with no known cardiomyopathy history and **identified a heterozygous -36A-C variant in the 5-prime untranslated region (172405.0006) in 22 CMD patients and 1 control.** Functional analysis demonstrated that the -36A-C variant increased PLN activity by 24% compared to wildtype and that this alteration in the steroid receptor sequence for the glucocorticoid nuclear receptor/transcription factor resulted in enhanced binding. 

Minamisawa et al. (2003) analyzed the candidate gene phospholamban (PLN) in 87 patients with hypertrophic cardiomyopathy (see CMH18; 613874), 10 with dilated cardiomyopathy, and 2 patients with restricted cardiomyopathy (RCM; see 115210). In the proband of a 2-generation family with CMH, **Minamisawa et al. (2003) identified heterozygosity for a mutation in the promoter region (172405.0004) that increased transcriptional activity 1.5-fold compared to wildtype and was not found in 296 Japanese controls.** No PLN mutations were identified in the remaining 98 cardiomyopathy patients..."

*Omim https://www.omim.org/entry/172405*

"We sequenced known and putative HCM genes in a new large prospective HCM cohort (n = 804) and analysed data alongside the largest published series of clinically genotyped HCM patients (n = 6179), previously published HCM cohorts and reference population samples from the exome aggregation consortium (ExAC, n = 60 706) to assess variation in 31 genes implicated in HCM. **PLN (calcium signalling) had an excess of 0.28% with one variant, p.L39X, significantly enriched in cases (4/5435 cases versus 1/60671 ExAC samples, P = 2 × 10−4).**" 

*Walsh et al 2017 PMID 28082330*

"Three hundred and eighty-seven consecutive unrelated patients with HCM were screened for genetic variants in the 5 most frequent genes (MYBPC3, MYH7, TNNT2, TNNI3 and TPM1) using Sanger sequencing (N = 84) or NGS (N = 303). In the NGS cohort we analyzed 20 additional minor or candidate genes, and applied a proprietary bioinformatics algorithm for detecting CNVs. In the NGS cohort, **4 patients (1.3%) had pathogenic CNVs: 2 deletions in MYBPC3 and 2 deletions involving the complete coding region of PLN.**"

*Mademont-Soler I et al 2017 PMID: 28771489*

"...**promoter and coding region variants of PLN have been associated with DCM/heart failure and HCM**. For example, a C to G conversion at position −42 (C>G −42) promoter variant has been described in one HCM case in a study that included 186 HCM and DCM patients21. This variant, found in a female, diagnosed with HCM at 67 years of age with atrial fibrillation, had reduced penetrance in a small familial pedigree. A second promoter variant, an A>G −77 mutation, was identified in one out of 87 HCM patients22."

*Landstrom AP et al 2011 PMID: 21167350*

"Mutations in PLN cause cardiomyopathy in a distinct way, as they may not directly influence the structural or functional characteristics of cardiomyocytes, but disrupt the calcium homeostasis inside cardiomyocytes and interrupt the rhythm of myocardial contraction. The known mutations in human include R14del, R9C, R9L, R9H, Leu-39stop, and R25C. Each mutation has a unique mechanism to eventually induce cardiomyopathy. In brief, the presumed mechanisms are: (1) Completely or partially losing the function to inhibit SERCA; (2) Interfering the normal PLN to inhibit SERCA in a dominant negative way. (3) Failing to be phosphorylated and regulated by PKA, or further, disabling the normal function of PKA6,10,11.

R14del is a well-known PLN mutation in Dutch people, with 10–15% of both dilated cardiomyopathy and arrhythmogenic cardiomyopathy patients are claimed to be caused by PLN-R14del..."

*Jiang X et al 2020 PMID: 33020536*

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

**Disease associated variant consequences:**

Decreased gene product level
Absent gene product
Altered gene product structure

**Narrative summary of molecular mechanisms:**

PLN functions to regulate SERCA2 function in the sarcoplasmic reticulum. The mechanism is not fully understood but it is likely loss of function of PLN through either decreased/absent gene product or altered gene product structure. Jiang et al note "presumed mechanisms are: (1) Completely or partially losing the function to inhibit SERCA; (2) Interfering the normal PLN to inhibit SERCA in a dominant negative way. (3) Failing to be phosphorylated and regulated by PKA, or further, disabling the normal function of PKA". They also note that PLN gain of function mutations have the potential to induce disorder of intracellular calcium. 

PLN is encoded by one exon (52 amino acids), and missense, nonsense, and frameshift mutations in the coding exon and the promoter region have been reported. In addition deletions of the whole coding region have been reported. In the Netherlands there is a founder mutation R14del. Up to 10–15% of both dilated cardiomyopathy and arrhythmogenic cardiomyopathy patients are claimed to be caused by PLN-R14del. As per ClinGen: "The typical inheritance for PLN related cardiomyopathy is autosomal dominant, albeit autosomal recessive inheritance has been noted and appears to follow a dosage effect or semidominance, as loss of both alleles results in an earlier and more severe phenotypic presentation. Given that PLN only has one exon, many of the mutations (even LOF) result in a protein product."

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame deletion
- Stop_gain
- Frameshift
- Promoter (regulatory_region_variant)
- Whole exon deletion

**Potential novel variant classes based on predicted functional
consequence**

- Frameshift predicted to escape NMD
- stop_gained predicted to escape NMD
- Frameshift predicted to undergo NMD
- stop_gained predicted to undergo NMD
- stop_lost
- inframe_insertion
- start lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
- ?Structural variants/transcript ablation


