# **PMS2 -- Lynch Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:9122

**ClinGen Evidence for Haploinsufficiency**

PMID 18602922 - Authors discuss results of 99 probands who had a Lynch syndrome associated tumor and absence of PMS2 protein by IHC. Total of 55 patients found to have deleterious mutations in PMS2. Most notable was an insertion/deletion mutation in exon 7. Several patients (10 patients) in this study showed deletions of one or more exons , two patients (Patient 28, 29) showed complete gene deletions.

PMID 23837913 - Authors described the use of long range PCR and MLPA analysis in the identification of deletions within the PMS2 gene. Six patients with suspected Lynch syndrome were identified to have mutations (5 with deletions of one or more exons with loss of PMS2 protein expression by IHC and 1 frameshift mutation). In addition 4 novel VOUS were also identified in this study.

PMID 22585707 - 1) a large (?125 kb) deletion containing the entire PMS2 gene, three other genes (ANKRD61, AIMP2 [JTV1], and EIF2AK1), and a portion of the coding region of RSPH10B. 2) a genomic deletion involving PMS2 exon 8. Special note from this article: Because of the high prevalence of nondeleterious hybrid PMS2 and PMS2CL alleles, the distribution of PMS2 and PMS2CL-specific sequences downstream of exon 12 displays high interindividual variability (Hayward et al., 2007;Ganster et al., 2010; van der Klift et al., 2010). Randomly chosen reference samples of DNA are thus likely to differ widely in terms of the distribution of gene- and pseudogene-derived sequences in this region. This is reflected by high standard deviations for the reference DNA signals generated with all the paralog-discriminating probes located downstream of exon 12 (Supporting Information Fig. S1). It is important to note that an unequal distribution of gene-derived and pseudogene-derived sequences in the reference DNA set will reduce the accuracy of copy number assessments at these loci in patient DNA samples. For this reason, reference DNAs must harbor two PMS2-specific copies and two PMS2CL-specific copies of each sequence bound by paralog-discriminating probes for exons 11-15.

**Literature review:**

PMS2 comprises 15 exons encoding a protein of 862 amino acids. Multiple pseudogenes have been identified at 7p22, 7p12-13, 7q11, and 7q22 [Nicolaides et al 1995]. For a detailed summary of gene and protein information, see Table A, Gene.

Pathogenic variants. Germline pathogenic variants in PMS2 are rare [Hendriks et al 2006]. Single-nucleotide variants and large gene rearrangements have been reported. **Studies that have included large deletion testing have found that up to 20% of pathogenic variants may be large deletions. Large deletion testing of PMS2 is technically difficult due to the numerous pseudogenes,** and it presents significant challenges to laboratories trying to provide comprehensive large deletion testing for the whole gene. The currently available MLPA (multiplex ligation-dependent probe amplification) kit can detect deletions but does not clarify whether the deletion may be in one of the pseudogenes. Testing in coordination with a panel of reference samples can help determine whether deletions are clinically significant [Vaughn et al 2011].

PMS2 acts in a recessive manner at the cellular level where there is an absence of functional PMS2 protein in the tumor cells. This results from inactivation of both PMS2 alleles in the tumor, which often occurs by the mechanism of loss of heterozygosity.

Heterozygosity for a PMS2 pathogenic variant is associated with the lowest risk (25%-32% risk) for any Lynch syndrome-related cancer [Senter et al 2008]. However, while the overall risk of CRC is lower, age of onset may still be early. A review of 234 PMS2 pathogenic variant carriers found that 8% were diagnosed before age 30 [Goodenberger et al 2016].

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1211/*

Types of variant in PMS2:

Missense 62%
Nonsense or frameshift 24%
In-frame 1% (includes deletions, insertions or indels which do not affect the reading frame)
Splice  3%
Large rearrangement 10%

*Data from the InSiGHT database presented in Tamura K et al 2019 PMID: 31273487*

Of the approximately 3,300 unique MMR gene variants listed in the International Society for Gastrointestinal Hereditary Tumors (InSiGHT) database, only 9% are reported in PMS2, compared to 39% in MLH1, 33% in MSH2 and approximately 19% reported in MSH6 (http://www.insight-database.org/genes). 

Analysis of the PMS2 gene has been complicated by the presence of a large family of pseudogenes that are highly homologous to PMS2. Fourteen pseudogenes (ψ1 – 14) overlap with some or all of exons 1 – 5 of PMS2 and vary in length (Figure 1A) whereas the PMS2CL pseudogene (formerly ψ0) possesses high sequence similarity to exons 9 and 11–15

*Blount J et al 2018 PMID: 29286535*

Variant types taken from InSiGHT database
*Plazzer JP et al 2013 PMID: 23443670*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

A heterodimer of MLH1 and PMS2 coordinates the interplay between the mismatch recognition complex and other proteins necessary for mismatch repair. In the absence of PMS2, MLH1 can pair with PMS1. This may partially explain the somewhat attenuated Lynch phenotype attributed to PMS2 mutations.
Single-nucleotide variants and large gene rearrangements (up to 20% in some studies) have been reported. Large deletions are difficult to identify due to the presence of a large family of pseudogenes highly homologous to PMS2.
Heterozygosity for a PMS2 pathogenic variant is associated with the lowest risk (25%-32% risk) for any Lynch syndrome-related cancer [Senter et al 2008]. However, while the overall risk of CRC is lower, age of onset may still be early. A review of 234 PMS2 pathogenic variant carriers found that 8% were diagnosed before age 30 [Goodenberger et al 2016].

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions
- In frame insertions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]

