**PTEN - PTEN hamartoma tumor syndrome (MIM 153480)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:9588

no single gene duplications of the entire PTEN locus have been reported.

**Literature Review:**

**OMIM:** https://www.omim.org/entry/158350

The chromosomal region containing the Cowden syndrome locus was known to
contain the **tumor suppressor gene PTEN**, which had been found to be
mutated in sporadic brain, breast, and prostate cancer. Liaw et al.
(1997) found germline mutations in the PTEN gene in 4 of 5 families with
Cowden syndrome. **Missense (601728.0001) and nonsense mutations were
predicted to disrupt the protein tyrosine/dual-specificity phosphatase
domai**n of the protein. All affected individuals of the 5 families
studied manifested trichilemmomas, regardless of whether their mutation
was a missense or nonsense mutation. Nonsense mutations were associated
with macrocephaly in 2 families. In 1 of these families, a **premature
stop codon at position** 157 (601728.0003) was also associated with
Lhermitte-Duclos disease (LDD), manifested by ataxia and dysplastic
cerebellar gangliocytomatosis. In the other family, a stop codon at
position 233 (601728.0002) was not associated with cerebellar
manifestation. Liaw et al. (1997) speculated that the larger N-terminal
truncation may be responsible for the more severe LDD phenotype.

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1488/

Pathogenic variants. Germline pathogenic variants have been found
throughout PTEN (with the exception of exon 9) and include **missense,
nonsense, and splice-site variants, small deletions, insertions, and
large deletions.** More than 150 unique pathogenic variants are
currently listed in the Human Gene Mutation Database (see Table A).
**Nearly 40% of pathogenic variants are found in exon 5, which encodes
the phosphate core motif** \[Eng 2003\]. Most pathogenic variants are
unique, although a number of recurrent pathogenic variants have been
reported, particularly those in Table 3.

Abnormal gene product. The **majority (76%) of germline pathogenic
variants in PTEN predict either truncated PTEN protein, lack of protein
(haploinsufficiency), or dysfunctional protein**. Many **missense
variants are functionally null and several act as dominant negatives**
\[Weng et al 2001a, Weng et al 2001b\]. When PTEN is absent, decreased,
or dysfunctional, phosphorylation of AKT1 is uninhibited, leading to the
inability to activate cell cycle arrest and/or to undergo apoptosis. In
addition, through lack of protein phosphatase activity, the
mitogen-activated protein kinase (MAPK) pathway is dysregulated, leading
to abnormal cell survival \[Eng 2003\].

...report a study of 3042 individuals who met relaxed criteria for
Cowden Syndrome and report 290 (9.5%) with pathogenic mutations in PTEN.
The majority were **presumed loss of function mutations - 92 (32%)
nonsense mutations, 42 (14%) small deletions, 24 (8%) small insertions,
3 (1%) indels, 8 (3%) large deletions, 19 (7%) splice-site donor
mutations, 9 (3%) splice-site acceptor mutations.**

***Tan et al, PMID 21194675***

***...***report on 481 patients with pathogenic germline PTEN variants.
This cohort includes those reported in Tan et al (PMID 21194675). The
mutation spectrum includes **242 truncating mutations, as well as 12
exon-level deletions.**

***Yahia et al, PMID 32003824***

Supplementary information 16 patients, no genotype:phenotype
correlation. **Missense, nonsense, splice donor, 2kb upstream variant
(-1084), deletion, insertion**

***Ciaccio et al, 2019, PMID: 30528446***

The mutational spectra of CS and BRRS overlap, with many of the
mutations occurring in exons 5, 7, and 8...... two-thirds of these
mutations were found in exons 5, 7,and 8 \[Marsh et al., 1998b\]. About
40% of all CS germline mutations are located in exon 5.

..Tabulated variant in **5'UTR, stop gain, frameshift, splice
acceptor/donor (truncating), missense null, insertions, deletions
(mostly out of frame), -8 splice region**

***Eng, 2003,PMID: 12938083***

... studied 34 different **germline PTEN intronic variants from 61 CS
patients**, characterized their PTEN mRNA processing, and analyzed PTEN
expression and downstream readouts of P-AKT and P-ERK1/2. While we found
that many **mutations near splice junctions result in exon skipping, we
also identified the presence of cryptic splicing that resulted in
premature termination or a shift in isoform usage.** PTEN protein
expression is significantly lower in the group with splicing changes
while P-AKT, but not P-ERK1/2, is significantly increased. Our
observations of these PTEN intronic variants should contribute to the
determination of pathogenicity of PTEN intronic variants and aid in
genetic counseling.

***Chen et al, 2017, PMID: 28677221***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly loss of function and the majority of variants
predict either truncated PTEN protein, lack of protein
(haploinsufficiency), or dysfunctional protein. Variant classes include
missense (many functionally null), nonsense, and splice-site variants,
small deletions, insertions, and large deletions. Nearly 40% of
pathogenic variants are found in exon 5 (encoding the phosphate core
motif) and several are recurrent. No pathogenic mutations have been
identified in exon 9. Variants in the 5'UTR and deeply intronic (exon
skipping and cryptic splice sites) have been reported.

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

(frameshift_variant predicted to escape NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Inframe_insertion

Missense

5_prime_UTR_variant

intron_variant

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

stop_lost

3_prime_UTR_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
