---
output:
  word_document: default
  html_document: default
---
**RB1 -- Retinoblastoma (MIM 180200)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:9884

Retinoblastoma (RB) is an embryonic malignant neoplasm of retinal
origin. It almost always presents in early childhood and is often
bilateral. The retinoblastoma gene (RB1) was the first **tumor
suppressor gene** identified based on location in the region of 13q14
germline deletion in some patients with retinoblastoma (PMID:
3480530). Oncogenic **point mutations** in RB1 were first reported by
Yandell DW, et al., 1989 (PMID: 2594029). RB1 encodes a negative
regulator of the cell cycle through its ability to bind the
transcription factor E2F and repress transcription of genes required for
S phase (PMID: 10647931). There is abundant evidence published
associating the RB1 gene with retinoblastoma. Multiple case level
studies (PMID: 2594029, 25928201) and a case control study (PMID: 28193182) have been performed with retinoblastoma patients that have
variants in the RB1 gene. The variants include **single amino acid
changes, the whole gene deletion (about 3-5%) and nonsense or frameshift
LOF variants**. Many de novo variants were reported. A significant
amount of case-level data is available, but the maximum points for
genetic evidence has been reached (12 points). A zebrafish model of RB1
has been established to show development of retinoblastoma consistent
with the disease phenotype. Tumor formation was observed when the
IRBP-Cre:Rb1-/- mice were crossed with Tp53 mice to heterozygosity. In
summary, RB1 is definitively associated with autosomal dominant
retinoblastoma. This has been repeatedly demonstrated in both the
research and clinical diagnostic settings, and has been upheld over
time.

**Literature Review:**

**OMIM:** https://www.omim.org/entry/180200

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1452/

Pathogenic variants. More than 2,500 nucleotide variants have been
observed in white blood cell DNA of individuals with retinoblastoma or
in tumors; more than 1,700 are archived (see Table A, Locus Specific).
The majority of RB1 pathogenic variants result in a **premature
termination codon, usually through single-base substitutions, frameshift
variants, or out-of-frame exon skipping caused by splice site
variants**. Pathogenic variants have been found scattered throughout
**exon 1 to exon 25 of RB1 and its promoter region**. In a single
family, a possible pathogenic variant in exon 27 was identified [Mitter et al 2009]. Recurrent **pathogenic variants are observed at methylated
CpG dinucleotides that are part of CGA codons or the splice donor site
of intron 12**. Other important types of pathogenic variants are
**complex rearrangements and deletions** [Albrecht et al 2005, Rushlow et al 2009, Castéra et al 2013].

Abnormal gene product. Pathogenic variants in RB1 lead to the expression
of proteins that have **lost cell cycle-regulating functions**.
Retention of partial activity has been observed in proteins resulting
from **pathogenic variants that are associated with low-penetrance**
retinoblastoma

......analyzed 19 patients with RB. RB1 mutations were identified in 13
tumors, including the following germline mutations: **55 bp duplication
within exon 10 (truncated protein) and a 10 bp deletion in exon 18
(truncated protein).**

***Dunn et al, 1989, PMID 2601691***

....analyzed tumors from 7 patients with RB. RB1 mutations were
identified in all tumors, including a de novo germline mutation
(ARG445TER) -- stop gain.

***Yandell, 1989, PMID 2594029***

...reported on 119 patients with RB and found RB1 mutations in 99
patients (83%). The mutation spectrum included **42% base substitutions,
26% small length alterations, and 15% large deletions.**

***Lohmann, 1996, PMID 8651278***

....intronic variants affecting less conserved splice motifs require
additional analysis to ascertain whether splicing is altered. Although
the frequency of these variations is low, their impact on genetic
counselling is high, since they are **usually associated with low
penetrance phenotypes** and unaffected carriers. In this work, we used
minigene assays to study infrequent germline intronic variations for
which functional data were not available. Using this approach, the
aberrant splicing and the resulting oncogenic nature of **three intronic
RB1 mutations was established (c.501-15T\>G, c.719-9C\>G,
c.2326-8T\>A).**

***Gamez-Pozo, 2007, PMID: 18000883***

....**Mosaicism was evident in 5.5% of bilateral probands (23 of 421),
in 3.8% of unilateral probands (22 of 572), and in one unaffected mother
of a unilateral proband.** Half of the mosaic mutations were only
detectable by AS-PCR for the 11 recurrent CGA\>TGA mutations, and not by
standard sequencing. This suggests that significant numbers of low-level
mosaics with other classes of RB1 mutations remain unidentified by
current technology.

***Rushlow, 2009, PMID: 19280657***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

Optional modifiers: incomplete penetrance

Optional modifiers: mosaicism

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

RB1 is a tumour suppressor gene. Mechanism of disease is loss of
function secondary to a truncated RB1 protein or lack of protein
(haploinsufficiency). Most variants are missense, frameshift variants or
splice site throughout exons 1-25 and the promotor region. Large complex
rearrangements and deletions are noted and recurrent pathogenic variants
are observed, including the splice donor site of intron 12. Mosaicism was evident in 5.5% of bilateral probands and
in 3.8% of unilateral probands. This suggests that significant numbers of low-level mosaics with other classes of RB1 mutations remain unidentified by current technology.

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

(frameshift_variant predicted to escape NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Inframe_insertion

Missense

intron_variant

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

stop_lost

5_prime_UTR_variant

3_prime_UTR_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
