# **RB1 -- Retinoblastoma**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:9884

Retinoblastoma (RB) is an embryonic malignant neoplasm of retinal origin. It almost always presents in early childhood and is often bilateral. The retinoblastoma gene (RB1) was the first tumor suppressor gene identified based on location in the region of 13q14 germline deletion in some patients with retinoblastoma (PMID: 3480530). Oncogenic point mutations in RB1 were first reported by Yandell DW, et al., 1989 (PMID: 2594029). RB1 encodes a negative regulator of the cell cycle through its ability to bind the transcription factor E2F and repress transcription of genes required for S phase (PMID: 10647931). There is abundant evidence published associating the RB1 gene with retinoblastoma. Multiple case level studies (PMID: 2594029, 25928201) and a case control study (PMID: 28193182) have been performed with retinoblastoma patients that have variants in the RB1 gene. The variants include single amino acid changes, the whole gene deletion (about 3-5%) and nonsense or frameshift LOF variants. Many de novo variants were reported. A significant amount of case-level data is available, but the maximum points for genetic evidence has been reached (12 points). A zebrafish model of RB1 has been established to show development of retinoblastoma consistent with the disease phenotype. Tumor formation was observed when the IRBP-Cre:Rb1-/- mice were crossed with Tp53 mice to heterozygosity. In summary, RB1 is definitively associated with autosomal dominant retinoblastoma. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**ClinGen Evidence for Haploinsufficiency**


**Literature review:**

More than 2,500 nucleotide variants have been observed in white blood cell DNA of individuals with retinoblastoma or in tumors; more than 1,700 are archived (see Table A, Locus Specific). **The majority of RB1 pathogenic variants result in a premature termination codon, usually through single-base substitutions, frameshift variants, or out-of-frame exon skipping caused by splice site variants.** Pathogenic variants have been found scattered throughout exon 1 to exon 25 of RB1 and its promoter region. In a single family, a possible pathogenic variant in exon 27 was identified [Mitter et al 2009]. Recurrent pathogenic variants are observed at methylated CpG dinucleotides that are part of CGA codons or the splice donor site of intron 12. Other important types of pathogenic variants are complex rearrangements and deletions [Albrecht et al 2005, Rushlow et al 2009, Castéra et al 2013].

Pathogenic variants in RB1 lead to the expression of proteins that have lost cell cycle-regulating functions. Retention of partial activity has been observed in proteins resulting from pathogenic variants that are associated with low-penetrance retinoblastoma [Bremner et al 1997, Otterson et al 1997, Sánchez-Sánchez et al 2007].

In the majority of families with heritable retinoblastoma, all members who have inherited the germline pathogenic variant develop multiple tumors in both eyes. It is not unusual to find, however, that the founder (i.e., the first person in the family to have retinoblastoma) has only unilateral retinoblastoma. Most of these families segregate RB1 null alleles that are altered by frameshift or nonsense variants. **With few exceptions, RB1 null alleles show nearly complete penetrance (>99%)** [Taylor et al 2007, Dommering et al 2014, Frenkel et al 2016].

**Fewer than 10% of families show a "low-penetrance" phenotype with reduced expressivity** (i.e., increased prevalence of unilateral retinoblastoma) and incomplete penetrance (i.e., ≤25%). This low-penetrance phenotype is usually associated with RB1 in-frame, missense, or distinct splice site variants, certain indel variants in exon 1, or pathogenic variants in the promoter region.

A third category of families shows differential penetrance depending on the parental origin of the pathogenic allele (parent-of-origin effect) [Klutz et al 2002, Eloy et al 2016, Imperatore et al 2018].

Cytogenetically visible deletions involving 13q14 that also result in deletions of additional genes in the same chromosome region as RB1 may cause developmental delay [Castéra et al 2013] and mild-to-moderate facial dysmorphism. As sizeable deletions of 13q14 show reduced expressivity, a considerable proportion of individuals with such deletions show unilateral retinoblastoma only; some of these children do not develop any tumors [Mitter et al 2011]. Contiguous loss of MED4, which is located centromeric to RB1, explains reduced expressivity in individuals with large deletions that include both RB1 and MED4 [Dehainault et al 2014].

**Gene reviews**
*https://www.ncbi.nlm.nih.gov/books/NBK1452/#retinoblastoma*

A searchable database (RBGMdb) has been constructed with 932 published RB1 mutations. The spectrum of these mutations has been analyzed with the following results: 1) the retinoblastoma protein is frequently inactivated by deletions and nonsense mutations while missense mutations are the main inactivating event in most genetic diseases. 2) Near 40% of RB1 gene mutations are recurrent and gather in sixteen hot points, including twelve nonsense, two missense and three splicing mutations. The remainder mutations are scattered along RB1, being most frequent in exons 9, 10, 14, 17, 18, 20, and 23. 3) The analysis of RB1 mutations by country of origin of the patients identifies two groups in which the incidence of nonsense and splicing mutations show differences extremely significant, and suggest the involvement of predisposing ethnic backgrounds. 4) A significant association between late age at diagnosis and splicing mutations in bilateral retinoblastoma patients suggests the occurrence of a delayed-onset genotype. 5) Most of the reported mutations in low-penetrance families fall in three groups: a) Mutations in regulatory sequences at the promoter resulting in low expression of a normal Rb; b) Missense and in-frame deletions affecting non-essential sequence motifs which result in a partial inactivation of Rb functions; c) Splicing mutations leading to the reduction of normal mRNA splicing or to alternative splicing involving either true oncogenic or defective (weak) alleles.

*Valverde JR et al 2005 PMID:16269091*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism appears to be loss of function of TSC2 leading to **decreased/absent or altered gene product** which results in uncontrolled cell growth and proliferation. Missense variants account for approximately 26% of all TSC2 pathogenic variants with approximately 50% concentrated in the carboxy domain. Variants are distributed throughout the coding regions of TSC2, except for the alternatively spliced exons (25 and 31). Approximately 33% of TSC2 pathogenic variants are located in exons 32-41 (and associated splice sites) that encode the carboxy domain of tuberin consisting of several important functional motifs (e.g., GAP domain, estrogen receptor- and calmodulin-binding domains, and multiple signal pathway kinase targets).
Penetrance is nearly 100% but there is variable expressivity.


**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame deletion
- In frame duplication
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[uORF\]
- Frameshift \[uORF\]
- Stop gained \[uORF\]
