# **SDHC -- Paraganglioma**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:10682

**ClinGen Evidence for Haploinsufficiency**

PMID 19546167 - Bayley et al (2009) describe patients with multi-exon deletions with paraganglioma, including one individual with a deletion of exons 5 and 6 of the SDHC.

PMID 15342702 - Baysal et al. (2004) described a multiplex "family with head and neck paragangliomas and...an 8.37 kb SDHC deletion, which spans two AluY elements and removes exon 6." They identified the same deletion in a reportedly unrelated sporadic case; further investigation identified "allele sharing with the familial cases at seven polymorphic markers near SDHC, suggesting a common ancestral origin."

PMID 17667967 - Pasini et al. (2008) described germline SDHC variants in individuals with paraganglioma and gastrointestinal stromal tumors. The authors identified one novel nonsense variant (c.43C>T; p. Arg15X) and one splice-site substitution (IVS5+1G>A or c.405+1G>A), which resulted in "exon 5 [being] spliced out, causing a frameshift and a stop codon in the 3? untranslated region of the gene."

**Literature review:**

SDHA, SDHB, SDHC, and SDHD are four nuclear genes responsible for hereditary PGL/PCC syndromes. They encode the four subunits of the mitochondrial enzyme succinate dehydrogenase (SDH).

SDHA, SDHAF2, SDHB, SDHC, and SDHD are tumor suppressor genes. Somatic second-hit variants in tumors include gross chromosomal rearrangements, recombination, single-nucleotide variants, or epigenetic changes that result in allelic inactivation.

**Nonsense, missense, splice site, regulatory, and whole-exon-deletion SDHC pathogenic variants have been reported in individuals** and pedigrees affected with hereditary PGL/PCC syndromes.

SDHC encodes the succinate dehydrogenase cytochrome b560 subunit, a 169-amino-acid protein.
Pathogenic variants in SDHC result in reduced or absent succinate dehydrogenase function because of loss or dysfunction of the affected subunit or failure of the SDH heterotetramer to assemble.

**Germline SDHC pathogenic variants appear to be primarily (but not exclusively) associated with HNPGL. However, up to 10% of SDHC-related tumors are observed in the thoracic cavity** [Peczkowska et al 2008, Else et al 2014].

The majority of GISTs associated with PGL (Carney Stratakis syndrome; OMIM 606864) occur in individuals with a germline pathogenic variant in SDHA or SDHC.

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1548/*

Three hundred and forty-four probands and 436 of their relatives harboured an intragenic mutation in SDHB/SDHC/SDHD
(table 1). Forty-five intragenic mutations in 134 probands were
reported previously.10 **The ratio of mutation classes among
probands was similar to that reported previously10 (44%
missense, 15% nonsense, 13% splice-site, 15% frameshift,
0.5% inframe deletions and 12% large CNAs).**

**We found 83% (5/6) of SDHC CNAs were exon 6 deletions,** and all three SDHD CNAs in our series were exon 4 deletions.

In 34 SDHC mutation carriers with detailed clinical information, 19 were clinically affected (15 HNPGL, 3 PPGL and 1 had both HNPGL and PPGL). One patient with HNPGL had local spread and malignant features. Age-related risks of symptomatic PPGL/HNPGL in SDHC were similar to that of SDHD. Compared with SDHB mutation carriers, SDHC carriers had a lower risk of PPGL (P=0.02 and P=0.06 before and after Bonferroni correction) and a higher risk of HNPGL (P<0.001 after Bonferroni correction for three comparisons, log-rank test) (figure 1).

*Andrews KA et al 2018 PMID: 29386252*

The spectrum of mutations in our series encompassed
partial deletions, missense, and nonsense mutations. We
report two new mutations not previously described in patients with SDHC-associated PGL: a novel missense mutation (c.214CG; p.Arg72Gly), and a splice site mutation (c.4051 GC). Both mutations affect highly
conserved sites, and mutations of the same codon and
splice junction have been described previously (10, 11).
The c.4051GC mutation has been described in a patient with a gastrointestinal stromal tumor (GIST) (11).

**SDHC mutations appear to be associated with a lower disease penetrance of PGL/PC when
compared with SDHB or SDHD mutations.** Mediastinal
and multiple PGLs might be more common in PGL3 than
was appreciated in prior studies. Patients with SDHC mutations should undergo screening with biochemical evaluation and imaging. Furthermore, up to one in 10 patients
with PGL3 may develop a mediastinal PGL, and imaging
of this area should be considered for surveillance in individuals with SDHC mutations.

*Else T et al 2014 PMID: 24758179*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Deletions and loss of function mutations in SDHC are associated with paraganglioms. Pathogenic variants in SDHC result in reduced or absent succinate dehydrogenase function because of loss or dysfunction of the affected subunit or failure of the SDH heterotetramer to assemble. Details of mechanisms by which SDH mutations activate hypoxic pathways and trigger subsequent neoplastic transformation remain poorly understood. Nonsense, missense, splice site variants, intragenic insertions and deletions, and a whole-gene deletion have been reported in SDHC. Approximately 10% of pathogenic variants appear to be larger deletions. One study found 5/6 copy number variants were exon 6 deletions. SDHC mutations appear to be associated with a lower disease penetrance of PGL/PC when compared with SDHB or SDHD mutations. 

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense
- In frame deletions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame insertions
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
