**SMAD4 -- Juvenile polyposis syndrome, (MIM 174900)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:6770

Mutation and phenotype analysis was used in 80 unrelated patients of
whom 65 met the clinical criteria for juvenile polyposis syndrome
(typical JPS) and 15 were suspected to have JPS. Using MLPA, **large
genomic deletions involving the SMAD4 were found in six** (9%) patients
with typical JPS. Four exhibited a heterozygous deletion of all SMAD4
probes encompassing the entire **SMAD4 gene and the promoter** region.
One patient had a deletion of **coding exons 5-11** and another had a
deletion of **coding exons 6-11**. All deletions were confirmed in a
second independent MLPA test.

***Aretz et al, 2007, PMID 17873119***

Archival material of 29 patients with JPS from 27 families was
collected. MLPA identified a germline **hemizygous large genomic
deletion involving SMAD4** in a one patient.

***Van Hattem et al, 2008, PMID 18178612***

DNA was extracted from 102 JPS probands. By MLPA, one **proband had
deletion of most of SMAD4 and another deletion of the 5\' end of
SMAD4**.

***Calva-Cerqueira et al, 2009, PMID 18823382***

**Literature Review:**

**OMIM:** https://www.omim.org/entry/174900 (and 600993)

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1469/

*Mechanism*

Although SMAD4 is a **tumor suppressor gene**, loss of heterozygosity
has not been demonstrated definitively as causal in the development of
polyps. Furthermore, whether such changes would affect cells in the
epithelium, the lamina propria, or both is also not known. SMAD4 is
**the common intracellular mediator of the TGF-β superfamily signaling**
pathways. BMPR1A is a type I cell surface receptor for the BMP pathway.
Ligands, such as TGF-β or BMP, bind to a receptor and **activate
signaling pathways** leading to protein complexes that migrate to the
nucleus and bind directly to DNA sequences **to regulate transcription**
\[Heldin et al 1997\]. The downstream genes under the control of these
signaling pathways are still being actively investigated.

*Pathogenic variants.*

. See Table 5. Germline pathogenic variants have been described in all
eleven coding exons. Changes include **small deletions, insertions, and
missense and nonsense** pathogenic variants. **Two splice site
variants** have been reported. Most pathogenic variants are unique, but
three have been reported in multiple unrelated families:
c.1244_1247delACAG, c.1162C\>T, and p.Arg361Cys. See Howe et al \[2004\]
and Calva-Cerqueira et al \[2009\] for a comprehensive list of the
pathogenic variants reported in SMAD4 (previously known as MADH4).
**Larger deletions of SMAD4 may also occur in up to 4% of affected**
individuals \[Aretz et al 2007, van Hattem et al 2008, Calva-Cerqueira
et al 2009\]. Deletions and pathogenic missense variants have also been
reported in **the promoter region** \[Calva-Cerqueira et al 2010\].

*Abnormal gene product.*

The **MH1** **domain** of the SMAD4 protein can directly bind to the DNA
of target genes. Pathogenic variants in this domain can **significantly
reduce the DNA binding activity** of SMAD4. Most pathogenic variants,
including the three recurrent pathogenic variants in Table 5, occur in
**the MH2 domain**, which plays an important role for nuclear
localization, interaction with other MAD proteins, and transcriptional
activation. In vitro studies demonstrate that **pathogenic nonsense
variants lead to significantly reduced bone morphogenetic protein
signaling**, with less of an effect for missense variants \[Carr et al
2012\].

Overall, **frameshift, nonsense, and missense** variants accounted for
the majority of pathogenic SMAD4 (72.9%) and BMPR1A (61.8%) alterations
in the ECS as well as the LBSB group (SMAD4: 79.9%; BMPR1A: 70.8%; Supplementary Table 3). Only
**large genomic, i.e., single or multiexon deletions** in SMAD4 were
significantly overrepresented in the ECS compared with the LBSB group.
**Splice site variants were noted** in 4-6% of SMAD4 and 10-16% of
BMPAR1A

***Blatter et al, 2020, PMID: 32398773***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Decreased gene product level

Absent gene product

*Altered gene product structure*

**Narrative summary of molecular mechanisms:**

SMAD4 is a tumour suppression gene and has a role in common
intracellular mediator of the TGF-β superfamily signalling. The
molecular mechanism of disease in relation to Juvenile polyposis is felt
to be loss of function principally due to reduced or absent DNA binding
activity in SMAD4. Variant classes include small and large deletions,
insertions, missense, nonsense, splice site and deletion/missense in the
promotor region. Of note, activating (gain of function) heterozygous de
novo mutations in the MH2 domain (principally reported to affect
residues 496 and 500 in the SMAD4 gene) cause Myhre syndrome.

**List variant classes in this gene proven to cause this disease:**

Stop_gained

missense

Frameshift_variant

Splice_acceptor_variant

Splice_donor_variant

regulatory_region_variant (deletions/missense in promotor region)

**Potential novel variant classes based on predicted functional
consequence**

**splice_region_variant**

**(splice_acceptor_variant predicted to undergo NMD)**

**(splice_acceptor_variant predicted to escape NMD)**

**(splice_donor_variant predicted to undergo NMD)**

**(splice_donor_variant predicted to escape NMD)**

**start_lost**

**(frameshift_variant predicted to undergo NMD)**

**(frameshift_variant predicted to escape NMD)**

**(stop_gained predicted to undergo NMD)**

**(stop_gained predicted to escape NMD)**

**stop_lost**

**inframe_insertion**

**inframe_deletion**

**(gain of upstream Start \[uORF\])**

**(gain of upstream Start \[oORF\])**

**(Stop lost \[uORF\])**

**(Stop lost \[oORF\])**

**(Start lost \[uORF\])**

**(Frameshift \[uORF\])**

**(Frameshift \[oORF\])**

**(Stop gained \[uORF\])**

**Not included**

**synonymous_variant**

**intron_variant**

**intergenic_variant**

**3_prime_UTR_variant**

**5_prime_UTR_variant**
