# **STK11 -- Peutz-Jeghers Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:11389

**ClinGen Evidence for Haploinsufficiency**

PMID 2563227 - Hearle et al 2006 report multiple (38) unrelated probands with Peutz-Jeghers syndrome assessed for STK11 mutations (exon deletions, nonsense, missense and loss of function from in-frame deletions in kinase domain). 16% (6/38) carry exon deletions and total 50% (19/38) had mutations in STK11: "four nonsense mutations, six deletions, and two insertions predicted to lead to truncation of the expressed protein, four missense mutations, and three splice site mutations".

PMID 20623358 - Resta et al 2010 reports multiple unrelated probands with Peutz-Jeghers syndrome carrying large deletions detected by MLPA (15/51 ~29%)

Haploinsufficiency phenotype comments:
Deletions (whole gene and exonic) and loss-of-function mutations in STK11 are associated with Peutz-Jeghers syndrome (PJS).

**Literature review:**

STK11 acts as a suppressor by activating TSC2 through an AMP-dependent protein kinase [Corradetti & Guan 2006] leading to accumulation of mTOR, which is critical for protein translation.

**More than 300 STK11 pathogenic variants have been reported in persons with Peutz-Jeghers syndrome. All types of variants have been reported, from missense variants to whole-gene deletion.**

Intragenic homologous recombination has been noted as a mechanism that can lead to deletion of exons 4-7 of STK11 [Ankala et al 2012].

It was also noted that recombination among Alu elements is a frequent cause of deletions of exons 2 and 3 [Borun et al 2015].

Penetrance - To date all reported individuals with pathogenic variants in STK11 have shown clinical manifestations.

Genotype - phenotype correlation - evidence is conflicting. Hearle et al found that the variant type and site within the functional domains of the expressed protein did not affect cancer risk. Other studies have reported that truncating variants are associated with earlier onset polyps or more GI surgeries. Amos et al 2004; Salloch et al 2010.

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1266/*

Jenne et al. (1998) performed mutation analysis in 5 unrelated PJS patients and found mutations in STK11 in each. In a 3-generation PJS family affected members carried an STK11 allele with a deletion of exons 4 and 5 and an inversion of exons 6 and 7. In 4 other unrelated PJS patients, they found 3 nonsense mutations and 1 acceptor splice site mutation. All 5 germline mutations were predicted to disrupt the function of the kinase domain. Jenne et al. (1998) concluded that germline mutations in STK11, probably in conjunction with acquired genetic defects of the second allele in somatic cells according to the Knudson model, caused the manifestations of PJS.

*Omim
https://www.omim.org/entry/602216*

Among STK11 mutations recorded in the Human Genome Mutation Database (HGMD), more than 300 point mutations and small range mutations are recorded, and 89 gross deletions or insertions

*Yu-Liang J et al 2018 PMID: 30528796*

A total of 419 Peutz-Jeghers syndrome patients (193 males
and 226 females) ascertained through 225 probands were
available for analysis

Eighty-five (83%) of the germ-line STK11/LKB1 mutations identified represent unique sequence changes. A diagrammatic representation of the coding sequence of STK11/LKB1 and the corresponding functional domains of the expressed protein is shown in Fig. 1A.

**Mutations were scattered throughout the gene, but no mutation
in exon 9 was identified in any patient. Over 85% of both
truncating and missense mutations localize to regions of
STK11/LKB1 encoding the kinase domain of the expressed
protein** (Fig. 1B). Sixty-one of the mutations resulted in the
truncation of the protein by the creation of premature
transcription termination signals, 16 occurred within highly
conserved splice sites (4 were in-frame deletions predicted to
lead to loss of kinase activity), and 15 were missense mutations.
All of the missense mutations led to nonconservative amino
acid changes that altered amino acids highly conserved in
evolution among human, mouse, and Xenopus homologues of
STK11/LKB1 and resided within the kinase domain of the
protein encoded by exons 1 to 8. Other mutations included one
large-scale genomic deletion and four exonic deletions. Seven
families had uncharacterized mutations.

*Hearle N et al 2006 PMID: 16707622*

The kinase domain of the human 433 amino acid protein is localised between residues 49 and 309,7 and shows homology to the conserved catalytic core of the kinase domain common to both serine/threonine and tyrosine protein kinase family members.10 Most mutations found in PJS patients are small deletions/insertions or single base substitutions leading to an abnormal truncated/kinase inactive protein.

*Schumacher V et al 2005 PMID: 15863673*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Deletions and loss of function mutations in STK11 are associated with Peutz-Jeghers syndrome. "STK11 is a multi-tasking tumor suppressor that has a role in apoptosis, cell cycle arrest, cell proliferation, cell polarity, and energy metabolism." All types of variants have been reported, from missense variants to whole-gene deletions. Mutations are seen throughout the gene but one study noted that no mutation in exon 9 was identified in any patient. Over 85% of both truncating and missense mutations localize to regions of STK11/LKB1 encoding the kinase domain.
Intragenic homologous recombination has been noted as a mechanism that can lead to deletion of exons 4-7 of STK11 [Ankala et al 2012]. It was also noted that recombination among Alu elements is a frequent cause of deletions of exons 2 and 3 [Borun et al 2015].
There are conflicting reports on whether there are genotype phenotype correlations. Hearle et al found that the variant type and site within the functional domains of the expressed protein did not affect cancer risk. Other studies have reported that truncating variants are associated with earlier onset polyps or more GI surgeries. Penetrance appears to be complete. 

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense
- In frame deletions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame insertions
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
