**TGFBR1 -- Loeys-Dietz syndrome type 1A/2A (MIM 609192/608967)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:11772

The first report indicating a relationship between the TGFBR1 gene and
autosomal dominant Loeys Dietz syndrome was reported by Dietz et al in
2005 (PMID: 15731757). Loeys Dietz syndrome is a multisystemic disorder
with craniofacial, skeletal, integumental and ocular involvement
(reviewed in Van Laer et al., 2014 PMID: 24443023 ). Loeys Dietz
syndrome comprises a broad phenotypic spectrum of severity including
seemingly isolated aortic aneurysm and/or dissection to the full
syndrome and thus represents a continuum of disease (reviewed in Van
Laer et al., 2014 PMID: 24443023; Verstraeten et al., 2016 PMID:
26919284). Aortic aneurysm and tortuosity represent the major morbidity
and mortality of Loeys Dietz syndrome. **Over 120 variants have been
identified in TGBFR1** (Verstraeten et al., 2016 PMID: 26919284)
according to the UMD-TGFBR1 database (http://www.umd.be/TGFBR1/), with
the **majority being missense and more rarely, nonsense, splice site and
small deletions.** The mechanism for the gene-disease relationship is
**perturbation of the TGF beta signaling pathway, however the exact
molecular mechanism remains unclear and could include gain of function
and dominant negative** (reviewed in Van Laer et al., 2014 PMID:
24443023; Verstraeten et al., 2016 PMID: 26919284). The majority of the
mutations result in **aberrant kinase activity of TGFBR1**. Evidence
supporting this gene-disease relationship includes case-level data,
segregation data, functional data, and model organisms. This
gene-disease relationship has been studied for more than 10 years and a
significant amount of case-level data, segregation data, and
experimental data is available and the maximum score for genetic
evidence (12 points) has been reached. Note, this curation effort may
not be exhaustive of all literature related to this gene-disease
relationship. In summary, TGFBR1 is definitively associated with
autosomal dominant Loeys Dietz syndrome. This has been repeatedly
demonstrated in both the research and clinical diagnostic settings, and
has been upheld over time. This classification was approved by the
ClinGen General GCEP on March 27, 2019 (SOP Version 6)

**Literature Review:**

**OMIM:** https://www.omim.org/entry/609192

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1133/

Pathogenic variants. The large majority of pathogenic variants
identified so far are located in the exons coding for the intracellular
serine-threonine kinase domain of both receptors. They most commonly
involve **pathogenic missense variants; only a few nonsense** variants
have been described.

*Loeys et al, 2015, PMID: 15731757*

LDS is induced by dominant-negative missense variants within or near STK
domain,

*Fujiwara et al, 2018, PMID: 29706644*

a Japanese familial case of LDS with a **novel splice donor site variant
in TGFBR1** gene (c.973 + 1G\>A; NG_007461.1). The intronic variant was
predicted to mediate in-frame exon 5 skipping within the
serine/threonine kinase(STK) domain... .... LDS variant generated two
types of in-frame transcription products, r.\[806_973del,965_973 del\],
and produced two functionally inactivated proteins

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Altered gene product structure*

(Decreased gene product level)

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly due to an abnormal gene product leading to
perturbation of the TGF beta signaling pathway, however the exact
molecular mechanism remains unclear and could include gain of function
and dominant negative. Some haploinsufficient variants (frameshift,
nonsense, splice and small deletions have been noted)

**List variant classes in this gene proven to cause this disease:**

Missense (most)

Stop_gained

Stop_gained predicted to undergo NMD

Splice_acceptor_variant

Small deletions (few)

Frameshift_variant

**Potential novel variant classes based on predicted functional
consequence**

Splice_donor_variant

Splice acceptor variant predicted to escape NMD

Splice acceptor variant predicted to undergo NMD

Splice donor variant predicted to escape NMD

Splice donor variant predicted to undergo NMD

Frameshift variant predicted to escape NMD

Frameshift variant predicted to undergo NMD

start_lost

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]

Inframe_insertion

Inframe_deletion
