**TGFBR2 -- Loeys-Dietz syndrome (type 1B and 2B MIM 610168/610380)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:11773

Assertion made by the Aortopathy working group. Considerable phenotypic
variability humans and mice with different genetic background.
Definitive for both LDS and nonsyndromic HTAAD.

**Numerous heterozygous missense and nonsense** mutations in TGFBR2 are
linked to Loeys-Dietz syndrome (LDS) types 1B (MIM \#610168) and
2B/Marfan (MIM \#610380). The **nonsense mutations are clustered in the
C-terminus**. The mechanism appears to be **dominant negative rather
than haploinsufficiency** (PMID 21098638). One paper describes a
**896-kb deletion including TGFBR2 but the patient had not signs of LDS
but instead exhibited microcephaly and developmental delay (PMID
21567932**). Due to the uncertainty regarding the exact mechanism by
which mutations in TGFBR2 cause LDS, the **haploinsufficiency rating is
deemed a 2.**

**Literature Review:**

**OMIM:** https://www.omim.org/entry/609192

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1133/

Pathogenic variants. The large majority of pathogenic variants
identified so far are located in the exons coding for the intracellular
serine-threonine kinase domain of both receptors. They most commonly
involve **pathogenic missense variants; only a few nonsense** variants
have been described.

A **splice mutation that disrupts reading frame, causing loss of
function**-type alteration, removing amino acids

*Loeys et al, 2015, PMID: 15731757*

Describes **missense** mutations and one **nonsense** mutation in
patients with /Loeys-Dietz 2B

*Loeys et al, PMID 16928994*

Screened 457 probands for mutations in TGFBR2 and found 23 mutations,
including 3 **nonsense** variants. PMID *18781618*

A **Loss-of-function splice mutation (synonymous** but causes abnormal
splicing) and a translocation disrupting TGFBR2 in Marfan type
2/Loeys-Dietz 2B. PMID *15235604*

**Missense mutations, one nonsense** mutation in patients with Marfan
type 2/Loeys-Dietz 2B. PMID *15235604*

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Altered gene product structure*

(Decreased Gene product)

**Narrative summary of molecular mechanisms:**

Mechanism is likely to be due to a dominant negative model with an
abnormal gene product leading to perturbation of the TGF beta signaling
pathway. There is uncertainty whether the phenotype may additionally be
caused as a result of haploinsufficiency. A few nonsense variants are
reported and one paper describes a 896-kb deletion including TGFBR2 but
the patient had not signs of LDS but instead exhibited microcephaly and
developmental delay (PMID 21567932).

**List variant classes in this gene proven to cause this disease:**

Missense (most)

Frameshift (few nonsense)

Splice_acceptor_variant

**Potential novel variant classes based on predicted functional
consequence**

Splice_donor_variant

Splice acceptor variant predicted to escape NMD

Splice donor variant predicted to escape NMD

Frameshift variant predicted to escape NMD

Stop gained predicted to escape NMD

Stop lost

Inframe_insertion

Inframe_deletion

*?? If consider haploinsufficiency as a potential MOA*

Frameshift

Splice region variant

Stop_gained

start_lost

Splice acceptor predicted to undergo NMD

Splice donor predicted to undergo NMD

stop_gained predicted to undergo NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
