# **TNNI3 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

https://search.clinicalgenome.org/kb/gene-validity/8770

The TNNI3 gene has been associated with hypertrophic cardiomyopathy (HCM). TNNI3 was first associated with this disease in humans in 1997 (Kimura et al, PMID 9241277). At least 60 unique variants, with varying levels of evidence to support their pathogenicity, have been reported in humans (reviewed in Mogensen et al, 2015, PMID 26440512). Variants in this gene segregated with disease in at least 6 families (Kimura et al, 1997, PMID 9241277; Rani et al, 2012, PMID 22876777; Choi et al, 2010, PMID 20641121; Mogensen et al, 2004, PMID 15607392). More evidence is available in the literature, but the maximum score for genetic evidence
(12 pts) was reached. **The mechanism for disease is likely dominant negative, as 91% of mutations reported are missense variants** (Mogensen et al, 2015, PMID 26440512). The gene-disease association is supported by the function of the gene product, animal models, and in vitro assays.
In summary, TNNI3 is definitively associated with hypertrophic
cardiomyopathy. This has been repeatedly demonstrated in both research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen Hypertrophic Cardiomyopathy Gene Curation Expert Panel on September 5, 2017.

**ClinGen Evidence for Haploinsufficiency**

TNNI3 encodes a cardiac muscle isoform of Troponin I. The majority of reported mutations are missense mutations that affect the TNNI3 coding sequence; however, these mutations are not reported to result in a loss of function. Instead, functional studies on these mutations have shown that they affect Ca2+ binding to myofilaments containing the mutant TNNI3 (PMIDs: 16531415 and 22675533) or result in an increased myofilament response to Ca2+ (PMID: 11735257).

At this time there is limited evidence to support haploinsufficiency of this gene. The only evidence supporting the haploinsufficiency of TNNI3 is a single report of a patient with a single nucleotide deletion in TNNI3 that resulted in decreased TNNI3 protein levels (PMID: 18006163).
There are currently no reported TNNI3 whole gene deletions.

Please note that there are additional indel/splice site mutations that have been reported in this gene. These mutations either do not have functional data to support a loss of function disease mechanism (PMID: 20474083, 25524337, 24111713, 12707239, 25940119, and 18467357), are thought to be associated with a mechanism other than loss of function (PMID: 11735257 and 21533915), or have a complex genotype (PMID: 21835320 and 25132132).

**Literature review:**

Mutations in the troponin complex introduce alterations in
Ca^2+^ affinity and protein-protein interactions which may ultimately lead to the development of cardiomyopathy

...a total of 37 studies of patients with HCM have reported 66 different cTnI mutations in 256 probands.
Ninety-one percent (n = 60) of all mutations reported were missense variants, 85% of which (n = 51) have been identified in exons 7 and 8 of cTnI
Only 2 splice-site mutations have been reported. Six of the mutations reported (Arg141Gln, Arg145Trp, Arg157Val, Arg162Gln, Ser166Phe, and Lys183Del) appeared with a particularly high frequency and were identified in 116 of the 256 probands (45%).

*Mogensen et al, 2015, PMID 26440512*

"A multivariable model stratified by evaluation with CMR demonstrated the independent predictive value of male sex and an abnormal ECG before the end of follow-up. Compared with patients with MYBPC3, individuals with TNNI3 variants had a lower penetrance of HCM. Older age at first evaluation was also associated with an increased risk"

*Lorenzini M et al 2020 PMID: 32731933*

**From our in-house Atlas of HCM:**

128/135 missense

1/135 inframe deletion

There was no statistically significant excess in truncating variants in HCM cases vs population controls

2/135 nonsense (both VUS)

4/135 frameshift (2 VUS, 2 pathogenic)

Both of the pathogenic fs variants are classified as uncertain
significance in ClinVar.

c.538del is a 'hot VUS'. It is predicted to lead to a premature
termination codon within the last 50 bases of the second to last exon and escape NMD

*Walsh et al, 2016 (PMID 27532257)*

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TNNI3&icc=HCM>

<https://www.ncbi.nlm.nih.gov/clinvar/variation/229332/>

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism is likely dominant negative due to **altered gene product structure.** rather than haploinsufficiency as the majority of pathogenic variants in TNNI3 are missense variants and there are no reported gene deletions. There are reports of nonsense and frameshift variants but there is conflicting evidence for pathogenecity. Functional studies on missense mutations have shown that they affect Ca2+ binding to myofilaments containing the mutant TNNI3 (PMIDs: 16531415 and 22675533) or result in an increased myofilament response to Ca2+ (PMID: 11735257). Mogensen et al reported "ninety-one percent (n = 60) of all mutations reported were missense variants, 85% of which (n = 51) have been identified in exons 7 and 8 of cTnI. Six of the mutations
reported (Arg141Gln, Arg145Trp, Arg157Val, Arg162Gln, Ser166Phe, and Lys183Del) appeared with a particularly high frequency and were identified in 116 of the 256 probands (45%)". When compared to MYPBC3 variants, penetrance appears to be lower.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**PM1** 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

There is a cluster of HCM cases in the troponin C and actin-binding domains in TNNI3, amino acid residues 141–209. Etiological fraction for this cluster is 0.974 (0.963–0.984) so PM1_strong could be activated.

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame deletion

**Potential novel variant classes based on predicted functional
consequence**

- Frameshift predicted to escape NMD
- stop_gained predicted to escape NMD
- Splice acceptor variant predicted to escape NMD 
- Splice donor variant
- Splice donor variant predicted to escape NMD
- stop_lost
- inframe_insertion

