# **TNNT2 -- Dilated Cardiomyopathy**

**Review of source material:**

**ClinGen:**

Not curated by ClinGen for DCM yet

**Literature review:**

Troponin T is a regulatory protein found in striated muscles that forms a complex with troponin I (TnI) and troponin C (TnC) that, together with tropomyosin (TM), is required for Ca^2+^-dependent regulation of muscle contraction.

Kamisago et al. (2000) identified a mutation in the TNNT2 gene as the cause of familial dilated cardiomyopathy. 

Mirza et al. (2005) studied all 8 published mutations causing dilated cardiomyopathy (CMD), including 5 in the TNNT2 gene (lys210del, R141W, R131W, R205L, and D270N; 191045.0006-191045.0010, respectively), 2 in the TPM1 gene (E54K, 191010.0004; and E40K, 191010.0005), and 1 in the TNNC1 gene (G159D; 191040.0001). Thin filaments, reconstituted with a 1:1 ratio of mutant:wildtype proteins, all showed reduced Ca(2+) sensitivity of activation in ATPase and motility assays, and, except for the E54K alpha-tropomyosin mutant which showed no effect, all showed lower maximum Ca(2+) activation. Incorporation of the TNNT2 mutations R141W and R205L into skinned guinea pig cardiac trabeculae also decreased Ca(2+) sensitivity of force generation. Thus, diverse thin filament CMD mutations appeared to affect different aspects of regulatory function yet change contractility in a consistent manner. Mirza et al. (2005) stated that the CMD mutations depressed myofibrillar function, an effect opposite to that of CMH-causing thin filament mutations, and suggested that decreased contractility might trigger pathways that ultimately lead to the clinical phenotype. 

Mogensen et al. (2004) analyzed the TNNT2 gene in 235 consecutive unrelated probands with dilated cardiomyopathy and identified 4 different mutations in 4 families, respectively. The mutations segregated with the disease in each family and were absent in 200 ethnically matched control chromosomes and 1,520 chromosomes from patients with hypertrophic cardiomyopathy. Functional studies showed significant impairment of mutated troponin interaction compared with wildtype control, indicating an altered regulation of myocardial contractility. 

***Omim** https://www.omim.org/entry/191045*

We propose that the variant classes significantly enriched in 1 or both DCM cohorts are likely to be most relevant for genetic testing in DCM. These were truncating variants in TTN, DSP, LMNA, BAG3, and VCL, and nontruncating variants in MYH7, TNNT2, TNNC1, LMNA, and TPM1.

*Mazzarotto F et al 2020 PMID: 31983221*

**From our in-house Atlas of HCM:**

36/36 non-truncatingÂ  (missense, in frame deletions)

0/36 truncating

*Walsh et al, 2016 (PMID 27532257)*
*Pugh TJ et al, 2014 (PMID: 24503780)*

We recently undertook bidirectional resequencing of TNNT2, the cardiac troponin T gene, in 313 probands with DCM. We identified six TNNT2 protein-altering variants in nine probands, all who had early onset, aggressive disease. Additional family members of mutation carriers were then studied when available. Four of the nine probands had DCM without a family history, and five had familial DCM. Only one mutation (Lys210del) could be attributed as definitively causative from prior reports. Four of the five missense mutations were novel (Arg134Gly, Arg151Cys, Arg159Gln, Arg205Trp), and one was previously reported with hypertrophic cardiomyopathy (Glu244Asp). Based on the clinical, pedigree and molecular genetic data these five mutations were considered possibly or likely disease causing. To further clarify their potential pathophysiologic impact, we undertook functional studies of these mutations in cardiac myocytes reconstituted with mutant troponin T proteins. We observed decreased Ca2+ sensitivity of force development, a hallmark of DCM, in support of the conclusion that these mutations are disease-causing.

*Hershberger RE et al 2010 PMID: 20031601*

#### **Pilot application of harmonised terms**

**Inheritance**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

(optional) modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Troponin T is a regulatory protein found in striated muscles that forms a complex with troponin I (TnI) and troponin C (TnC) that, together with tropomyosin (TM), is required for Ca^2+^-dependent regulation of muscle contraction. Non-truncating variants (missense, in-frame deletions) are enriched in TNNT2 in DCM patients compared to controls (this is not the case for truncating variants). The mechanism appears to be dominant negative rather than haploinsufficiency. Functional studies have shown pathogenic missense variants in TNNT2 causing decreased Ca2+ sensitivity of force development.

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame deletion

**Potential novel variant classes based on predicted functional
consequence**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- Stop gained predicted to escape NMD
- stop_lost
- inframe_insertion
