# **TNNT2 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

https://search.clinicalgenome.org/kb/gene-validity/8770

The TNNT2 gene has been associated with hypertrophic cardiomyopathy.
TNNT2 was first associated with the disease in humans in 1994
(Thierfelder et al, 1994, PMID 8205619; Watkins et al, 1993, PMID
7981753; Watkins et al, 1995, PMID 7898523). Many unique variants, with varying levels of evidence to support their pathogenicity, have been reported in humans, as variants in TNNT2 account for \~5% of HCM cases (Cirino and Ho, 2014, PMID 20301725). Variants in this gene segregated with disease in at least 3 families (Thierfelder et al, 1994, PMID8205619). More evidence is available in the literature, but the maximum score for genetic evidence (12 pts) was reached. **The mechanism for disease is likely dominant negative, as most mutations reported are missense variants**. The gene-disease association is supported by the function of the gene product, animal models, and in vitro assays. 
In summary, **TNNT2 is definitively associated with hypertrophic
cardiomyopathy**. This has been repeatedly demonstrated in both research and clinical diagnostic settings, and has been upheld over time. This classification was by the ClinGen Hypertrophic Cardiomyopathy Gene Curation Expert Panel on November 7th, 2017.

It is unknown if haploinsufficiency results in a phenotype. In PMID 19666196, MLPA was used to screen TNNT2 for large rearrangements and none were found. There are a few isolated reports of splice site mutations and stop mutations but pathogenicity remains unproven.

**Literature review:**

Troponin T is a regulatory protein found in striated muscles that forms a complex with troponin I (TnI) and troponin C (TnC) that, together with tropomyosin (TM), is required for Ca^2+^-dependent regulation of muscle contraction.

*Thierfelder et al, 1994, PMID 8205619* demonstrated that affected
individuals from 3 unrelated families with the form of familial
hypertrophic cardiomyopathy linked to 1q
(CMH2; [[115195]{.ul}](https://www.omim.org/entry/115195)) contained point mutations: missense mutations (ile79-to-asn, 
arg92-to-gln) in 2 of them and a mutation in the splice donor sequence of intron 15 in the third. The
abnormalities were demonstrated by screening by RNase A protection
assays followed by sequencing.

***Omim** https://www.omim.org/entry/191045*

*Tardiff JC et al. 1998, PMID:9637714* stated that 9 mutations had been described in the TNNT2 gene that cause familial hypertrophic
cardiomyopathy, including 7 missense mutations, a deletion of an
internal amino acid, and a splice site mutation that would result in the loss of either the 14 or 28 C-terminal residues with the addition of 7 non-TNNT2 amino acids in the latter case.

*Tardiff JC et al. 1998, PMID:9637714* 

Twelve different mutations in exons 8, 9, 11, 15, and 16 were identified in 20 families: Arg278Cys in 3 families; Arg92Leu in 2 families; Arg92Trp in 3 families; ΔGlu163 in 3 families; IVS15+1G\>A in 2 families; and Ala104Val, Arg278His, Arg92Gln, Arg94Leu, Glu163Lys, Glu83Lys, Ile79Asn in single families.

*Pasquale et al.2011 PMID: 22144547*

The entire coding sequences of 9 genes
(*MYH7*, *MYBPC3*, *TNNI3*, *TNNT2*, *MYL2*, *MYL3*, *TPM1*, *ACTC*, and *TNNC1*) were analyzed in 197 unrelated index cases with familial or sporadic hypertrophic cardiomyopathy. Analysis of *TNNT2* showed 5 missense mutations, one codon deletion (Del E160), and one nonsense mutation (W287ter). *TNNT2* mutations are located in regions essential for anchoring the troponin-tropomyosin complex onto the thin filament.[**^[18]{.ul}^**](https://www.ahajournals.org/doi/full/10.1161/01.CIR.0000066323.15244.54?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed#R18-126913) 
In 2 unrelated patients, a termination codon (W287X) involving the last residue of the protein was identified.

*Richard P et al. 2003 PMID: 12707239*

"Residues 80--180 of TnT, which bind directly to Tm, harbor most of the known disease-causing single-residue substitutions, with two hotspots at residues 92 and 160--163.

....suggests that the primary mechanism underlying disease for mutants in the TNT1 region may be the altering of the affinity of TnT for Tm."

*Gangadharan B et al. 2017 PMID: [28973951]*

**From our in-house Atlas of HCM:**

[[https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TNNT2&icc=HCM]{.ul}](https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TNNT2&icc=HCM)

92/119  missense

16/119  inframe deletion

4/119 nonsense

*Walsh et al, 2016 (PMID 27532257)*

These 4 reported are all the same variant - W287X -- leads to a
premature termination codon at position 287. This alteration occurs within the last exon and may therefore escape nonsense mediated decay (NMD), resulting in a truncated protein that is lacking the last 2 amino acids.

Other nonsense mutations on ClinVar have conflicting evidence.

7/119 essential splice site

Variant c.821 + 1G\>C is reported as pathogenic. A variant at same
position c.821 + 1G\>A is predicted to result in skipping of exon 15.
(Int. J. Mol. Sci. 2016, 17, 1883; doi:10.3390/ijms17111883
supplementary data).

#### **Pilot application of harmonised terms**

**Inheritance**

Autosomal dominant

(optional) modifiers: Incomplete penetrance

**Allelic requirement**

Monoallelic_aut

(optional) modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Troponin T is a regulatory protein found in striated muscles that forms a complex with troponin I (TnI) and troponin C (TnC) that, together with tropomyosin (TM), is required for Ca^2+^-dependent regulation of muscle contraction. Mechanism may be an alteration in the affinity of troponin T for tropomyosin due to **altered gene product structure.** The mechanism appears to be dominant negative rather than haploinsufficiency. Most of the pathogenic variants in TNNT2 are missense variants. Several splicing variants have been reported but depending on the location of these there is conflicting evidence in the literature regarding pathogenecity. Nonsense variants have been identified in the penultimate and final exon. It is possible these will lead to a truncated protein escaping nonsense mediated decay. Variants in HCM cases cluster in the tropomyosin-binding domain, amino acid residues 79--179, which bind directly to Tm. Gangadharan et al suggest there are two hotspots within that region at residues 92 and 160--163.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**PM1** 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

The tropomyosin-binding domain in TNNT2 is a cluster for HCM non-truncating variants, amino acid residues 79-179
Etiological fraction - 0.958 (0.941–0.974). PM1_strong could be applied for this cluster.

**PVS1**
??

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame deletion
- stop_gained predicted to escape NMD
- ?? Splice donor variant

**List other variant classes predicted to lead to the same functional consequence**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- stop_lost
- inframe_insertion
