**TP53 -- Li-Fraumeni syndrome 1 (MIM 151623)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:11998

TP53 was first reported in relation to autosomal dominant Li-Fraumeni syndrome in 1991 (Malkin et al., 1991, PMID: 1978757; Li et al., 1998,
PMID: 3409256). Li Fraumeni is associated with increased risk of multiple pediatric and adult malignancies (see discussion below).
Numerous variants have been reported in TP53 in relation to the development of Li-Fraumeni syndrome and include **missense, small duplications, small deletions, frameshift, and nonsense mutations** have been reported in humans. De novo inheritance has been noted in
Li-Fraumeni syndrome in 7-20% of cases (Schneider et al., OMID: 20301488; Gene Reviews). There are several databases describing TP53 variants of interest in Li-Fraumeni Syndrome, including: (1) the TP53
database in LOVD (https://databases.lovd.nl/shared/genes/TP53) ; (2) The TP53 Website (http://p53.fr/) (Leroy et al, 2013; PMID: 23161690) which
houses the IARC TP53 database (http://p53.iarc.fr/) (Olivier et al.,
2002, PMID: 12007217); (3) Database of germline p53 mutations
(http://stary.lf2.cuni.cz/projects/germline_mut_p53.htm); (4) The UMD
TP53 website (http://www.umd.be:2072/IFAMTP53A.shtml). There is
significant genetic evidence supporting this gene-disease relationship
includes case-level data and segregation data and the maximum score for
genetic evidence (12 points) has been reached. This gene-disease
relationship is also supported by expression studies and multiple animal
models that get LFS associated tumors...... The molecular mechanism of
TP53 dysfunction for all the associated disease entities listed above
are either **loss of function or dominant negative**; both of which
result in **reduced (or absent) transcriptional activity and loss of the
tumor suppressive function of the p53 protein** **function**. Per the
criteria outlined by the ClinGen Lumping and Splitting Working Group, we
have found no difference in molecular mechanism, inheritance pattern, or
phenotypic expressivity, and therefore have lumped the above listed
disease entities into the curation for TP53 in Li-Fraumeni Syndrome. In
summary, TP53 is definitively associated with autosomal dominant
Li-Fraumeni syndrome. This has been repeatedly demonstrated in both the
research and clinical diagnostic settings, and has been upheld over
time.

Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**Literature Review:**

**OMIM:** https://www.omim.org/entry/151623

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1311/

Mechanism of disease causation. Germline TP53 pathogenic variants create
a **constitutive defect of p53 DNA binding and transcriptional response
to DNA damage**. According to Zerdoumi et al \[2017\], \"germline TP53
mutations represent a genetic permissive context facilitating malignant
transformation of cells in which DNA damage has occurred.\"

TP53-specific laboratory technical considerations. TP53 missense
variants are the variants most commonly identified in tumors and they
present challenges in germline interpretation.

A recent study reported that individuals with germline TP53 pathogenic
variants resulting in p53 loss of function appeared to have a more
severe phenotype than individuals with pathogenic variants that caused
partial deficiency of p53. ......in contract another report showed
dominant-negative pathogenic variants (in which the mutated p53 protein
interferes with the function of the wild type p53 protein) appeared to
have more clinically severe phenotypes than did individuals with other
TP53 pathogenic variants \[Bougeard et al 2015\]. A laboratory study
also reported that **dominant-negative pathogenic variants appear to
cause a more profound alteration of p53** DNA binding than other
pathogenic variants \[Zerdoumi et al 2017\].

However, the **penetrance of LFS may be overestimated** as more
individuals recently identified with a germline TP53 pathogenic variant
do not meet classic LFS or Chompret criteria due to a less striking
family and personal history of cancer \[Rana et al 2018\]. **Individuals
with TP53 pathogenic variant p.Arg337His appear to have a lower
lifetime risk of cancer than those with other TP53 pathogenic variants**
\[Ferreira et al 2019\].

...study of 214 families harboring 133 distinct TP53 alterations..... Of
these 133 alterations, 124 were **point mutations. Nine families
carried distinct TP53 genomic rearrangements (two whole gene
deletions, six partial deletions and one partial duplication**).
Functional comparison studies indicated that patients with
dominant-negative-acting missense mutations had significantly
earlier tumor onset, non-dominant-negative missense mutations were
intermediate in severity, and loss-of-function type mutations
(including nonsense, frameshift and genomic alterations) are
associated with later tumor onset.... A deletion segregated with LFS
in four subjects but was also identified in an unaffected 70-year old
relative, indicating incomplete penetrance.

***Bougeard et al, 2015, PMID 26014290***

....eight patients with **CNVs affecting TP53** from a cohort of 4524
patients with diverse phenotypes tested across multiple diagnostic
laboratories in Canada and the US. Four patients with focal TP53
deletions **(intragenic/exonic or 3? partial**) were affected with
early-onset cancer while four patients with non-focal deletions
encompassing TP53 and additional genes in 17p13.1 (size range 543 kb
to 2.4 Mb) had overlapping developmental phenotypes but did not have
cancer at the age of ascertainment (Ages 3.4, 5.75, 7.58, 33.67).

**Shlien et al, 2010, PMID 21056402**

**The spectrum of mutations shows that the majority of all germline
mutations cluster in exons 5-8,** however this is undoubtedly skewed
because most studies only examine these exons. In our own study, of 19
mutations found to date, five fall outside these exons (26%;

Varley et al, 1997a). This finding has profound implications for genetic
testing within Li-Fraumeni families, and we recommend that all exons,
both coding and non-coding, plus all splice junctions and the promoter
region are analysed. We have analysed all the reported germline
mutations to determine the mutation type. There is no difference in the
frequency of **missense and nonsense** mutations between Li-Fraumeni
families,other types of families and individuals with no family history.
In addition, there are no differences between LFS and LFL families.
There are two reports of **large germline deletions, one involving the
whole of exon 10 (Plummer et al, 1994) and the other removing 167 bp,
including part of exon 1 and intron 1** (Varley et al, 1997a). Both of
these deletions were flanked by short direct repeat sequences. Thirteen
other mutations involve **short insertion or deletion** events
(Sameshima et al, 1992; Toguchida et al, 1992; Felix et al, 1993; Birch
et al, 1994a; Hamelin et al, 1994; Mazoyer et al, 1994; Stolzenberg et
al, 1994; Lubbe et al, 1995; Strauss et al, 1995; Felix et al, 1996;
Varley et al, 1996a, 1997a).

***Varley, Evans and Birch, 1997, PMID: 9218725***

Rare germline variant (rs78378222) in the TP53 3\' UTR - Evidence for a
new mechanism of cancer predisposition in Li-Fraumeni syndrome. .... a
rare variant that is located in the 3\' untranslated region
(3\' UTR) of TP53, in 7 probands (5.4%) of a cohort from LFS/LFL
patients without TP53 germline mutations in the coding regions.

***Macedo et al, 2016, PMID: 26823150***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

Optional Modifier -- Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Molecular mechanism is loss of function or dominant negative; both of
which result in reduced (or absent) transcriptional activity and loss of
the tumor suppressive function of the p53 protein. Variant classes
include missense, small duplications, small deletions, frameshift, and
nonsense mutations. Variants are distributed throughout the coding
sequence and whole gene deletions are also documented. Some variant have
been shown to have incomplete penetrance (p.Arg337His).

**List variant classes in this gene proven to cause this disease:**

Splice_acceptor_variant

Splice_acceptor_variant predicted to undergo NMD

Splice_donor_variant

Splice_donor_variant predicted to undergo NMD

Frameshift_variant

Frameshift_variant predicted to undergo NMD

Stop_gained

Stop_gained predicted to undergo NMD

Inframe_deletion

Inframe_insertion

Missense

3_prime_UTR_variant

**Potential novel variant classes based on predicted functional
consequence**

Splice_region_variant

Splice_acceptor_variant predicted to escape NMD

Splice_donor_variant predicted to escape NMD

Frameshift_variant predicted to escape NMD

Start_lost

Stop_gained predicted to escape NMD

Stop_lost

5_prime_UTR_variant

Gain of upstream Start \[uORF\]

Gain of upstream Start \[oORF\]

Stop lost \[uORF\]

Stop lost \[oORF\]

Start lost \[uORF\]

Frameshift \[uORF\]

Frameshift \[oORF\]

Stop gained \[uORF\]

**Not included**

intron_variant
