# **TPM1 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

<https://www.ahajournals.org/doi/10.1161/CIRCGEN.119.002460>

**ClinGen:**

<https://search.clinicalgenome.org/kb/gene-validity/8771>

The TPM1 gene has been associated with hypertrophic cardiomyopathy
(HCM). TPM1 was first associated with this disease in humans in 1994
(Thierfelder et al, PMID 8205619). At least 15 unique heterozygous
variants (missense), with varying levels of evidence to support their
pathogenicity, have been reported in humans (reviewed in Redwood and
Robinson, 2013, PMID 24005378). Variants in this gene segregated with
disease in at least 7 families (Thierfelder et al, 1994, PMID 8205619;
Jääskeläinen et al, 1998, PMID 9822100; Karibe et al, 2001, PMID
11136687; Jongbloed et al, 2003, PMID 12651045). More evidence is
available in the literature, but the maximum score for genetic evidence
(12 pts) has been reached. The mechanism for disease is likely dominant
negative (Redwood and Robinson, 2013, PMID 24005378). The gene-disease
association is supported the function of the gene product, animal
models, and in vitro assays. In summary, TMP1 is definitively associated
with HCM. This has been repeatedly demonstrated in both research and
clinical diagnostic settings, and has been upheld over time. This
classification was approved by the ClinGen Hypertrophic Cardiomyopathy
Gene Curation Expert Panel on December 20, 2016.

**Literature review:**

Redwood and Robinson reviewed the literature in 2013 and reported that there were at least 15 unique missense mutations reported in humans with HCM. Mutations in TPM1 have also been associated with DCM. None of the mutations associated with DCM have been seen to cause HCM in other individuals or families.

***"TPM1* mutations most likely act via a dominant negative, poison polypeptide mechanism rather than via haploinsufficiency.**

Analysis of vastus lateralis muscles from two HCM patients carrying the Asp175Asn mutation revealed equal expression of wild type and mutant α-tropomyosin proteins (Bottinelli et
al. [1998])

*Redwood and Robinson, 2013, PMID 24005378*

"Of the 4 human tropomyosin genes, TPM1 is the most versatile and
encodes at least 10 tissue-specific variants via alternative splicing and/or the use of 2 promoters."

Mutations in TPM1 "like those in the cardiac troponin T gene
(TNNT2; [191045](https://omim.org/entry/191045)), are characterized by relatively mild and sometimes subclinical hypertrophy but a high incidence of sudden death. Genetic testing may therefore be especially important in this group"

*Omim <https://omim.org/entry/191010>*

"...studies revealed that mutations in thin filament proteins cause HCM and DCM via an opposing functional defect. HCM mutations cause increased myofilament Ca2+ sensitivity, whereas DCM mutations have been shown to reduce myofilament Ca2+ sensitivity.
In inherited cardiomyopathies, the evidence presented in TABLES 2–5 indicates a direct link between the effect of thin filament gene mutations on Ca2+ sensitivity of force and the prevailing phenotype: an association between HCM and DCM with an increase or decrease in Ca2+ sensitivity of force, respectively."

*van der Velden J et al 2019 PMID: 30379622* 

"One of these sarcomere genes is TPM1, which encodes alpha-tropomyosin and causes HCM (10 TPM1 gene variants), DCM (11 TPM1 variants), cardiomyopathy (2 TPM1 variants) and left-ventricular non-compaction (3 TPM1 variants) according to ClinVar database (January 2020) [4]. **All reported TPM1 variants are missense variants that result in a single amino acid substitution, and they account for 1.5% of all likely pathogenic and pathogenic gene variants in HCM and 1.9% in DCM [5]."**

Our study confirms this clinical heterogeneity in two families with TPM1 variants and different cardiomyopathy subtypes, namely TPM1T201M (DCM) and TPM1E62Q and TPM1M281T (HCM; and in combination RCM). We also describe, for the first time, compound heterozygote TPM1E62Q/M281T variants in a child with RCM. Overall, the genotypes and diverse clinical characteristics of the family members of the DCM and RCM probands suggest that additional causal factors are necessary for the development of cardiac abnormalities in carriers of a heterozygous TPM1 variant, whereas compound heterozygosity for TPM1 variants causes severe cardiomyopathy in childhood.

"...we observed a decrease in Ca transients for both HCM- and DCM-related TPM1 variants...Our current data indicate that TPM1 variant-related changes in Ca2+ affinity and CaT are independent of the type of cardiac remodeling, which suggests that the diverse clinical cardiac phenotypes seen in affected subjects may rather involve modifier genes and/or environmental factors [17]."

*Dorsch LM et al 2020 PMID: 32882290*

**From our in-house Atlas of HCM:**

66/66 missense

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TPM1&icc=HCM>

*Walsh et al, 2016 (PMID 27532257)*

#### **Pilot application of harmonised terms**

**Inheritance**

Autosomal dominant

Optional modifiers: ?age related penetrance

**Allelic requirement**

Monoallelic_aut

Optional modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily due to **altered gene product structure.** All reported TPM1 variants are missense variants that result in a single amino acid substitution, and they account for 1.5% of all likely pathogenic and pathogenic gene variants in HCM and 1.9% in DCM. In the 2013 Redwood et al review 15 unique missense mutations had been reported in humans with HCM. None of the mutations associated with DCM have been seen to cause HCM in other individuals or families. Redwood et al propose the most likely mechanism to be dominant negative, there is no evidence currently for haploinsufficiency. 
A review by van der Velden J et al proposed "that mutations in thin filament proteins cause HCM and DCM via an opposing functional defect. HCM mutations cause increased myofilament Ca2+ sensitivity, whereas DCM mutations have been shown to reduce myofilament Ca2+ sensitivity." More recently Dorsch et al observed "a decrease in Ca transients for both HCM- and DCM-related TPM1 variants" suggesting "that TPM1 variant-related changes in Ca2+ affinity and CaT are independent of the type of cardiac remodeling, which suggests that the diverse clinical cardiac phenotypes seen in affected subjects may rather involve modifier genes and/or environmental factors..." 
There is significant clinical heterogeneity even within the same family.  There is one report of compound heterozygote TPM1E62Q/M281T variants in a child with RCM. The authors suggest that compound heterozygosity for TPM1 variants causes severe cardiomyopathy in childhood.

**List variant classes in this gene proven to cause this disease:**

- Missense

**Potential novel variant classes based on predicted functional
consequence**

- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Splice acceptor variant predicted to escape NMD
- Stop_lost
- In frame_insertion
- In frame deletion

