# **TPM1 -- Dilated Cardiomyopathy**

**Review of source material:**

**ClinGen:**

Not yet curated by ClinGen for DCM

**Literature review:**

Recent study compared the burden of rare protein-altering variation in DCM-associated genes in up to 2538 patients with DCM.  The variant classes significantly enriched in 1 or both DCM cohorts were truncating variants in TTN, DSP, LMNA, BAG3, and VCL,
and nontruncating variants in MYH7, TNNT2, TNNC1, LMNA, and TPM1. 

*Mazzarotto F et al 2020 PMID: 31983221*

Using depletion and reconstitution of TPM1 and troponin in porcine cardiac myofibrils, Chang et al. (2005) studied 3 hypertrophic cardiomyopathy (CMH) -associated TPM1 mutations (E62Q; E180G, 190010.0001; and L185R) and 2 dilated cardiomyopathy (CMD) -associated mutations (E54K and E40K) and found that all CMH-associated mutations increased the Ca(2+) sensitivity of ATPase activity and had decreased abilities to inhibit ATPase activity, whereas **the CMD-associated mutations decreased the Ca(2+) sensitivity of ATPase activity and had no effect on the inhibition of ATPase activity.** Chang et al. (2005) concluded that mutations that cause CMH and CMD disrupt discrete mechanisms, and suggested that this may explain the manifestation of distinct cardiomyopathic phenotypes. 

Mirza et al. (2005) studied all 8 published mutations causing dilated cardiomyopathy (CMD), including 5 in the TNNT2 gene (lys210del, R141W, R131W, R205L, and D270N; 191045.0006-191045.0010, respectively), 2 in the TPM1 gene (E54K and E40K), and 1 in the TNNC1 gene (G159D, 191040.0001). Thin filaments, reconstituted with a 1:1 ratio of mutant:wildtype proteins, all showed reduced Ca(2+) sensitivity of activation in ATPase and motility assays, and, except for the E54K alpha-tropomyosin mutant which showed no effect, all showed lower maximum Ca(2+) activation

*Omim <https://omim.org/entry/191010>*

**From our in-house Atlas of HCM:**

19/21 missense
1/21 frameshift (VUS)
1/21 nonsense (VUS)

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TPM1&icc=HCM>

*Walsh et al, 2016 (PMID 27532257)*
*Pugh TJ et al, 2014 (PMID: 24503780)*

DCM is in general characterised by reduced penetrance and variable expressivity.

*Hershberger RE et al 2013 PMID: 23900355*

"...studies revealed that mutations in thin filament proteins cause HCM and DCM via an opposing functional defect. HCM mutations cause increased myofilament Ca2+ sensitivity, whereas DCM mutations have been shown to reduce myofilament Ca2+ sensitivity.
In inherited cardiomyopathies, the evidence presented in TABLES 2–5 indicates a direct link between the effect of thin filament gene mutations on Ca2+ sensitivity of force and the prevailing phenotype: an association between HCM and DCM with an increase or decrease in Ca2+ sensitivity of force, respectively."

*van der Velden J et al 2019 PMID: 30379622* 

"One of these sarcomere genes is TPM1, which encodes alpha-tropomyosin and causes HCM (10 TPM1 gene variants), DCM (11 TPM1 variants), cardiomyopathy (2 TPM1 variants) and left-ventricular non-compaction (3 TPM1 variants) according to ClinVar database (January 2020) [4]. **All reported TPM1 variants are missense variants that result in a single amino acid substitution, and they account for 1.5% of all likely pathogenic and pathogenic gene variants in HCM and 1.9% in DCM [5]."**

Our study confirms this clinical heterogeneity in two families with TPM1 variants and different cardiomyopathy subtypes, namely TPM1T201M (DCM) and TPM1E62Q and TPM1M281T (HCM; and in combination RCM). We also describe, for the first time, compound heterozygote TPM1E62Q/M281T variants in a child with RCM. Overall, the genotypes and diverse clinical characteristics of the family members of the DCM and RCM probands suggest that additional causal factors are necessary for the development of cardiac abnormalities in carriers of a heterozygous TPM1 variant, whereas compound heterozygosity for TPM1 variants causes severe cardiomyopathy in childhood.

"...we observed a decrease in Ca transients for both HCM- and DCM-related TPM1 variants. The previous study and our current data indicate that TPM1 variant-related changes in Ca2+ affinity and CaT are independent of the type of cardiac remodeling, which suggests that the diverse clinical cardiac phenotypes seen in affected subjects may rather involve modifier genes and/or environmental factors [17]."

*Dorsch LM et al 2020 PMID: 32882290*

The aim of the present study is to investigate the correlation between the 3' untranslated region (3'UTR) single nucleotide polymorphisms (SNPs) of the TPM1 gene and dilated cardiomyopathy (DCM). A total of 245 patients with DCM and 245 healthy controls were recruited to analyze the TPM1 gene rs12148828, rs11558748, rs707602, rs6738, rs7178040 loci genotypes. The risk of DCM development in the rs6738 locus G allele carriers were 1.69 times more than A allele carriers (95% CI: 1.22-2.33, P = .001). Age and gender had no effect on the association of TPM1 gene SNPs with DCM risk (P > .05).

*Yao Q et al 2019 PMID: 31689804*

#### **Pilot application of harmonised terms**

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

Optional modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily due to **altered gene product structure.** The majority of pathogenic variants described are missense variants. Non-truncating variants in TPM1 were enriched in a large cohort of DCM patients (2538) in comparison to controls. Pathogenic variants in TPM1 have also been associated with HCM. None of the variants associated with DCM have been seen to cause HCM in other individuals or families. Pathogenic variants associated with DCM appear to decrease the Ca(2+) sensitivity of ATPase activity and had no effect on the inhibition of ATPase activity. This was in contrast to variants that caused HCM which appeared to increase the Ca(2+) sensitivity of ATPase activity and had decreased abilities to inhibit ATPase activity. More recently Dorsch et al observed "a decrease in Ca transients for **both** HCM- and DCM-related TPM1 variants" suggesting "that TPM1 variant-related changes in Ca2+ affinity and CaT are independent of the type of cardiac remodeling, which suggests that the diverse clinical cardiac phenotypes seen in affected subjects may rather involve modifier genes and/or environmental factors..."  DCM is in general characterised by reduced penetrance and variable expressivity.

**List variant classes in this gene proven to cause this disease:**

- Missense

**Potential novel variant classes based on predicted functional
consequence**

- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Splice acceptor variant predicted to escape NMD
- Stop_lost
- In frame_insertion
- In frame deletion

