# **TSC1 -- Tuberous sclerosis**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:12362

TSC1, which encodes the protein hamartin, was first reported in relation to autosomal dominant tuberous sclerosis complex (TSC) in 1997 (van Slegtenhorst et al., 1997 PMID: 9242607). Numerous variants have been reported in TSC1 in relation to tuberous sclerosis complex, **including missense, nonsense, indels and small deletions that result in frameshift, as well as intragenic deletions (Hasbani and Crino, 2018 PMID: 29478616).** The TSC1 database in LOVD (http://chromium.lovd.nl/LOVD2/TSC/home.php?select_db=TSC1) shows a total of 928 unique variants associated with TSC. Evidence supporting this gene-disease relationship includes case-level data, segregation and experimental data. This gene-disease relationship has been studied for 20 years, therefore a significant amount of case-level data and segregation is available, however the maximum score for genetic evidence (12 points) has been reached. This gene-disease relationship is supported by functional studies including expression and protein interaction, as well as animal models and rescue experiments, thus the maximum score for experimental evidence (6 points) has been reached. TP53 has been associated with several disease entities and/or phenotypes through germline inheritance, including: (1) Tuberous sclerosis-1 (MIM:191100) and (2) Lymphangioleiomyomatosis (MIM:606690) Evidence in the literature supports that Lymphangioleiomyomatosis occurs in individuals with TSC and mutations in TSC1. Additionally, a sporadic form of Lymphangioleiomyomatosis (S-LAM) can occur through by the two hit model, in which the wildtype allele of TSC1 incurs loss of heterozygosity (LOH) specifically in the lung (Murphy et al., 2017 PMID: 28643793). Of note, the involvement of TSC1 in Lymphangioleiomyomatosis (sporadic) is rare, and the TSC2 gene is more frequently shown to have a relationship with S-LAM. Hamartin, the protein product of the TSC1 gene, is a known tumor suppressor, therefore loss of function results in uncontrolled cell proliferation. Consistent with this model the **molecular mechanism for TSC1 is loss of function, as indicated by deletion and nonsense variation, as well as biochemical loss of tumor suppressor activity (Hasbani and Crino, 2018 PMID: 29478616).** Per the criteria outlined by the ClinGen Lumping and Splitting Working Group, we have found no difference in molecular mechanism, inheritance pattern, or phenotypic expressivity, and therefore Lymphangioleiomyomatosis due to germline mutations in TSC1 alone that occur in TSC patients has been lumped into the disease entity, Tuberous sclerosis complex. However, the somatically occurring Lymphangioleiomyomatosis due to one (or two) somatic mutations in TSC1 is not covered under the current Gene Disease Validity Classification matrix, asit is restricted to germline inheritance, therefore no evidence for somatic mutations in TSC1 were included in the current curation. In summary, TSC1 is DEFINITIVELY associated with autosomal dominant tuberous sclerosis complex. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time.

**ClinGen Evidence for Haploinsufficiency**
PMID 9242607 van Slegtenhorst et al. (1997): Reports numerous nonsense mutations identified within TSC1 amongst individuals with a clinical diagnosis of tuberous sclerosis.

PMID 9924605 Kwiatkowska et al. (1998): Reports two additional nonsense mutations (not previously reported in van Slegtenhorst et al. 1997) identified within TSC1 amongst individuals with a clinical diagnosis of tuberous sclerosis

PMID 9803264 Young et al. (1999): Reports several other nonsense mutations identified amongst individuals with a clinical diagnosis of tuberous sclerosis not previously reported in van Slegtenhorst et al.

**Literature review:**

**"Almost all TSC1 pathogenic variants are predicted to cause truncation of the hamartin protein; the location of the TSC1 pathogenic variant does not appear to associate with disease severity. Approximately 650 unique TSC1 pathogenic variants have been identified in more than 1,950 individuals/families with TSC1-related TSC** (Table A). Most pathogenic variants are unique, but a few are known to recur, including those in specific codons of exon 15. Other pathogenic variants are scattered throughout the exons and splice sites. **A small percentage of pathogenic missense variants have been identified and located mostly in the region encoding the N-terminal of hamartin** 

TSC1 is approximately 50 kb in size and the longest transcript variant (NM_000368.4), comprising 23 exons. The first two exons are noncoding. Exon 5 and exon 12 are alternatively spliced, producing shorter transcript variants.

Tuberin and hamartin were shown to be key regulators of the AKT/mTOR signaling pathway and to participate in several other signaling pathways including the MAPK, AMPK, b-catenin, calmodulin, CDK, autophagy, and cell cycle pathways [Kozma & Thomas 2002, Astrinidis et al 2003, El-Hashemite et al 2003, Harris & Lawrence 2003, Yeung 2003, Au et al 2004, Birchenall-Roberts et al 2004, Li et al 2004, Mak & Yeung 2004, Zhang et al 2013]. The hamartin tuberin complex can also regulate mTORC2 complex activity that affects cytoskeleton formation and AKT activation [Han & Sahin 2011].

Most TSC1 pathogenic variants and 70% of TSC2 pathogenic variants are predicted to result in a loss of functional protein products. Subsequent loss of function leads to uncontrolled cell growth and cell proliferation resulting in the formation of hamartias (a focal malformation consisting of disorganized arrangement of tissue types that are normally present in the anatomic area) and hamartomas...

...because tuberin and hamartin are subjected to multiple cell signaling pathway regulation, the quantity and quality of both somatic pathogenic variants and environmental factors targeting these pathways are expected to modify disease expression in individuals who have only one normal germline copy of TSC1 or TSC2.

A pathogenic variant is defined as a variant that clearly inactivates the function of the TSC1 or TSC2 proteins (i.e., out-of-frame indel or nonsense variant), prevents protein synthesis (i.e., large genomic deletion), or is a pathogenic missense variant whose effect on protein function has been established by functional assessment (see LOVD Database – TSC1, LOVD Database – TSC2, Hoogeveen-Westerveld et al [2012], and Hoogeveen-Westerveld et al [2013])."

**Gene reviews**
*https://www.ncbi.nlm.nih.gov/books/NBK1220/#tuberous-sclerosis.Molecular_Genetics*

"Over 1,800 different small TSC-causing mutations
have been defined and these are distributed throughout
the coding regions of both genes, except for the final
exon (23) of TSC1 and the alternatively spliced exons
(25 and 31) of TSC2. Mutations are catalogued at http://
chromium.lovd.nl/LOVD2/TSC. The great majority
of TSC1 mutations are small truncating nonsense and
insertion or deletion (indel) mutations with only a small
number of functionally confirmed missense mutations
identified, all of which occur in the 5ʹ region of the
gene15,16. 

Penetrance is almost complete, but some individuals carrying ‘mild mutations’ may not fulfil clinical criteria for definite TSC"

*Henske E et al. 2016 PMID: 27226234*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism appears to be loss of function of TSC1 leading to **decreased/absent or altered gene product** which results in uncontrolled cell growth and proliferation. TSC1 is approximately 50 kb in size and the longest transcript (NM_000368.4), comprises 23 exons. The first two exons are noncoding. Exon 5 and exon 12 are alternatively spliced, producing shorter transcript variants. Most variants are unique with only a few recurrent mutations reported in specific regions including certain codons of exon 15. Mutations are distributed throughout the coding regions of the gene, except for the final exon (23). The majority of mutations are small truncating nonsense and insertion or deletion mutations predicted to cause truncation of the hamartin protein with only a small number of functionally confirmed missense mutations
identified, all of which occur in the 5ʹ region of the gene. Hamartin is a key regulator of the mTOR-AKT pathway. Penetrance is nearly 100% but there is variable expressivity.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame deletion
- In frame duplication
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
