# **TSC2 -- Tuberous sclerosis**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:12363

TSC2, which encodes the protein tuberin, was first reported in relation to autosomal dominant tuberous sclerosis complex (TSC) in 1993 (European Chromosome 16 tuberous Sclerosis Consortium PMID: 8269512). **Numerous variants have been reported in TSC2 in relation to tuberous sclerosis complex, including missense, nonsense, indels and small deletions that result in frameshift, as well as intragenic deletions** (Hasbani and Crino, 2018 PMID: 29478616). The TSC2 database in LOVD (http://chromium.lovd.nl/LOVD2/TSC/home.php?select_db=TSC2) shows a total of 2689 unique variants associated with TSC2 (as of Jan 07, 2019). Evidence supporting this gene-disease relationship includes case-level data and experimental data. This gene-disease relationship has been studied for 20 years, therefore a significant amount of case-level data is available, however the maximum score for genetic evidence (12 points) has been reached. This gene-disease relationship is supported by functional studies including expression and protein interaction and animal models. The molecular mechanism for TSC2 in tuberous sclerosis is loss of function, as indicated by deletion and nonsense variation, as well as biochemical loss of tumor suppressor activity (Hasbani and Crino, 2018 PMID: 29478616). In summary, TSC2 is DEFINITIVELY associated with autosomal dominant tuberous sclerosis complex. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time.

**ClinGen Evidence for Haploinsufficiency**

PMID 31525612 Published in 2019, This study found 3 nonsense variants, 6 splicing variants, a total of 18 frameshift variants, and 2 large deletions in TSC2. Additionally, 51.9% of the cohort (39 individuals) had de novo variants in TSC2.

PMID 10205261 Published in 1999, Jones et al. used PCR and single-strand confirmation polymorphism (SSCP) and Heteroduplex analysis on 150 Tuberous sclerosis complex patients and their families to assess variants in TSC1 and TSC2. The methodology found that 9 cases of variants in TSC2 were familial; 88 other cases involving TSC2 variants were sporadic. This study identified 20 nonsense variants, 21 small insertions or deletions, 8 splice variants, 5 in-frame deletions, and 8 large intragenic deletions in TSC2.

PMID 29932062 Published in 2014, Glushkova et al. used Sanger sequencing and multiplex ligation-dependent probe amplification (MLPA) on 17 unrelated individuals who met the clinical description for Tuberous sclerosis complex (TSC) to search for variants in TSC1 and TSC2. This study identified 2 nonsense variants, 3 frame-shift variants, 1 splice variant, and 1 large deletion in TSC2. In terms of inheritance, 4 individuals had a de novo variant.

Haploinsufficiency phenotype comments:
Variants involving the TSC2 gene, including intragenic and whole-gene heterozygous deletions, cause tuberous sclerosis complex (TSC). TSC is an autosomal dominant multi-system disorder, which affects 1 in 6000 people. About half of these patients are affected by intellectual disability. **PMID 11030407: Reviewed 222 TSC2 mutations that had been reported in the literature. Large genomic deletions (intragenic and whole-gene) accounted for 16% of all variants reported.**

**Literature review:**

Fifty percent (145/292) of TSC2 mutations were point
mutations (see online tables 2, 3). In contrast to TSC1,
nonsense mutations in TSC2 made up only 38% (55 of
145) of this class. TSC2 has seven CpG sites that can undergo transition to nonsense codons; however, mutations
have only been seen at five of these and they comprise
only 36% of the nonsense mutations. TSC2 missense mutations (35 verified and 24 probable) were also common
and accounted for 41% (59 of 145). **TSC2 has two CpG
sites that appear to be mutational hotspots for missense
mutations: 1831–2 (611R→W and 611R→Q) and 5024–5
(1675P→L). These two CpG sites contain 22 of the 59
(37%) verified or probable missense mutations. Other
than the CpG sites, missense mutations were clustered
(8/574 versus 23/4850, P<0.02) in the region with
rap1GAP homology (exons 34–38).** Splice mutations (of
which three were seen recurrently) made up the remaining
21% of point mutations in TSC2.

*Cheadle J 2000 PMID 11030407*

70% of TSC2 pathogenic variants are predicted to result in a loss of functional protein products. Subsequent loss of function leads to uncontrolled cell growth and cell proliferation resulting in the formation of hamartias (a focal malformation consisting of disorganized arrangement of tissue types that are normally present in the anatomic area) and hamartomas.

More than 1,900 unique TSC2 pathogenic variants distributed throughout the gene have been identified in more than 5,800 individuals/families with TSC2-related TSC. **Approximately 33% of TSC2 pathogenic variants are located in exons 32-41 (and associated splice sites) that encode the carboxy domain of tuberin** consisting of several important functional motifs (e.g., GAP domain, estrogen receptor- and calmodulin-binding domains, and multiple signal pathway kinase targets).

Missense variants account for approximately 26% of all TSC2 pathogenic variants with approximately 50% concentrated in the carboxy domain. Missense variants are rarely the direct target of kinases: only two missense variants at the Tyr1571 residue are the predicted target of tyrosine kinase [Hoogeveen-Westerveld et al 2013].

A higher percentage of individuals with more severe features of TSC have a de novo TSC2 pathogenic variant versus a de novo TSC1 pathogenic variant [Au et al 2007]. Individuals representing simplex cases (i.e., a single occurrence in a family) are more likely to have a TSC2 pathogenic variant, while those with familial TSC have an almost equal proportion of TSC1 and TSC2 pathogenic variants [Au et al 2007].

Individuals with a TSC2 pathogenic variant are at greater risk for:
Renal malignancy [Yang et al 2014]
Intellectual disability[Kothare et al 2014]
Autistic disorder, low IQ, and infantile spasms [Numis et al 2011]

**Gene reviews**
*https://www.ncbi.nlm.nih.gov/books/NBK1220/#tuberous-sclerosis.Molecular_Genetics*

"Over 1,800 different small TSC-causing mutations
have been defined and these are distributed throughout
the coding regions of both genes, except for the final
exon (23) of TSC1 and the alternatively spliced exons
(25 and 31) of TSC2. Mutations are catalogued at http://
chromium.lovd.nl/LOVD2/TSC. TSC2 mutations include frequent
missense mutations (30% of cases) and large deletions
and other rearrangements (5% of cases).

Penetrance is almost complete, but some individuals carrying ‘mild mutations’ may not fulfil clinical criteria for definite TSC"

*Henske E et al. 2016 PMID: 27226234*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism appears to be loss of function of TSC2 leading to **decreased/absent or altered gene product** which results in uncontrolled cell growth and proliferation. Missense variants account for approximately 30% of all TSC2 pathogenic variants with approximately 50% concentrated in the carboxy domain. Variants are distributed throughout the coding regions of TSC2, except for the alternatively spliced exons (25 and 31). Approximately 33% of TSC2 pathogenic variants are located in exons 32-41 (and associated splice sites) that encode the carboxy domain of tuberin. PMID 11030407: Reviewed 222 TSC2 mutations that had been reported in the literature. Large genomic deletions (intragenic and whole-gene) accounted for 16% of all variants reported.
Penetrance is nearly 100% but there is variable expressivity.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame deletion
- In frame duplication
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
