# **VHL -- Von Hippel-Lindau**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:12687

The VHL gene is associated with the autosomal dominant cancer syndrome von Hippel Lindau disease which includes retinal hemangiomas, cerebellar and spinal hemangioblastomas, renal cell carcinoma and pheochromocytomas https://www.ncbi.nlm.nih.gov/books/NBK1463/. . Following multiple reports of segregation and linkage analysis from large pedigrees of VHL-affected families (PMID: 6582782, PMID: 2642584, PMID: 1982450, PMID: 1680799), the VHL gene was identified by Latif et al in 1993 (PMID: 8493574). Numerous variants have been reported in VHL in relation to the development of von Hippel Lindau disease, and this gene curation relies on a cohort of 114 cases with 12 unique VHL truncating and missense variants from Chen et al 1995 (PMID:7728151) . Although the present curation does not distinguish between VHL Type 1 and Type 2, generally, truncating and grossly damaging missense are associated with VHL Type 1 while less damaging missense are associated with VHL Type 2. The present curation does not include Chuvash Polycythemia or associated variants. Other databases describing germline VHL variants include: Clinical Interpretations of Variants in Cancer (https://civicdb.org), VHLdb (http://vhldb.bio.unipd.it/home), and large compendiums of genetic variants (PMID: 20151405). There is extensive genetic evidence supporting this gene-disease relationship including both case-level and familial data. The VHL gene is a tumor-suppressor with a wide variety of cellular functions (PMID: 25533676), most notably in regulation of Hypoxia Inducible Factor A (HIFa) through binding to hydroxylated HIFa, which induces subsequent ubiquitination and proteasomal degradation (PMID: 25533676). To bind hydroxylated HIFa, VHL is assembled into a complex with CUL2, RBX1 and Elongin B and C (VCB complex). The molecular mechanism of VHL dysfunction in von Hippel Lindau disease involves mutations which result in absent or reduced assembly of the VCB complex, or absent/reduced binding of the VCB complex (VHL) to HIFa. This gene-disease relationship is supported experimentally by expression studies, biochemical function, functional alterations and animal models. In summary, VHL is definitively associated with the autosomal autosomal dominant von Hippel Lindau syndrome. This has been demonstrated in both the research and clinical diagnostic settings and has been upheld over time.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**ClinGen Evidence for Haploinsufficiency**
Loss-of-function-type mutations (including whole gene deletions) in the gene VHL are associated with autosomal dominant von Hippel-Lindau syndrome (VHL, OMIM: 193300). VHL is a cancer predisposition syndrome characterized by the development of hemangioblastomas of the brain, spine, and retina, renal lesions (including renal cysts and renal cell carcinoma), pheochromocytoma, paragangliomas, pancreatic lesions (including pancreatic cysts and neuroendocrine tumors), endolymphatic sac tumors, epididymal or papillary cystadenomas, and additional tumor types. The mean age-of-onset for VHL is ~26 years and the penetrance is approximately 87-97% by age 60 (see GeneReviews, OMIM, and PMIDs: 27966541 and 2274658).

Of note, homozygous or trans-heterozygous (germline) mutations in VHL are associated with the autosomal recessive condition, Familial Erythrocytosis-2 (OMIM: 263400)

**Literature review:**

**Our analysis of all VHL families found 52% had missense, 13%
had frameshift, 11% had nonsense, 6% had in-frame deletions/
insertions, 11% had large/complete deletions, and 7% had splice
mutations** (Tables 2–4, Fig. 3). Of the VHL Type 1 families 43, 17,13, 9, 8, and 10 were missense, frameshift, nonsense, splice, inframe deletion/insertions, and partial/complete deletions, respectively (Fig. 3). As expected, VHL Type 2 families mainly had
missense mutations (83.5%). However, this is not as high as some
studies, reporting up to 96% of those with pheochromocytomas
to have missense mutations [Zbar et al., 1996]. We also found
VHL Type 2 families had 7, 5, 4, 0.5, and 0.5% nonsense,
frameshift, splice, in-frame deletion/insertions, and partial deletions, respectively (Fig. 3). The small percentage of nonsense and partial deletions along with the absence of complete deletions
supports theories that an intact though altered pVHL is associated
with pheochromocytomas.

*Nordstrom-O'Brien M et al 2010 PMID: 20151405*

**Gene reviews**
More than 500 germline pathogenic variants have been identified in families with von Hippel-Lindau (VHL) syndrome (see Table A) [Nordstrom-O'Brien et al 2010]. The spectrum of pathogenic variants reported includes 52% missense, 13% frameshift, 11% nonsense, 6% in-frame deletions/insertions, 11% large/complete deletions, and 7% splice-site variants. Single-nucleotide variants have been identified in all three exons. The arginine codon 167 is considered a mutational "hot spot."

Synonymous pathogenic variants in VHL exon 2 that alter splicing through exon 2-skipping are associated with erythrocytosis or VHL disease in five families [Lenglet et al 2018].

Pathogenic variants in VHL either prevent its expression (i.e., deletions, frameshifts, nonsense variants, and splice site variants) or lead to the expression of an abnormal protein (i.e., pathogenic missense variants). The type of VHL that results from a pathogenic missense variant depends on its effect on the three-dimensional structure of the protein [Stebbins et al 1999]. Pathogenic variants in VHL cause misfolding and subsequent chaperonin-mediated breakdown [Feldman et al 2003]. Pathogenic missense variants that destabilize packing of the alpha-helical domains, decrease the stability of the alpha-beta domain interface, interfere with binding of elongin C and HIF1α, or disrupt hydrophobic core residues result in loss of HIF regulation and are more likely to result in VHL type 1. Pathogenic missense variants that result in pVHL that is normal with respect to HIF regulation are more likely to be associated with VHL type 2 (see Genotype-Phenotype Correlations). Furthermore, VHL pathogenic variants affect vessel branching and maturation via the Notch signaling pathway [Arreola et al 2018].

VHL pathogenic variants are highly penetrant. Almost all individuals who have a pathogenic variant in VHL are symptomatic by age 65 years [Maher et al 1991].

*https://www.ncbi.nlm.nih.gov/books/NBK1463/#vhl*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Loss-of-function-type mutations (including whole gene deletions) in the gene VHL are associated with autosomal dominant von Hippel-Lindau syndrome. "The spectrum of pathogenic variants reported includes 52% missense, 13% frameshift, 11% nonsense, 6% in-frame deletions/insertions, 11% large/complete deletions, and 7% splice-site variants. Single-nucleotide variants have been identified in all three exons. The arginine codon 167 is considered a mutational "hot spot." Decreased/absent or altered gene product causes absent or reduced assembly of the VCB complex, or absent/reduced binding of the VCB complex (VHL) to HIFa. The mean age-of-onset for VHL is ~26 years. VHL pathogenic variants are highly penetrant. Almost all individuals who have a pathogenic variant in VHL are symptomatic by age 65 years
As per ClinGen, truncating variants and grossly damaging missense are associated with VHL Type 1 while less damaging missense are associated with VHL Type 2. Of note, homozygous or trans-heterozygous (germline) mutations in VHL are associated with the autosomal recessive condition, Familial Erythrocytosis-2 (OMIM: 263400)

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense
- In frame deletion
- In frame duplication
- Synonymous (alter splicing)
- Structural variant
- transcript ablation

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
